{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "sample_df = pd.read_csv(\"/Users/hatice.irmak/Desktop/somatic_rules/cinli.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "pop_columns = [i for i in sample_df.columns if \"ExAC\" in i or \"gnomAD\" in i]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['ExAC_ALL',\n",
       " 'ExAC_AFR',\n",
       " 'ExAC_AMR',\n",
       " 'ExAC_EAS',\n",
       " 'ExAC_FIN',\n",
       " 'ExAC_NFE',\n",
       " 'ExAC_OTH',\n",
       " 'ExAC_SAS',\n",
       " 'gnomAD_exome_ALL',\n",
       " 'gnomAD_exome_AFR',\n",
       " 'gnomAD_exome_AMR',\n",
       " 'gnomAD_exome_ASJ',\n",
       " 'gnomAD_exome_EAS',\n",
       " 'gnomAD_exome_FIN',\n",
       " 'gnomAD_exome_NFE',\n",
       " 'gnomAD_exome_OTH',\n",
       " 'gnomAD_exome_SAS',\n",
       " 'gnomAD_genome_ALL',\n",
       " 'gnomAD_genome_AFR',\n",
       " 'gnomAD_genome_AMR',\n",
       " 'gnomAD_genome_ASJ',\n",
       " 'gnomAD_genome_EAS',\n",
       " 'gnomAD_genome_FIN',\n",
       " 'gnomAD_genome_NFE',\n",
       " 'gnomAD_genome_OTH']"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pop_columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "def pop_rules(row, pop_columns_):\n",
    "    print(row[pop_columns_])\n",
    "    val = row[pop_columns_].replace(\".\",\"0.0\").astype(float)\n",
    "    if (val >= 0.05).any():\n",
    "        return \"SBVS1\"\n",
    "    elif (val >= 0.01).any():\n",
    "        return \"SBS1\"\n",
    "    else:\n",
    "        return \"OP4\"\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ExAC_ALL             3.323e-05\n",
      "ExAC_AFR              9.87e-05\n",
      "ExAC_AMR             8.715e-05\n",
      "ExAC_EAS                0.0001\n",
      "ExAC_FIN                     0\n",
      "ExAC_NFE              1.51e-05\n",
      "ExAC_OTH                     0\n",
      "ExAC_SAS                     0\n",
      "gnomAD_exome_ALL     2.041e-05\n",
      "gnomAD_exome_AFR        0.0001\n",
      "gnomAD_exome_AMR     2.993e-05\n",
      "gnomAD_exome_ASJ             0\n",
      "gnomAD_exome_EAS     5.818e-05\n",
      "gnomAD_exome_FIN             0\n",
      "gnomAD_exome_NFE     9.005e-06\n",
      "gnomAD_exome_OTH             0\n",
      "gnomAD_exome_SAS             0\n",
      "gnomAD_genome_ALL    3.234e-05\n",
      "gnomAD_genome_AFR       0.0001\n",
      "gnomAD_genome_AMR            0\n",
      "gnomAD_genome_ASJ            0\n",
      "gnomAD_genome_EAS            0\n",
      "gnomAD_genome_FIN            0\n",
      "gnomAD_genome_NFE            0\n",
      "gnomAD_genome_OTH            0\n",
      "Name: 0, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 1, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 2, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     0\n",
      "gnomAD_exome_AFR     0\n",
      "gnomAD_exome_AMR     0\n",
      "gnomAD_exome_ASJ     0\n",
      "gnomAD_exome_EAS     0\n",
      "gnomAD_exome_FIN     0\n",
      "gnomAD_exome_NFE     0\n",
      "gnomAD_exome_OTH     0\n",
      "gnomAD_exome_SAS     0\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 3, dtype: object\n",
      "ExAC_ALL             1.648e-05\n",
      "ExAC_AFR                     0\n",
      "ExAC_AMR                     0\n",
      "ExAC_EAS                     0\n",
      "ExAC_FIN                     0\n",
      "ExAC_NFE             2.998e-05\n",
      "ExAC_OTH                     0\n",
      "ExAC_SAS                     0\n",
      "gnomAD_exome_ALL     2.461e-05\n",
      "gnomAD_exome_AFR     6.535e-05\n",
      "gnomAD_exome_AMR             0\n",
      "gnomAD_exome_ASJ             0\n",
      "gnomAD_exome_EAS             0\n",
      "gnomAD_exome_FIN             0\n",
      "gnomAD_exome_NFE     4.573e-05\n",
      "gnomAD_exome_OTH             0\n",
      "gnomAD_exome_SAS             0\n",
      "gnomAD_genome_ALL            .\n",
      "gnomAD_genome_AFR            .\n",
      "gnomAD_genome_AMR            .\n",
      "gnomAD_genome_ASJ            .\n",
      "gnomAD_genome_EAS            .\n",
      "gnomAD_genome_FIN            .\n",
      "gnomAD_genome_NFE            .\n",
      "gnomAD_genome_OTH            .\n",
      "Name: 4, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 5, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 6, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 7, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 8, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 9, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 10, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 11, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 12, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 13, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 14, dtype: object\n",
      "ExAC_ALL             8.251e-05\n",
      "ExAC_AFR                0.0002\n",
      "ExAC_AMR                     0\n",
      "ExAC_EAS                     0\n",
      "ExAC_FIN                     0\n",
      "ExAC_NFE                0.0001\n",
      "ExAC_OTH                     0\n",
      "ExAC_SAS                     0\n",
      "gnomAD_exome_ALL     5.687e-05\n",
      "gnomAD_exome_AFR             0\n",
      "gnomAD_exome_AMR             0\n",
      "gnomAD_exome_ASJ             0\n",
      "gnomAD_exome_EAS             0\n",
      "gnomAD_exome_FIN             0\n",
      "gnomAD_exome_NFE        0.0001\n",
      "gnomAD_exome_OTH             0\n",
      "gnomAD_exome_SAS             0\n",
      "gnomAD_genome_ALL            .\n",
      "gnomAD_genome_AFR            .\n",
      "gnomAD_genome_AMR            .\n",
      "gnomAD_genome_ASJ            .\n",
      "gnomAD_genome_EAS            .\n",
      "gnomAD_genome_FIN            .\n",
      "gnomAD_genome_NFE            .\n",
      "gnomAD_genome_OTH            .\n",
      "Name: 15, dtype: object\n",
      "ExAC_ALL             0.0500\n",
      "ExAC_AFR             0.0788\n",
      "ExAC_AMR             0.0374\n",
      "ExAC_EAS             0.0072\n",
      "ExAC_FIN             0.0250\n",
      "ExAC_NFE             0.0557\n",
      "ExAC_OTH             0.0331\n",
      "ExAC_SAS             0.0512\n",
      "gnomAD_exome_ALL     0.0486\n",
      "gnomAD_exome_AFR     0.0786\n",
      "gnomAD_exome_AMR     0.0358\n",
      "gnomAD_exome_ASJ     0.0866\n",
      "gnomAD_exome_EAS     0.0062\n",
      "gnomAD_exome_FIN     0.0250\n",
      "gnomAD_exome_NFE     0.0551\n",
      "gnomAD_exome_OTH     0.0595\n",
      "gnomAD_exome_SAS     0.0509\n",
      "gnomAD_genome_ALL    0.0554\n",
      "gnomAD_genome_AFR    0.0794\n",
      "gnomAD_genome_AMR    0.0310\n",
      "gnomAD_genome_ASJ    0.0795\n",
      "gnomAD_genome_EAS    0.0099\n",
      "gnomAD_genome_FIN    0.0278\n",
      "gnomAD_genome_NFE    0.0545\n",
      "gnomAD_genome_OTH    0.0440\n",
      "Name: 16, dtype: object\n",
      "ExAC_ALL             1.648e-05\n",
      "ExAC_AFR                     0\n",
      "ExAC_AMR                     0\n",
      "ExAC_EAS                     0\n",
      "ExAC_FIN                     0\n",
      "ExAC_NFE             2.997e-05\n",
      "ExAC_OTH                     0\n",
      "ExAC_SAS                     0\n",
      "gnomAD_exome_ALL             0\n",
      "gnomAD_exome_AFR             0\n",
      "gnomAD_exome_AMR             0\n",
      "gnomAD_exome_ASJ             0\n",
      "gnomAD_exome_EAS             0\n",
      "gnomAD_exome_FIN             0\n",
      "gnomAD_exome_NFE             0\n",
      "gnomAD_exome_OTH             0\n",
      "gnomAD_exome_SAS             0\n",
      "gnomAD_genome_ALL            .\n",
      "gnomAD_genome_AFR            .\n",
      "gnomAD_genome_AMR            .\n",
      "gnomAD_genome_ASJ            .\n",
      "gnomAD_genome_EAS            .\n",
      "gnomAD_genome_FIN            .\n",
      "gnomAD_genome_NFE            .\n",
      "gnomAD_genome_OTH            .\n",
      "Name: 17, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 18, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     0\n",
      "gnomAD_exome_AFR     0\n",
      "gnomAD_exome_AMR     0\n",
      "gnomAD_exome_ASJ     0\n",
      "gnomAD_exome_EAS     0\n",
      "gnomAD_exome_FIN     0\n",
      "gnomAD_exome_NFE     0\n",
      "gnomAD_exome_OTH     0\n",
      "gnomAD_exome_SAS     0\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 19, dtype: object\n",
      "ExAC_ALL                0.0042\n",
      "ExAC_AFR                0.0440\n",
      "ExAC_AMR                0.0032\n",
      "ExAC_EAS                     0\n",
      "ExAC_FIN                     0\n",
      "ExAC_NFE                0.0002\n",
      "ExAC_OTH                0.0022\n",
      "ExAC_SAS                     0\n",
      "gnomAD_exome_ALL        0.0034\n",
      "gnomAD_exome_AFR        0.0455\n",
      "gnomAD_exome_AMR        0.0026\n",
      "gnomAD_exome_ASJ             0\n",
      "gnomAD_exome_EAS     5.798e-05\n",
      "gnomAD_exome_FIN             0\n",
      "gnomAD_exome_NFE        0.0003\n",
      "gnomAD_exome_OTH        0.0038\n",
      "gnomAD_exome_SAS        0.0001\n",
      "gnomAD_genome_ALL       0.0124\n",
      "gnomAD_genome_AFR       0.0431\n",
      "gnomAD_genome_AMR       0.0012\n",
      "gnomAD_genome_ASJ            0\n",
      "gnomAD_genome_EAS            0\n",
      "gnomAD_genome_FIN            0\n",
      "gnomAD_genome_NFE       0.0003\n",
      "gnomAD_genome_OTH       0.0031\n",
      "Name: 20, dtype: object\n",
      "ExAC_ALL             0.0028\n",
      "ExAC_AFR             0.0296\n",
      "ExAC_AMR             0.0014\n",
      "ExAC_EAS                  0\n",
      "ExAC_FIN                  0\n",
      "ExAC_NFE             0.0002\n",
      "ExAC_OTH             0.0011\n",
      "ExAC_SAS             0.0002\n",
      "gnomAD_exome_ALL     0.0022\n",
      "gnomAD_exome_AFR     0.0289\n",
      "gnomAD_exome_AMR     0.0016\n",
      "gnomAD_exome_ASJ     0.0004\n",
      "gnomAD_exome_EAS          0\n",
      "gnomAD_exome_FIN          0\n",
      "gnomAD_exome_NFE     0.0002\n",
      "gnomAD_exome_OTH     0.0011\n",
      "gnomAD_exome_SAS     0.0002\n",
      "gnomAD_genome_ALL    0.0091\n",
      "gnomAD_genome_AFR    0.0316\n",
      "gnomAD_genome_AMR    0.0012\n",
      "gnomAD_genome_ASJ         0\n",
      "gnomAD_genome_EAS         0\n",
      "gnomAD_genome_FIN         0\n",
      "gnomAD_genome_NFE    0.0002\n",
      "gnomAD_genome_OTH    0.0031\n",
      "Name: 21, dtype: object\n",
      "ExAC_ALL                0.0005\n",
      "ExAC_AFR                     0\n",
      "ExAC_AMR                0.0003\n",
      "ExAC_EAS                     0\n",
      "ExAC_FIN                     0\n",
      "ExAC_NFE                0.0009\n",
      "ExAC_OTH                0.0011\n",
      "ExAC_SAS                     0\n",
      "gnomAD_exome_ALL        0.0006\n",
      "gnomAD_exome_AFR             0\n",
      "gnomAD_exome_AMR        0.0006\n",
      "gnomAD_exome_ASJ        0.0045\n",
      "gnomAD_exome_EAS             0\n",
      "gnomAD_exome_FIN             0\n",
      "gnomAD_exome_NFE        0.0006\n",
      "gnomAD_exome_OTH        0.0009\n",
      "gnomAD_exome_SAS     3.249e-05\n",
      "gnomAD_genome_ALL       0.0002\n",
      "gnomAD_genome_AFR            0\n",
      "gnomAD_genome_AMR       0.0036\n",
      "gnomAD_genome_ASJ       0.0033\n",
      "gnomAD_genome_EAS            0\n",
      "gnomAD_genome_FIN            0\n",
      "gnomAD_genome_NFE       0.0001\n",
      "gnomAD_genome_OTH            0\n",
      "Name: 22, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 23, dtype: object\n",
      "ExAC_ALL             0.0649\n",
      "ExAC_AFR             0.2089\n",
      "ExAC_AMR             0.0504\n",
      "ExAC_EAS                  0\n",
      "ExAC_FIN             0.0242\n",
      "ExAC_NFE             0.0702\n",
      "ExAC_OTH             0.0633\n",
      "ExAC_SAS             0.0181\n",
      "gnomAD_exome_ALL     0.0618\n",
      "gnomAD_exome_AFR     0.2096\n",
      "gnomAD_exome_AMR     0.0522\n",
      "gnomAD_exome_ASJ     0.1313\n",
      "gnomAD_exome_EAS          0\n",
      "gnomAD_exome_FIN     0.0259\n",
      "gnomAD_exome_NFE     0.0657\n",
      "gnomAD_exome_OTH     0.0816\n",
      "gnomAD_exome_SAS     0.0193\n",
      "gnomAD_genome_ALL    0.0952\n",
      "gnomAD_genome_AFR    0.2145\n",
      "gnomAD_genome_AMR    0.0682\n",
      "gnomAD_genome_ASJ    0.1589\n",
      "gnomAD_genome_EAS         0\n",
      "gnomAD_genome_FIN    0.0166\n",
      "gnomAD_genome_NFE    0.0578\n",
      "gnomAD_genome_OTH    0.0471\n",
      "Name: 24, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 25, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 26, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     0\n",
      "gnomAD_exome_AFR     0\n",
      "gnomAD_exome_AMR     0\n",
      "gnomAD_exome_ASJ     0\n",
      "gnomAD_exome_EAS     0\n",
      "gnomAD_exome_FIN     0\n",
      "gnomAD_exome_NFE     0\n",
      "gnomAD_exome_OTH     0\n",
      "gnomAD_exome_SAS     0\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 27, dtype: object\n",
      "ExAC_ALL                0.0004\n",
      "ExAC_AFR                     0\n",
      "ExAC_AMR                0.0002\n",
      "ExAC_EAS                0.0059\n",
      "ExAC_FIN                     0\n",
      "ExAC_NFE                     0\n",
      "ExAC_OTH                     0\n",
      "ExAC_SAS                     0\n",
      "gnomAD_exome_ALL        0.0004\n",
      "gnomAD_exome_AFR             0\n",
      "gnomAD_exome_AMR     5.973e-05\n",
      "gnomAD_exome_ASJ             0\n",
      "gnomAD_exome_EAS        0.0058\n",
      "gnomAD_exome_FIN             0\n",
      "gnomAD_exome_NFE             0\n",
      "gnomAD_exome_OTH        0.0002\n",
      "gnomAD_exome_SAS      9.76e-05\n",
      "gnomAD_genome_ALL       0.0003\n",
      "gnomAD_genome_AFR            0\n",
      "gnomAD_genome_AMR            0\n",
      "gnomAD_genome_ASJ            0\n",
      "gnomAD_genome_EAS       0.0062\n",
      "gnomAD_genome_FIN            0\n",
      "gnomAD_genome_NFE            0\n",
      "gnomAD_genome_OTH            0\n",
      "Name: 28, dtype: object\n",
      "ExAC_ALL             8.317e-06\n",
      "ExAC_AFR                     0\n",
      "ExAC_AMR                     0\n",
      "ExAC_EAS                     0\n",
      "ExAC_FIN                     0\n",
      "ExAC_NFE             1.505e-05\n",
      "ExAC_OTH                     0\n",
      "ExAC_SAS                     0\n",
      "gnomAD_exome_ALL     4.083e-06\n",
      "gnomAD_exome_AFR             0\n",
      "gnomAD_exome_AMR             0\n",
      "gnomAD_exome_ASJ             0\n",
      "gnomAD_exome_EAS             0\n",
      "gnomAD_exome_FIN             0\n",
      "gnomAD_exome_NFE      9.04e-06\n",
      "gnomAD_exome_OTH             0\n",
      "gnomAD_exome_SAS             0\n",
      "gnomAD_genome_ALL            .\n",
      "gnomAD_genome_AFR            .\n",
      "gnomAD_genome_AMR            .\n",
      "gnomAD_genome_ASJ            .\n",
      "gnomAD_genome_EAS            .\n",
      "gnomAD_genome_FIN            .\n",
      "gnomAD_genome_NFE            .\n",
      "gnomAD_genome_OTH            .\n",
      "Name: 29, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 30, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 31, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 32, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 33, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     0\n",
      "gnomAD_exome_AFR     0\n",
      "gnomAD_exome_AMR     0\n",
      "gnomAD_exome_ASJ     0\n",
      "gnomAD_exome_EAS     0\n",
      "gnomAD_exome_FIN     0\n",
      "gnomAD_exome_NFE     0\n",
      "gnomAD_exome_OTH     0\n",
      "gnomAD_exome_SAS     0\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 34, dtype: object\n",
      "ExAC_ALL                     .\n",
      "ExAC_AFR                     .\n",
      "ExAC_AMR                     .\n",
      "ExAC_EAS                     .\n",
      "ExAC_FIN                     .\n",
      "ExAC_NFE                     .\n",
      "ExAC_OTH                     .\n",
      "ExAC_SAS                     .\n",
      "gnomAD_exome_ALL     4.065e-06\n",
      "gnomAD_exome_AFR             0\n",
      "gnomAD_exome_AMR             0\n",
      "gnomAD_exome_ASJ             0\n",
      "gnomAD_exome_EAS             0\n",
      "gnomAD_exome_FIN             0\n",
      "gnomAD_exome_NFE     8.964e-06\n",
      "gnomAD_exome_OTH             0\n",
      "gnomAD_exome_SAS             0\n",
      "gnomAD_genome_ALL            .\n",
      "gnomAD_genome_AFR            .\n",
      "gnomAD_genome_AMR            .\n",
      "gnomAD_genome_ASJ            .\n",
      "gnomAD_genome_EAS            .\n",
      "gnomAD_genome_FIN            .\n",
      "gnomAD_genome_NFE            .\n",
      "gnomAD_genome_OTH            .\n",
      "Name: 35, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 36, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 37, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 38, dtype: object\n",
      "ExAC_ALL             8.238e-06\n",
      "ExAC_AFR                     0\n",
      "ExAC_AMR                     0\n",
      "ExAC_EAS                     0\n",
      "ExAC_FIN                     0\n",
      "ExAC_NFE             1.499e-05\n",
      "ExAC_OTH                     0\n",
      "ExAC_SAS                     0\n",
      "gnomAD_exome_ALL             0\n",
      "gnomAD_exome_AFR             0\n",
      "gnomAD_exome_AMR             0\n",
      "gnomAD_exome_ASJ             0\n",
      "gnomAD_exome_EAS             0\n",
      "gnomAD_exome_FIN             0\n",
      "gnomAD_exome_NFE             0\n",
      "gnomAD_exome_OTH             0\n",
      "gnomAD_exome_SAS             0\n",
      "gnomAD_genome_ALL            .\n",
      "gnomAD_genome_AFR            .\n",
      "gnomAD_genome_AMR            .\n",
      "gnomAD_genome_ASJ            .\n",
      "gnomAD_genome_EAS            .\n",
      "gnomAD_genome_FIN            .\n",
      "gnomAD_genome_NFE            .\n",
      "gnomAD_genome_OTH            .\n",
      "Name: 39, dtype: object\n",
      "ExAC_ALL                     .\n",
      "ExAC_AFR                     .\n",
      "ExAC_AMR                     .\n",
      "ExAC_EAS                     .\n",
      "ExAC_FIN                     .\n",
      "ExAC_NFE                     .\n",
      "ExAC_OTH                     .\n",
      "ExAC_SAS                     .\n",
      "gnomAD_exome_ALL     4.062e-06\n",
      "gnomAD_exome_AFR             0\n",
      "gnomAD_exome_AMR     2.978e-05\n",
      "gnomAD_exome_ASJ             0\n",
      "gnomAD_exome_EAS             0\n",
      "gnomAD_exome_FIN             0\n",
      "gnomAD_exome_NFE             0\n",
      "gnomAD_exome_OTH             0\n",
      "gnomAD_exome_SAS             0\n",
      "gnomAD_genome_ALL            .\n",
      "gnomAD_genome_AFR            .\n",
      "gnomAD_genome_AMR            .\n",
      "gnomAD_genome_ASJ            .\n",
      "gnomAD_genome_EAS            .\n",
      "gnomAD_genome_FIN            .\n",
      "gnomAD_genome_NFE            .\n",
      "gnomAD_genome_OTH            .\n",
      "Name: 40, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 41, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 42, dtype: object\n",
      "ExAC_ALL                     .\n",
      "ExAC_AFR                     .\n",
      "ExAC_AMR                     .\n",
      "ExAC_EAS                     .\n",
      "ExAC_FIN                     .\n",
      "ExAC_NFE                     .\n",
      "ExAC_OTH                     .\n",
      "ExAC_SAS                     .\n",
      "gnomAD_exome_ALL     5.774e-05\n",
      "gnomAD_exome_AFR             0\n",
      "gnomAD_exome_AMR             0\n",
      "gnomAD_exome_ASJ             0\n",
      "gnomAD_exome_EAS        0.0002\n",
      "gnomAD_exome_FIN             0\n",
      "gnomAD_exome_NFE        0.0001\n",
      "gnomAD_exome_OTH             0\n",
      "gnomAD_exome_SAS             0\n",
      "gnomAD_genome_ALL     6.66e-05\n",
      "gnomAD_genome_AFR            0\n",
      "gnomAD_genome_AMR            0\n",
      "gnomAD_genome_ASJ            0\n",
      "gnomAD_genome_EAS            0\n",
      "gnomAD_genome_FIN            0\n",
      "gnomAD_genome_NFE       0.0001\n",
      "gnomAD_genome_OTH            0\n",
      "Name: 43, dtype: object\n",
      "ExAC_ALL             .\n",
      "ExAC_AFR             .\n",
      "ExAC_AMR             .\n",
      "ExAC_EAS             .\n",
      "ExAC_FIN             .\n",
      "ExAC_NFE             .\n",
      "ExAC_OTH             .\n",
      "ExAC_SAS             .\n",
      "gnomAD_exome_ALL     .\n",
      "gnomAD_exome_AFR     .\n",
      "gnomAD_exome_AMR     .\n",
      "gnomAD_exome_ASJ     .\n",
      "gnomAD_exome_EAS     .\n",
      "gnomAD_exome_FIN     .\n",
      "gnomAD_exome_NFE     .\n",
      "gnomAD_exome_OTH     .\n",
      "gnomAD_exome_SAS     .\n",
      "gnomAD_genome_ALL    .\n",
      "gnomAD_genome_AFR    .\n",
      "gnomAD_genome_AMR    .\n",
      "gnomAD_genome_ASJ    .\n",
      "gnomAD_genome_EAS    .\n",
      "gnomAD_genome_FIN    .\n",
      "gnomAD_genome_NFE    .\n",
      "gnomAD_genome_OTH    .\n",
      "Name: 44, dtype: object\n"
     ]
    }
   ],
   "source": [
    "sample_df[\"rules\"] = sample_df.apply(lambda x: pop_rules(x, pop_columns), axis=1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0</th>\n",
       "      <th>Chr</th>\n",
       "      <th>Start</th>\n",
       "      <th>End</th>\n",
       "      <th>Ref</th>\n",
       "      <th>Alt</th>\n",
       "      <th>Func.refGene</th>\n",
       "      <th>Gene.refGene</th>\n",
       "      <th>GeneDetail.refGene</th>\n",
       "      <th>ExonicFunc.refGene</th>\n",
       "      <th>...</th>\n",
       "      <th>dbscSNV_ADA_SCORE</th>\n",
       "      <th>dbscSNV_RF_SCORE</th>\n",
       "      <th>MCAP</th>\n",
       "      <th>CLNALLELEID</th>\n",
       "      <th>CLNDN</th>\n",
       "      <th>CLNDISDB</th>\n",
       "      <th>CLNREVSTAT</th>\n",
       "      <th>CLNSIG</th>\n",
       "      <th>REVEL</th>\n",
       "      <th>rules</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>12</td>\n",
       "      <td>25209871</td>\n",
       "      <td>25209871</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.568996615134</td>\n",
       "      <td>54295</td>\n",
       "      <td>Rasopathy|not_specified</td>\n",
       "      <td>MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...</td>\n",
       "      <td>criteria_provided\\x2c_multiple_submitters\\x2c_...</td>\n",
       "      <td>Uncertain_significance</td>\n",
       "      <td>0.745</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>12</td>\n",
       "      <td>25227330</td>\n",
       "      <td>25227330</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.123105525756</td>\n",
       "      <td>432423</td>\n",
       "      <td>Noonan_syndrome_3|Rasopathy</td>\n",
       "      <td>MONDO:MONDO:0012371\\x2cMedGen:C1860991\\x2cOMIM...</td>\n",
       "      <td>reviewed_by_expert_panel</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>0.600</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>12</td>\n",
       "      <td>25227343</td>\n",
       "      <td>25227343</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0655137715661</td>\n",
       "      <td>175715</td>\n",
       "      <td>Non-small_cell_lung_cancer|Neoplasm_of_the_thy...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0030358\\x2cMONDO:M...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.630</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>12</td>\n",
       "      <td>25245350</td>\n",
       "      <td>25245350</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0910971912578</td>\n",
       "      <td>54289</td>\n",
       "      <td>Multiple_myeloma|Lung_adenocarcinoma|Non-small...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0006775\\x2cMONDO:M...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Conflicting_interpretations_of_pathogenicity</td>\n",
       "      <td>0.841</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>13</td>\n",
       "      <td>28018483</td>\n",
       "      <td>28018483</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.293007099215</td>\n",
       "      <td>362981</td>\n",
       "      <td>Acute_myeloid_leukemia</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>0.925</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>5</td>\n",
       "      <td>13</td>\n",
       "      <td>28018503</td>\n",
       "      <td>28018503</td>\n",
       "      <td>A</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.225817333818</td>\n",
       "      <td>362851</td>\n",
       "      <td>Acute_myeloid_leukemia</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.629</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>6</td>\n",
       "      <td>13</td>\n",
       "      <td>28018504</td>\n",
       "      <td>28018505</td>\n",
       "      <td>TC</td>\n",
       "      <td>AT</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonframeshift substitution</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>7</td>\n",
       "      <td>13</td>\n",
       "      <td>28018504</td>\n",
       "      <td>28018504</td>\n",
       "      <td>T</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.264924784705</td>\n",
       "      <td>31311</td>\n",
       "      <td>Acute_myeloid_leukemia|Acute_lymphoid_leukemia</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.894</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>8</td>\n",
       "      <td>13</td>\n",
       "      <td>28018505</td>\n",
       "      <td>28018505</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0876328368379</td>\n",
       "      <td>31312</td>\n",
       "      <td>Acute_myeloid_leukemia</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.878</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>9</td>\n",
       "      <td>13</td>\n",
       "      <td>28018505</td>\n",
       "      <td>28018505</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0837176557721</td>\n",
       "      <td>31315</td>\n",
       "      <td>Acute_myeloid_leukemia|Acute_lymphoid_leukemia</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.875</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10</td>\n",
       "      <td>13</td>\n",
       "      <td>28027217</td>\n",
       "      <td>28027217</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.214409313169</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.883</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>11</td>\n",
       "      <td>13</td>\n",
       "      <td>28027218</td>\n",
       "      <td>28027218</td>\n",
       "      <td>A</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.297566568267</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.917</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>12</td>\n",
       "      <td>13</td>\n",
       "      <td>28027222</td>\n",
       "      <td>28027222</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0964035798672</td>\n",
       "      <td>362984</td>\n",
       "      <td>Acute_myeloid_leukemia</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>0.632</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>13</td>\n",
       "      <td>13</td>\n",
       "      <td>28028203</td>\n",
       "      <td>28028203</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.108688019891</td>\n",
       "      <td>362988</td>\n",
       "      <td>Acute_myeloid_leukemia</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>0.465</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>14</td>\n",
       "      <td>2</td>\n",
       "      <td>208242060</td>\n",
       "      <td>208242060</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0238719972834</td>\n",
       "      <td>626059</td>\n",
       "      <td>Lymphoma</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0002665\\x2cMONDO:M...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Uncertain_significance</td>\n",
       "      <td>0.264</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>15</td>\n",
       "      <td>2</td>\n",
       "      <td>208243466</td>\n",
       "      <td>208243466</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.417115451694</td>\n",
       "      <td>138258</td>\n",
       "      <td>not_specified</td>\n",
       "      <td>MedGen:CN169374</td>\n",
       "      <td>no_assertion_provided</td>\n",
       "      <td>not_provided</td>\n",
       "      <td>0.908</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>16</td>\n",
       "      <td>2</td>\n",
       "      <td>208243593</td>\n",
       "      <td>208243593</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>138259</td>\n",
       "      <td>not_specified</td>\n",
       "      <td>MedGen:CN169374</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Benign</td>\n",
       "      <td>0.421</td>\n",
       "      <td>SBVS1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>17</td>\n",
       "      <td>2</td>\n",
       "      <td>208248388</td>\n",
       "      <td>208248388</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.560930781166</td>\n",
       "      <td>166215</td>\n",
       "      <td>Hepatocellular_carcinoma|Acute_myeloid_leukemi...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...</td>\n",
       "      <td>criteria_provided\\x2c_multiple_submitters\\x2c_...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.852</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>18</td>\n",
       "      <td>2</td>\n",
       "      <td>208248389</td>\n",
       "      <td>208248389</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.612118206497</td>\n",
       "      <td>362772</td>\n",
       "      <td>Hepatocellular_carcinoma|Acute_myeloid_leukemi...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Pathogenic/Likely_pathogenic</td>\n",
       "      <td>0.904</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>19</td>\n",
       "      <td>2</td>\n",
       "      <td>208248389</td>\n",
       "      <td>208248389</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.530890426099</td>\n",
       "      <td>362770</td>\n",
       "      <td>Hepatocellular_carcinoma|Acute_myeloid_leukemi...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...</td>\n",
       "      <td>criteria_provided\\x2c_multiple_submitters\\x2c_...</td>\n",
       "      <td>Pathogenic/Likely_pathogenic</td>\n",
       "      <td>0.814</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>20</td>\n",
       "      <td>2</td>\n",
       "      <td>208248572</td>\n",
       "      <td>208248572</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>138255</td>\n",
       "      <td>not_specified|not_provided</td>\n",
       "      <td>MedGen:CN169374|MedGen:CN517202</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Benign</td>\n",
       "      <td>0.103</td>\n",
       "      <td>SBS1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>21</td>\n",
       "      <td>2</td>\n",
       "      <td>208248664</td>\n",
       "      <td>208248664</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>splicing</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>NM_001282387:exon4:c.123-4C&gt;T;NM_005896:exon4:...</td>\n",
       "      <td>.</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>.</td>\n",
       "      <td>778916</td>\n",
       "      <td>not_provided</td>\n",
       "      <td>MedGen:CN517202</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Benign</td>\n",
       "      <td>.</td>\n",
       "      <td>SBS1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>22</td>\n",
       "      <td>2</td>\n",
       "      <td>208251458</td>\n",
       "      <td>208251458</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0549463899991</td>\n",
       "      <td>138254</td>\n",
       "      <td>not_specified|not_provided</td>\n",
       "      <td>MedGen:CN169374|MedGen:CN517202</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Likely_benign</td>\n",
       "      <td>0.311</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>23</td>\n",
       "      <td>3</td>\n",
       "      <td>179198826</td>\n",
       "      <td>179198826</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>startloss</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.2576582387</td>\n",
       "      <td>827717</td>\n",
       "      <td>Epidermal_nevus|Megalencephaly-capillary_malfo...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0010816\\x2cMONDO:M...</td>\n",
       "      <td>criteria_provided\\x2c_multiple_submitters\\x2c_...</td>\n",
       "      <td>Uncertain_significance</td>\n",
       "      <td>0.700</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>24</td>\n",
       "      <td>3</td>\n",
       "      <td>179209622</td>\n",
       "      <td>179209622</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>138777</td>\n",
       "      <td>not_specified</td>\n",
       "      <td>MedGen:CN169374</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Benign</td>\n",
       "      <td>0.236</td>\n",
       "      <td>SBVS1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>25</td>\n",
       "      <td>3</td>\n",
       "      <td>179210186</td>\n",
       "      <td>179210186</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.9987</td>\n",
       "      <td>0.914</td>\n",
       "      <td>0.134419997912</td>\n",
       "      <td>54631</td>\n",
       "      <td>Congenital_lipomatous_overgrowth\\x2c_vascular_...</td>\n",
       "      <td>MONDO:MONDO:0013038\\x2cMedGen:C2752042\\x2cOMIM...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Uncertain_significance</td>\n",
       "      <td>0.519</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>26</td>\n",
       "      <td>3</td>\n",
       "      <td>179210291</td>\n",
       "      <td>179210291</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.113788654374</td>\n",
       "      <td>363349</td>\n",
       "      <td>Hepatocellular_carcinoma|Abnormality_of_cardio...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...</td>\n",
       "      <td>criteria_provided\\x2c_multiple_submitters\\x2c_...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.544</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>27</td>\n",
       "      <td>3</td>\n",
       "      <td>179218304</td>\n",
       "      <td>179218304</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.088721466783</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.715</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>28</td>\n",
       "      <td>3</td>\n",
       "      <td>179224091</td>\n",
       "      <td>179224091</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0164815254628</td>\n",
       "      <td>452429</td>\n",
       "      <td>Cowden_syndrome</td>\n",
       "      <td>MONDO:MONDO:0016063\\x2cMedGen:C0018553\\x2cOMIM...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Benign</td>\n",
       "      <td>0.269</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>29</td>\n",
       "      <td>3</td>\n",
       "      <td>179234297</td>\n",
       "      <td>179234297</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0373781845074</td>\n",
       "      <td>28691</td>\n",
       "      <td>MACRODACTYLY\\x2c_SOMATIC|Hepatocellular_carcin...</td>\n",
       "      <td>.|Human_Phenotype_Ontology:HP:0001402\\x2cHuman...</td>\n",
       "      <td>criteria_provided\\x2c_multiple_submitters\\x2c_...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.455</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>30</td>\n",
       "      <td>3</td>\n",
       "      <td>179234298</td>\n",
       "      <td>179234298</td>\n",
       "      <td>T</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.026514517841</td>\n",
       "      <td>363360</td>\n",
       "      <td>Hepatocellular_carcinoma|Medulloblastoma|Adren...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>0.428</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>31</td>\n",
       "      <td>3</td>\n",
       "      <td>179234350</td>\n",
       "      <td>179234350</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0220153461798</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.384</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>32</td>\n",
       "      <td>3</td>\n",
       "      <td>179234360</td>\n",
       "      <td>179234360</td>\n",
       "      <td>-</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>frameshift insertion</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>33</td>\n",
       "      <td>5</td>\n",
       "      <td>1282455</td>\n",
       "      <td>1282455</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>TERT</td>\n",
       "      <td>.</td>\n",
       "      <td>stopgain</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>829942</td>\n",
       "      <td>Idiopathic_Pulmonary_Fibrosis|Dyskeratosis_con...</td>\n",
       "      <td>MONDO:MONDO:0008345\\x2cMedGen:C1800706\\x2cOMIM...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>.</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>34</td>\n",
       "      <td>5</td>\n",
       "      <td>1294198</td>\n",
       "      <td>1294198</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>TERT</td>\n",
       "      <td>.</td>\n",
       "      <td>stopgain</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>633029</td>\n",
       "      <td>Idiopathic_Pulmonary_Fibrosis|Dyskeratosis_con...</td>\n",
       "      <td>MONDO:MONDO:0008345\\x2cMedGen:C1800706\\x2cOMIM...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>.</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>35</td>\n",
       "      <td>7</td>\n",
       "      <td>140753334</td>\n",
       "      <td>140753334</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0820752694957</td>\n",
       "      <td>29005</td>\n",
       "      <td>Neoplasm|Melanoma|Chronic_lymphocytic_leukemia...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0002664\\x2cHuman_P...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.916</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>36</td>\n",
       "      <td>7</td>\n",
       "      <td>140753336</td>\n",
       "      <td>140753337</td>\n",
       "      <td>AC</td>\n",
       "      <td>TT</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonframeshift substitution</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>362820</td>\n",
       "      <td>Trametinib-Dabrafenib_Response|Vemurafenib-Cob...</td>\n",
       "      <td>.|.|Human_Phenotype_Ontology:HP:0002861\\x2cHum...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>drug_response</td>\n",
       "      <td>.</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>37</td>\n",
       "      <td>7</td>\n",
       "      <td>140753355</td>\n",
       "      <td>140753355</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.596855831475</td>\n",
       "      <td>53980</td>\n",
       "      <td>Melanoma|Adrenocortical_carcinoma|Chronic_lymp...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0002861\\x2cHuman_P...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>0.904</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>38</td>\n",
       "      <td>7</td>\n",
       "      <td>140778037</td>\n",
       "      <td>140778051</td>\n",
       "      <td>TAGGTGCTGTCACAT</td>\n",
       "      <td>-</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonframeshift deletion</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>653824</td>\n",
       "      <td>Dabrafenib_response</td>\n",
       "      <td>MedGen:CN239586</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>drug_response</td>\n",
       "      <td>.</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>39</td>\n",
       "      <td>7</td>\n",
       "      <td>140781602</td>\n",
       "      <td>140781602</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.612203233818</td>\n",
       "      <td>29013</td>\n",
       "      <td>Melanoma|Squamous_cell_carcinoma_of_the_skin|T...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0002861\\x2cHuman_P...</td>\n",
       "      <td>reviewed_by_expert_panel</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.921</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>40</td>\n",
       "      <td>7</td>\n",
       "      <td>140781602</td>\n",
       "      <td>140781602</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.219344139139</td>\n",
       "      <td>29010</td>\n",
       "      <td>Neoplasm|Squamous_cell_carcinoma_of_the_skin|T...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0002664\\x2cHuman_P...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.724</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>41</td>\n",
       "      <td>7</td>\n",
       "      <td>140781675</td>\n",
       "      <td>140781675</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.100260968362</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.627</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>42</td>\n",
       "      <td>7</td>\n",
       "      <td>140783127</td>\n",
       "      <td>140783127</td>\n",
       "      <td>G</td>\n",
       "      <td>-</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>frameshift deletion</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>43</td>\n",
       "      <td>7</td>\n",
       "      <td>140924612</td>\n",
       "      <td>140924612</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.261712773584</td>\n",
       "      <td>54000</td>\n",
       "      <td>Rasopathy|not_specified</td>\n",
       "      <td>MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...</td>\n",
       "      <td>reviewed_by_expert_panel</td>\n",
       "      <td>Likely_benign</td>\n",
       "      <td>0.207</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>44</td>\n",
       "      <td>7</td>\n",
       "      <td>140924663</td>\n",
       "      <td>140924663</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0891133498287</td>\n",
       "      <td>48804</td>\n",
       "      <td>Rasopathy|not_provided</td>\n",
       "      <td>MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...</td>\n",
       "      <td>criteria_provided\\x2c_multiple_submitters\\x2c_...</td>\n",
       "      <td>Uncertain_significance</td>\n",
       "      <td>0.093</td>\n",
       "      <td>OP4</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>45 rows  115 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "    Unnamed: 0  Chr      Start        End              Ref Alt Func.refGene  \\\n",
       "0            0   12   25209871   25209871                C   T       exonic   \n",
       "1            1   12   25227330   25227330                C   A       exonic   \n",
       "2            2   12   25227343   25227343                G   T       exonic   \n",
       "3            3   12   25245350   25245350                C   G       exonic   \n",
       "4            4   13   28018483   28018483                T   C       exonic   \n",
       "5            5   13   28018503   28018503                A   T       exonic   \n",
       "6            6   13   28018504   28018505               TC  AT       exonic   \n",
       "7            7   13   28018504   28018504                T   A       exonic   \n",
       "8            8   13   28018505   28018505                C   G       exonic   \n",
       "9            9   13   28018505   28018505                C   A       exonic   \n",
       "10          10   13   28027217   28027217                T   C       exonic   \n",
       "11          11   13   28027218   28027218                A   T       exonic   \n",
       "12          12   13   28027222   28027222                A   C       exonic   \n",
       "13          13   13   28028203   28028203                G   T       exonic   \n",
       "14          14    2  208242060  208242060                C   T       exonic   \n",
       "15          15    2  208243466  208243466                T   C       exonic   \n",
       "16          16    2  208243593  208243593                C   T       exonic   \n",
       "17          17    2  208248388  208248388                C   T       exonic   \n",
       "18          18    2  208248389  208248389                G   T       exonic   \n",
       "19          19    2  208248389  208248389                G   A       exonic   \n",
       "20          20    2  208248572  208248572                C   T       exonic   \n",
       "21          21    2  208248664  208248664                G   A     splicing   \n",
       "22          22    2  208251458  208251458                A   C       exonic   \n",
       "23          23    3  179198826  179198826                A   G       exonic   \n",
       "24          24    3  179209622  179209622                A   G       exonic   \n",
       "25          25    3  179210186  179210186                G   A       exonic   \n",
       "26          26    3  179210291  179210291                G   A       exonic   \n",
       "27          27    3  179218304  179218304                A   C       exonic   \n",
       "28          28    3  179224091  179224091                A   G       exonic   \n",
       "29          29    3  179234297  179234297                A   G       exonic   \n",
       "30          30    3  179234298  179234298                T   G       exonic   \n",
       "31          31    3  179234350  179234350                C   T       exonic   \n",
       "32          32    3  179234360  179234360                -   A       exonic   \n",
       "33          33    5    1282455    1282455                C   T       exonic   \n",
       "34          34    5    1294198    1294198                G   A       exonic   \n",
       "35          35    7  140753334  140753334                T   C       exonic   \n",
       "36          36    7  140753336  140753337               AC  TT       exonic   \n",
       "37          37    7  140753355  140753355                C   T       exonic   \n",
       "38          38    7  140778037  140778051  TAGGTGCTGTCACAT   -       exonic   \n",
       "39          39    7  140781602  140781602                C   T       exonic   \n",
       "40          40    7  140781602  140781602                C   G       exonic   \n",
       "41          41    7  140781675  140781675                C   G       exonic   \n",
       "42          42    7  140783127  140783127                G   -       exonic   \n",
       "43          43    7  140924612  140924612                G   C       exonic   \n",
       "44          44    7  140924663  140924663                G   C       exonic   \n",
       "\n",
       "   Gene.refGene                                 GeneDetail.refGene  \\\n",
       "0          KRAS                                                  .   \n",
       "1          KRAS                                                  .   \n",
       "2          KRAS                                                  .   \n",
       "3          KRAS                                                  .   \n",
       "4          FLT3                                                  .   \n",
       "5          FLT3                                                  .   \n",
       "6          FLT3                                                  .   \n",
       "7          FLT3                                                  .   \n",
       "8          FLT3                                                  .   \n",
       "9          FLT3                                                  .   \n",
       "10         FLT3                                                  .   \n",
       "11         FLT3                                                  .   \n",
       "12         FLT3                                                  .   \n",
       "13         FLT3                                                  .   \n",
       "14         IDH1                                                  .   \n",
       "15         IDH1                                                  .   \n",
       "16         IDH1                                                  .   \n",
       "17         IDH1                                                  .   \n",
       "18         IDH1                                                  .   \n",
       "19         IDH1                                                  .   \n",
       "20         IDH1                                                  .   \n",
       "21         IDH1  NM_001282387:exon4:c.123-4C>T;NM_005896:exon4:...   \n",
       "22         IDH1                                                  .   \n",
       "23       PIK3CA                                                  .   \n",
       "24       PIK3CA                                                  .   \n",
       "25       PIK3CA                                                  .   \n",
       "26       PIK3CA                                                  .   \n",
       "27       PIK3CA                                                  .   \n",
       "28       PIK3CA                                                  .   \n",
       "29       PIK3CA                                                  .   \n",
       "30       PIK3CA                                                  .   \n",
       "31       PIK3CA                                                  .   \n",
       "32       PIK3CA                                                  .   \n",
       "33         TERT                                                  .   \n",
       "34         TERT                                                  .   \n",
       "35         BRAF                                                  .   \n",
       "36         BRAF                                                  .   \n",
       "37         BRAF                                                  .   \n",
       "38         BRAF                                                  .   \n",
       "39         BRAF                                                  .   \n",
       "40         BRAF                                                  .   \n",
       "41         BRAF                                                  .   \n",
       "42         BRAF                                                  .   \n",
       "43         BRAF                                                  .   \n",
       "44         BRAF                                                  .   \n",
       "\n",
       "            ExonicFunc.refGene  ... dbscSNV_ADA_SCORE dbscSNV_RF_SCORE  \\\n",
       "0            nonsynonymous SNV  ...                 .                .   \n",
       "1            nonsynonymous SNV  ...                 .                .   \n",
       "2            nonsynonymous SNV  ...                 .                .   \n",
       "3            nonsynonymous SNV  ...                 .                .   \n",
       "4            nonsynonymous SNV  ...                 .                .   \n",
       "5            nonsynonymous SNV  ...                 .                .   \n",
       "6   nonframeshift substitution  ...                 .                .   \n",
       "7            nonsynonymous SNV  ...                 .                .   \n",
       "8            nonsynonymous SNV  ...                 .                .   \n",
       "9            nonsynonymous SNV  ...                 .                .   \n",
       "10           nonsynonymous SNV  ...                 .                .   \n",
       "11           nonsynonymous SNV  ...                 .                .   \n",
       "12           nonsynonymous SNV  ...                 .                .   \n",
       "13           nonsynonymous SNV  ...                 .                .   \n",
       "14           nonsynonymous SNV  ...                 .                .   \n",
       "15           nonsynonymous SNV  ...                 .                .   \n",
       "16           nonsynonymous SNV  ...                 .                .   \n",
       "17           nonsynonymous SNV  ...                 .                .   \n",
       "18           nonsynonymous SNV  ...                 .                .   \n",
       "19           nonsynonymous SNV  ...                 .                .   \n",
       "20           nonsynonymous SNV  ...                 .                .   \n",
       "21                           .  ...                 0                0   \n",
       "22           nonsynonymous SNV  ...                 .                .   \n",
       "23                   startloss  ...                 .                .   \n",
       "24           nonsynonymous SNV  ...                 .                .   \n",
       "25           nonsynonymous SNV  ...            0.9987            0.914   \n",
       "26           nonsynonymous SNV  ...                 .                .   \n",
       "27           nonsynonymous SNV  ...                 .                .   \n",
       "28           nonsynonymous SNV  ...                 .                .   \n",
       "29           nonsynonymous SNV  ...                 .                .   \n",
       "30           nonsynonymous SNV  ...                 .                .   \n",
       "31           nonsynonymous SNV  ...                 .                .   \n",
       "32        frameshift insertion  ...                 .                .   \n",
       "33                    stopgain  ...                 .                .   \n",
       "34                    stopgain  ...                 .                .   \n",
       "35           nonsynonymous SNV  ...                 .                .   \n",
       "36  nonframeshift substitution  ...                 .                .   \n",
       "37           nonsynonymous SNV  ...                 .                .   \n",
       "38      nonframeshift deletion  ...                 .                .   \n",
       "39           nonsynonymous SNV  ...                 .                .   \n",
       "40           nonsynonymous SNV  ...                 .                .   \n",
       "41           nonsynonymous SNV  ...                 .                .   \n",
       "42         frameshift deletion  ...                 .                .   \n",
       "43           nonsynonymous SNV  ...                 .                .   \n",
       "44           nonsynonymous SNV  ...                 .                .   \n",
       "\n",
       "               MCAP CLNALLELEID  \\\n",
       "0    0.568996615134       54295   \n",
       "1    0.123105525756      432423   \n",
       "2   0.0655137715661      175715   \n",
       "3   0.0910971912578       54289   \n",
       "4    0.293007099215      362981   \n",
       "5    0.225817333818      362851   \n",
       "6                 .           .   \n",
       "7    0.264924784705       31311   \n",
       "8   0.0876328368379       31312   \n",
       "9   0.0837176557721       31315   \n",
       "10   0.214409313169           .   \n",
       "11   0.297566568267           .   \n",
       "12  0.0964035798672      362984   \n",
       "13   0.108688019891      362988   \n",
       "14  0.0238719972834      626059   \n",
       "15   0.417115451694      138258   \n",
       "16                .      138259   \n",
       "17   0.560930781166      166215   \n",
       "18   0.612118206497      362772   \n",
       "19   0.530890426099      362770   \n",
       "20                .      138255   \n",
       "21                .      778916   \n",
       "22  0.0549463899991      138254   \n",
       "23     0.2576582387      827717   \n",
       "24                .      138777   \n",
       "25   0.134419997912       54631   \n",
       "26   0.113788654374      363349   \n",
       "27   0.088721466783           .   \n",
       "28  0.0164815254628      452429   \n",
       "29  0.0373781845074       28691   \n",
       "30   0.026514517841      363360   \n",
       "31  0.0220153461798           .   \n",
       "32                .           .   \n",
       "33                .      829942   \n",
       "34                .      633029   \n",
       "35  0.0820752694957       29005   \n",
       "36                .      362820   \n",
       "37   0.596855831475       53980   \n",
       "38                .      653824   \n",
       "39   0.612203233818       29013   \n",
       "40   0.219344139139       29010   \n",
       "41   0.100260968362           .   \n",
       "42                .           .   \n",
       "43   0.261712773584       54000   \n",
       "44  0.0891133498287       48804   \n",
       "\n",
       "                                                CLNDN  \\\n",
       "0                             Rasopathy|not_specified   \n",
       "1                         Noonan_syndrome_3|Rasopathy   \n",
       "2   Non-small_cell_lung_cancer|Neoplasm_of_the_thy...   \n",
       "3   Multiple_myeloma|Lung_adenocarcinoma|Non-small...   \n",
       "4                              Acute_myeloid_leukemia   \n",
       "5                              Acute_myeloid_leukemia   \n",
       "6                                                   .   \n",
       "7      Acute_myeloid_leukemia|Acute_lymphoid_leukemia   \n",
       "8                              Acute_myeloid_leukemia   \n",
       "9      Acute_myeloid_leukemia|Acute_lymphoid_leukemia   \n",
       "10                                                  .   \n",
       "11                                                  .   \n",
       "12                             Acute_myeloid_leukemia   \n",
       "13                             Acute_myeloid_leukemia   \n",
       "14                                           Lymphoma   \n",
       "15                                      not_specified   \n",
       "16                                      not_specified   \n",
       "17  Hepatocellular_carcinoma|Acute_myeloid_leukemi...   \n",
       "18  Hepatocellular_carcinoma|Acute_myeloid_leukemi...   \n",
       "19  Hepatocellular_carcinoma|Acute_myeloid_leukemi...   \n",
       "20                         not_specified|not_provided   \n",
       "21                                       not_provided   \n",
       "22                         not_specified|not_provided   \n",
       "23  Epidermal_nevus|Megalencephaly-capillary_malfo...   \n",
       "24                                      not_specified   \n",
       "25  Congenital_lipomatous_overgrowth\\x2c_vascular_...   \n",
       "26  Hepatocellular_carcinoma|Abnormality_of_cardio...   \n",
       "27                                                  .   \n",
       "28                                    Cowden_syndrome   \n",
       "29  MACRODACTYLY\\x2c_SOMATIC|Hepatocellular_carcin...   \n",
       "30  Hepatocellular_carcinoma|Medulloblastoma|Adren...   \n",
       "31                                                  .   \n",
       "32                                                  .   \n",
       "33  Idiopathic_Pulmonary_Fibrosis|Dyskeratosis_con...   \n",
       "34  Idiopathic_Pulmonary_Fibrosis|Dyskeratosis_con...   \n",
       "35  Neoplasm|Melanoma|Chronic_lymphocytic_leukemia...   \n",
       "36  Trametinib-Dabrafenib_Response|Vemurafenib-Cob...   \n",
       "37  Melanoma|Adrenocortical_carcinoma|Chronic_lymp...   \n",
       "38                                Dabrafenib_response   \n",
       "39  Melanoma|Squamous_cell_carcinoma_of_the_skin|T...   \n",
       "40  Neoplasm|Squamous_cell_carcinoma_of_the_skin|T...   \n",
       "41                                                  .   \n",
       "42                                                  .   \n",
       "43                            Rasopathy|not_specified   \n",
       "44                             Rasopathy|not_provided   \n",
       "\n",
       "                                             CLNDISDB  \\\n",
       "0   MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...   \n",
       "1   MONDO:MONDO:0012371\\x2cMedGen:C1860991\\x2cOMIM...   \n",
       "2   Human_Phenotype_Ontology:HP:0030358\\x2cMONDO:M...   \n",
       "3   Human_Phenotype_Ontology:HP:0006775\\x2cMONDO:M...   \n",
       "4   Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...   \n",
       "5   Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...   \n",
       "6                                                   .   \n",
       "7   Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...   \n",
       "8   Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...   \n",
       "9   Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...   \n",
       "10                                                  .   \n",
       "11                                                  .   \n",
       "12  Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...   \n",
       "13  Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...   \n",
       "14  Human_Phenotype_Ontology:HP:0002665\\x2cMONDO:M...   \n",
       "15                                    MedGen:CN169374   \n",
       "16                                    MedGen:CN169374   \n",
       "17  Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...   \n",
       "18  Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...   \n",
       "19  Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...   \n",
       "20                    MedGen:CN169374|MedGen:CN517202   \n",
       "21                                    MedGen:CN517202   \n",
       "22                    MedGen:CN169374|MedGen:CN517202   \n",
       "23  Human_Phenotype_Ontology:HP:0010816\\x2cMONDO:M...   \n",
       "24                                    MedGen:CN169374   \n",
       "25  MONDO:MONDO:0013038\\x2cMedGen:C2752042\\x2cOMIM...   \n",
       "26  Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...   \n",
       "27                                                  .   \n",
       "28  MONDO:MONDO:0016063\\x2cMedGen:C0018553\\x2cOMIM...   \n",
       "29  .|Human_Phenotype_Ontology:HP:0001402\\x2cHuman...   \n",
       "30  Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...   \n",
       "31                                                  .   \n",
       "32                                                  .   \n",
       "33  MONDO:MONDO:0008345\\x2cMedGen:C1800706\\x2cOMIM...   \n",
       "34  MONDO:MONDO:0008345\\x2cMedGen:C1800706\\x2cOMIM...   \n",
       "35  Human_Phenotype_Ontology:HP:0002664\\x2cHuman_P...   \n",
       "36  .|.|Human_Phenotype_Ontology:HP:0002861\\x2cHum...   \n",
       "37  Human_Phenotype_Ontology:HP:0002861\\x2cHuman_P...   \n",
       "38                                    MedGen:CN239586   \n",
       "39  Human_Phenotype_Ontology:HP:0002861\\x2cHuman_P...   \n",
       "40  Human_Phenotype_Ontology:HP:0002664\\x2cHuman_P...   \n",
       "41                                                  .   \n",
       "42                                                  .   \n",
       "43  MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...   \n",
       "44  MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...   \n",
       "\n",
       "                                           CLNREVSTAT  \\\n",
       "0   criteria_provided\\x2c_multiple_submitters\\x2c_...   \n",
       "1                            reviewed_by_expert_panel   \n",
       "2              criteria_provided\\x2c_single_submitter   \n",
       "3                      no_assertion_criteria_provided   \n",
       "4                      no_assertion_criteria_provided   \n",
       "5                      no_assertion_criteria_provided   \n",
       "6                                                   .   \n",
       "7                      no_assertion_criteria_provided   \n",
       "8                      no_assertion_criteria_provided   \n",
       "9                      no_assertion_criteria_provided   \n",
       "10                                                  .   \n",
       "11                                                  .   \n",
       "12                     no_assertion_criteria_provided   \n",
       "13                     no_assertion_criteria_provided   \n",
       "14                     no_assertion_criteria_provided   \n",
       "15                              no_assertion_provided   \n",
       "16             criteria_provided\\x2c_single_submitter   \n",
       "17  criteria_provided\\x2c_multiple_submitters\\x2c_...   \n",
       "18                     no_assertion_criteria_provided   \n",
       "19  criteria_provided\\x2c_multiple_submitters\\x2c_...   \n",
       "20             criteria_provided\\x2c_single_submitter   \n",
       "21             criteria_provided\\x2c_single_submitter   \n",
       "22             criteria_provided\\x2c_single_submitter   \n",
       "23  criteria_provided\\x2c_multiple_submitters\\x2c_...   \n",
       "24             criteria_provided\\x2c_single_submitter   \n",
       "25             criteria_provided\\x2c_single_submitter   \n",
       "26  criteria_provided\\x2c_multiple_submitters\\x2c_...   \n",
       "27                                                  .   \n",
       "28             criteria_provided\\x2c_single_submitter   \n",
       "29  criteria_provided\\x2c_multiple_submitters\\x2c_...   \n",
       "30                     no_assertion_criteria_provided   \n",
       "31                                                  .   \n",
       "32                                                  .   \n",
       "33             criteria_provided\\x2c_single_submitter   \n",
       "34             criteria_provided\\x2c_single_submitter   \n",
       "35             criteria_provided\\x2c_single_submitter   \n",
       "36             criteria_provided\\x2c_single_submitter   \n",
       "37             criteria_provided\\x2c_single_submitter   \n",
       "38                     no_assertion_criteria_provided   \n",
       "39                           reviewed_by_expert_panel   \n",
       "40             criteria_provided\\x2c_single_submitter   \n",
       "41                                                  .   \n",
       "42                                                  .   \n",
       "43                           reviewed_by_expert_panel   \n",
       "44  criteria_provided\\x2c_multiple_submitters\\x2c_...   \n",
       "\n",
       "                                          CLNSIG  REVEL  rules  \n",
       "0                         Uncertain_significance  0.745    OP4  \n",
       "1                              Likely_pathogenic  0.600    OP4  \n",
       "2                                     Pathogenic  0.630    OP4  \n",
       "3   Conflicting_interpretations_of_pathogenicity  0.841    OP4  \n",
       "4                              Likely_pathogenic  0.925    OP4  \n",
       "5                                     Pathogenic  0.629    OP4  \n",
       "6                                              .      .    OP4  \n",
       "7                                     Pathogenic  0.894    OP4  \n",
       "8                                     Pathogenic  0.878    OP4  \n",
       "9                                     Pathogenic  0.875    OP4  \n",
       "10                                             .  0.883    OP4  \n",
       "11                                             .  0.917    OP4  \n",
       "12                             Likely_pathogenic  0.632    OP4  \n",
       "13                             Likely_pathogenic  0.465    OP4  \n",
       "14                        Uncertain_significance  0.264    OP4  \n",
       "15                                  not_provided  0.908    OP4  \n",
       "16                                        Benign  0.421  SBVS1  \n",
       "17                                    Pathogenic  0.852    OP4  \n",
       "18                  Pathogenic/Likely_pathogenic  0.904    OP4  \n",
       "19                  Pathogenic/Likely_pathogenic  0.814    OP4  \n",
       "20                                        Benign  0.103   SBS1  \n",
       "21                                        Benign      .   SBS1  \n",
       "22                                 Likely_benign  0.311    OP4  \n",
       "23                        Uncertain_significance  0.700    OP4  \n",
       "24                                        Benign  0.236  SBVS1  \n",
       "25                        Uncertain_significance  0.519    OP4  \n",
       "26                                    Pathogenic  0.544    OP4  \n",
       "27                                             .  0.715    OP4  \n",
       "28                                        Benign  0.269    OP4  \n",
       "29                                    Pathogenic  0.455    OP4  \n",
       "30                             Likely_pathogenic  0.428    OP4  \n",
       "31                                             .  0.384    OP4  \n",
       "32                                             .      .    OP4  \n",
       "33                                    Pathogenic      .    OP4  \n",
       "34                                    Pathogenic      .    OP4  \n",
       "35                                    Pathogenic  0.916    OP4  \n",
       "36                                 drug_response      .    OP4  \n",
       "37                             Likely_pathogenic  0.904    OP4  \n",
       "38                                 drug_response      .    OP4  \n",
       "39                                    Pathogenic  0.921    OP4  \n",
       "40                                    Pathogenic  0.724    OP4  \n",
       "41                                             .  0.627    OP4  \n",
       "42                                             .      .    OP4  \n",
       "43                                 Likely_benign  0.207    OP4  \n",
       "44                        Uncertain_significance  0.093    OP4  \n",
       "\n",
       "[45 rows x 115 columns]"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sample_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [],
   "source": [
    "sample_df.to_csv(\"cinli.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "import re\n",
    "import pandas as pd\n",
    "def convert_float(value):\n",
    "    try:\n",
    "        return float(value)\n",
    "    except:\n",
    "        return False\n",
    "\n",
    "def prepare_data(sample_df, cancer_hotspots_df):\n",
    "    sample_df['id'] = sample_df.apply(lambda x: x['AAChange.refGene'].split(':')[0]+'_'+x['AAChange.refGene'].split(':p.')[-1], axis=1)\n",
    "    sample_df = pd.merge(sample_df, cancer_hotspots_df, left_on=[\"id\"], right_on=[\"id\"], how=\"left\")\n",
    "    return sample_df\n",
    "\n",
    "class Rules:\n",
    "    def __init__(self, row):\n",
    "        self.variant = row\n",
    "        self.pop_rules()\n",
    "        self.in_silico_rules()\n",
    "        self.cancer_hotspot_rules()\n",
    "\n",
    "    def pop_rules(self):\n",
    "        pop_columns = [i for i in self.variant.index if \"ExAC\" in i or \"gnomAD\" in i]\n",
    "        val = self.variant[pop_columns].replace(\".\",\"0.0\").astype(float)\n",
    "        if (val >= 0.05).any():\n",
    "            self.SBVS1 = True\n",
    "            self.SBS1 = False\n",
    "            self.OP4 =  False\n",
    "        elif (val >= 0.01).any():\n",
    "            self.SBVS1 = False\n",
    "            self.SBS1 = True\n",
    "            self.OP4 =  False\n",
    "        else:\n",
    "            self.SBVS1 = False\n",
    "            self.SBS1 = False\n",
    "            self.OP4 =  True\n",
    "\n",
    "\n",
    "    def in_silico_rules(self):\n",
    "        pathogen_list = 0\n",
    "        benign_list = 0\n",
    "        \n",
    "        if self.variant[\"FATHMM_pred\"] == \"T\":\n",
    "            benign_list += 1\n",
    "        elif self.variant[\"FATHMM_pred\"] == \"D\":\n",
    "            pathogen_list += 1\n",
    "        else:\n",
    "            pass\n",
    "        \n",
    "        if convert_float(self.variant[\"CADD_phred\"]) <= 23.2:\n",
    "            benign_list += 1\n",
    "        elif convert_float(self.variant[\"CADD_phred\"]) >= 25.6:\n",
    "            pathogen_list += 1\n",
    "        else:\n",
    "            pass\n",
    "\n",
    "        if convert_float(self.variant[\"DANN_score\"]) < 0.974:\n",
    "            benign_list += 1\n",
    "        elif convert_float(self.variant[\"DANN_score\"]) >= 0.995:\n",
    "            pathogen_list +=1\n",
    "        else:\n",
    "            pass\n",
    "\n",
    "        if convert_float(self.variant[\"REVEL\"]) <= 0.25:\n",
    "            benign_list += 1\n",
    "        elif convert_float(self.variant[\"REVEL\"]) >= 0.75:\n",
    "            pathogen_list += 1\n",
    "        else:\n",
    "            pass\n",
    "\n",
    "        if self.variant[\"MetaLR_pred\"] == \"T\":\n",
    "            benign_list += 1\n",
    "        elif self.variant[\"MetaLR_pred\"] == \"D\":\n",
    "            pathogen_list += 1\n",
    "        else:\n",
    "            pass\n",
    "\n",
    "        if pathogen_list > benign_list:\n",
    "            self.OP1 = True\n",
    "            self.SBP1 = False\n",
    "            self.SBP2 = False\n",
    "        elif benign_list > pathogen_list:\n",
    "            self.OP1 = False\n",
    "            self.SBP1 = True\n",
    "            self.SBP2 = False\n",
    "        else:\n",
    "            if self.variant[\"ExonicFunc.refGene\"] == \"synonymous SNV\":\n",
    "                self.OP1 = False\n",
    "                self.SBP1 = False\n",
    "                self.SBP2 = True\n",
    "            else:\n",
    "                self.OP1 = False\n",
    "                self.SBP1 = False\n",
    "                self.SBP2 = False\n",
    "\n",
    "    def cancer_hotspot_rules(self):\n",
    "\n",
    "\n",
    "        if int(self.variant[\"total_variant_count\"]) >= 10:\n",
    "            if int(self.variant[\"Mutation_Count\"]) >= 50:\n",
    "                self.OS3 = True\n",
    "                self.OM3 = False\n",
    "                self.OP3 = False\n",
    "            else:\n",
    "                self.OS3 = False\n",
    "                self.OM3 = True\n",
    "                self.OP3 = False\n",
    "        elif int(self.variant[\"total_variant_count\"]) > 0:\n",
    "                self.OS3 = False\n",
    "                self.OM3 = False\n",
    "                self.OP3 = True\n",
    "        else:\n",
    "            self.OS3 = False\n",
    "            self.OM3 = False\n",
    "            self.OP3 = False\n",
    "        \n",
    "    def print_pop_rules(self):\n",
    "        print(f\"SBVS1:{self.SBVS1}\")\n",
    "        print(f\"OP4:{self.OP4}\")\n",
    "        print(f\"SBS1:{self.SBS1}\")\n",
    "        print(f\"OP1:{self.OP1}\")\n",
    "        print(f\"SBP1:{self.SBP1}\")\n",
    "        print(f\"SBP2:{self.SBP2}\")\n",
    "        print(f\"OS3:{self.OS3}\")\n",
    "        print(f\"OM3:{self.OM3}\")\n",
    "        print(f\"OP3:{self.OP3}\")\n",
    "\n",
    "cancer_hotspots_df = pd.read_csv(\"/Users/hatice.irmak/Desktop/somatic_rules/cancer_hotspots_revised.csv\")\n",
    "\n",
    "sample_df_ = prepare_data(sample_df, cancer_hotspots_df)\n",
    "rule_object = Rules(sample_df_.iloc[18])\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SBVS1:False\n",
      "OP4:True\n",
      "SBS1:False\n",
      "OP1:True\n",
      "SBP1:False\n",
      "SBP2:False\n",
      "OS3:True\n",
      "OM3:False\n",
      "OP3:False\n"
     ]
    }
   ],
   "source": [
    "rule_object.print_pop_rules()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SBVS1:False\n",
      "OP4:True\n",
      "SBS1:False\n",
      "OP1:True\n",
      "SBP1:False\n",
      "SBP2:False\n"
     ]
    }
   ],
   "source": [
    "rule_object.print_pop_rules()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "sample_df.to_clipboard()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [],
   "source": [
    "in_silico_columns = [\"FATHMM_score\", \"FATHMM_converted_rankscore\", \"FATHMM_pred\", \"CADD_raw\", \"CADD_raw_rankscore\", \"CADD_phred\", \"DANN_score\", \"DANN_rankscore\",\"REVEL\", \"MetaLR_score\", \"MetaLR_rankscore\", \"MetaLR_pred\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [],
   "source": [
    "def convert_float(value):\n",
    "    try:\n",
    "        return float(value)\n",
    "    except:\n",
    "        return False\n",
    "\n",
    "def in_silico_rules(row):\n",
    "    pathogen_list = 0\n",
    "    benign_list = 0\n",
    "    \n",
    "    print(row)\n",
    "    if row[\"FATHMM_pred\"] == \"T\":\n",
    "        benign_list += 1\n",
    "    elif row[\"FATHMM_pred\"] == \"D\":\n",
    "        pathogen_list += 1\n",
    "    else:\n",
    "        pass\n",
    "    \n",
    "    if convert_float(row[\"CADD_phred\"]) <= 23.2:\n",
    "        benign_list += 1\n",
    "    elif convert_float(row[\"CADD_phred\"]) >= 25.6:\n",
    "        pathogen_list += 1\n",
    "    else:\n",
    "        pass\n",
    "\n",
    "    if convert_float(row[\"DANN_score\"]) < 0.974:\n",
    "        benign_list += 1\n",
    "    elif convert_float(row[\"DANN_score\"]) >= 0.995:\n",
    "        pathogen_list +=1\n",
    "    else:\n",
    "        pass\n",
    "\n",
    "    if convert_float(row[\"REVEL\"]) <= 0.25:\n",
    "        benign_list += 1\n",
    "    elif convert_float(row[\"REVEL\"]) >= 0.75:\n",
    "        pathogen_list += 1\n",
    "    else:\n",
    "        pass\n",
    "\n",
    "    if row[\"MetaLR_pred\"] == \"T\":\n",
    "        benign_list += 1\n",
    "    elif row[\"MetaLR_pred\"] == \"D\":\n",
    "        pathogen_list += 1\n",
    "    else:\n",
    "        pass\n",
    "\n",
    "    \n",
    "    if pathogen_list > benign_list:\n",
    "        return \"OP1\"\n",
    "    elif benign_list > pathogen_list:\n",
    "        return \"SBP1\"\n",
    "    else:\n",
    "        if row[\"ExonicFunc.refGene\"] == \"synonymous SNV\":\n",
    "            return \"SBP2\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unnamed: 0                                                    0\n",
      "Chr                                                          12\n",
      "Start                                                  25209871\n",
      "End                                                    25209871\n",
      "Ref                                                           C\n",
      "                                    ...                        \n",
      "CLNDISDB      MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...\n",
      "CLNREVSTAT    criteria_provided\\x2c_multiple_submitters\\x2c_...\n",
      "CLNSIG                                   Uncertain_significance\n",
      "REVEL                                                     0.745\n",
      "rules                                                       OP4\n",
      "Name: 0, Length: 115, dtype: object\n",
      "Unnamed: 0                                                    1\n",
      "Chr                                                          12\n",
      "Start                                                  25227330\n",
      "End                                                    25227330\n",
      "Ref                                                           C\n",
      "                                    ...                        \n",
      "CLNDISDB      MONDO:MONDO:0012371\\x2cMedGen:C1860991\\x2cOMIM...\n",
      "CLNREVSTAT                             reviewed_by_expert_panel\n",
      "CLNSIG                                        Likely_pathogenic\n",
      "REVEL                                                     0.600\n",
      "rules                                                       OP4\n",
      "Name: 1, Length: 115, dtype: object\n",
      "Unnamed: 0                                                    2\n",
      "Chr                                                          12\n",
      "Start                                                  25227343\n",
      "End                                                    25227343\n",
      "Ref                                                           G\n",
      "                                    ...                        \n",
      "CLNDISDB      Human_Phenotype_Ontology:HP:0030358\\x2cMONDO:M...\n",
      "CLNREVSTAT               criteria_provided\\x2c_single_submitter\n",
      "CLNSIG                                               Pathogenic\n",
      "REVEL                                                     0.630\n",
      "rules                                                       OP4\n",
      "Name: 2, Length: 115, dtype: object\n",
      "Unnamed: 0                                                    3\n",
      "Chr                                                          12\n",
      "Start                                                  25245350\n",
      "End                                                    25245350\n",
      "Ref                                                           C\n",
      "                                    ...                        \n",
      "CLNDISDB      Human_Phenotype_Ontology:HP:0006775\\x2cMONDO:M...\n",
      "CLNREVSTAT                       no_assertion_criteria_provided\n",
      "CLNSIG             Conflicting_interpretations_of_pathogenicity\n",
      "REVEL                                                     0.841\n",
      "rules                                                       OP4\n",
      "Name: 3, Length: 115, dtype: object\n",
      "Unnamed: 0                                                    4\n",
      "Chr                                                          13\n",
      "Start                                                  28018483\n",
      "End                                                    28018483\n",
      "Ref                                                           T\n",
      "                                    ...                        \n",
      "CLNDISDB      Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...\n",
      "CLNREVSTAT                       no_assertion_criteria_provided\n",
      "CLNSIG                                        Likely_pathogenic\n",
      "REVEL                                                     0.925\n",
      "rules                                                       OP4\n",
      "Name: 4, Length: 115, dtype: object\n",
      "Unnamed: 0                                                    5\n",
      "Chr                                                          13\n",
      "Start                                                  28018503\n",
      "End                                                    28018503\n",
      "Ref                                                           A\n",
      "                                    ...                        \n",
      "CLNDISDB      Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...\n",
      "CLNREVSTAT                       no_assertion_criteria_provided\n",
      "CLNSIG                                               Pathogenic\n",
      "REVEL                                                     0.629\n",
      "rules                                                       OP4\n",
      "Name: 5, Length: 115, dtype: object\n",
      "Unnamed: 0           6\n",
      "Chr                 13\n",
      "Start         28018504\n",
      "End           28018505\n",
      "Ref                 TC\n",
      "                ...   \n",
      "CLNDISDB             .\n",
      "CLNREVSTAT           .\n",
      "CLNSIG               .\n",
      "REVEL                .\n",
      "rules              OP4\n",
      "Name: 6, Length: 115, dtype: object\n",
      "Unnamed: 0                                                    7\n",
      "Chr                                                          13\n",
      "Start                                                  28018504\n",
      "End                                                    28018504\n",
      "Ref                                                           T\n",
      "                                    ...                        \n",
      "CLNDISDB      Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...\n",
      "CLNREVSTAT                       no_assertion_criteria_provided\n",
      "CLNSIG                                               Pathogenic\n",
      "REVEL                                                     0.894\n",
      "rules                                                       OP4\n",
      "Name: 7, Length: 115, dtype: object\n",
      "Unnamed: 0                                                    8\n",
      "Chr                                                          13\n",
      "Start                                                  28018505\n",
      "End                                                    28018505\n",
      "Ref                                                           C\n",
      "                                    ...                        \n",
      "CLNDISDB      Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...\n",
      "CLNREVSTAT                       no_assertion_criteria_provided\n",
      "CLNSIG                                               Pathogenic\n",
      "REVEL                                                     0.878\n",
      "rules                                                       OP4\n",
      "Name: 8, Length: 115, dtype: object\n",
      "Unnamed: 0                                                    9\n",
      "Chr                                                          13\n",
      "Start                                                  28018505\n",
      "End                                                    28018505\n",
      "Ref                                                           C\n",
      "                                    ...                        \n",
      "CLNDISDB      Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...\n",
      "CLNREVSTAT                       no_assertion_criteria_provided\n",
      "CLNSIG                                               Pathogenic\n",
      "REVEL                                                     0.875\n",
      "rules                                                       OP4\n",
      "Name: 9, Length: 115, dtype: object\n",
      "Unnamed: 0          10\n",
      "Chr                 13\n",
      "Start         28027217\n",
      "End           28027217\n",
      "Ref                  T\n",
      "                ...   \n",
      "CLNDISDB             .\n",
      "CLNREVSTAT           .\n",
      "CLNSIG               .\n",
      "REVEL            0.883\n",
      "rules              OP4\n",
      "Name: 10, Length: 115, dtype: object\n",
      "Unnamed: 0          11\n",
      "Chr                 13\n",
      "Start         28027218\n",
      "End           28027218\n",
      "Ref                  A\n",
      "                ...   \n",
      "CLNDISDB             .\n",
      "CLNREVSTAT           .\n",
      "CLNSIG               .\n",
      "REVEL            0.917\n",
      "rules              OP4\n",
      "Name: 11, Length: 115, dtype: object\n",
      "Unnamed: 0                                                   12\n",
      "Chr                                                          13\n",
      "Start                                                  28027222\n",
      "End                                                    28027222\n",
      "Ref                                                           A\n",
      "                                    ...                        \n",
      "CLNDISDB      Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...\n",
      "CLNREVSTAT                       no_assertion_criteria_provided\n",
      "CLNSIG                                        Likely_pathogenic\n",
      "REVEL                                                     0.632\n",
      "rules                                                       OP4\n",
      "Name: 12, Length: 115, dtype: object\n",
      "Unnamed: 0                                                   13\n",
      "Chr                                                          13\n",
      "Start                                                  28028203\n",
      "End                                                    28028203\n",
      "Ref                                                           G\n",
      "                                    ...                        \n",
      "CLNDISDB      Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...\n",
      "CLNREVSTAT                       no_assertion_criteria_provided\n",
      "CLNSIG                                        Likely_pathogenic\n",
      "REVEL                                                     0.465\n",
      "rules                                                       OP4\n",
      "Name: 13, Length: 115, dtype: object\n",
      "Unnamed: 0                                                   14\n",
      "Chr                                                           2\n",
      "Start                                                 208242060\n",
      "End                                                   208242060\n",
      "Ref                                                           C\n",
      "                                    ...                        \n",
      "CLNDISDB      Human_Phenotype_Ontology:HP:0002665\\x2cMONDO:M...\n",
      "CLNREVSTAT                       no_assertion_criteria_provided\n",
      "CLNSIG                                   Uncertain_significance\n",
      "REVEL                                                     0.264\n",
      "rules                                                       OP4\n",
      "Name: 14, Length: 115, dtype: object\n",
      "Unnamed: 0                       15\n",
      "Chr                               2\n",
      "Start                     208243466\n",
      "End                       208243466\n",
      "Ref                               T\n",
      "                      ...          \n",
      "CLNDISDB            MedGen:CN169374\n",
      "CLNREVSTAT    no_assertion_provided\n",
      "CLNSIG                 not_provided\n",
      "REVEL                         0.908\n",
      "rules                           OP4\n",
      "Name: 15, Length: 115, dtype: object\n",
      "Unnamed: 0                                        16\n",
      "Chr                                                2\n",
      "Start                                      208243593\n",
      "End                                        208243593\n",
      "Ref                                                C\n",
      "                               ...                  \n",
      "CLNDISDB                             MedGen:CN169374\n",
      "CLNREVSTAT    criteria_provided\\x2c_single_submitter\n",
      "CLNSIG                                        Benign\n",
      "REVEL                                          0.421\n",
      "rules                                          SBVS1\n",
      "Name: 16, Length: 115, dtype: object\n",
      "Unnamed: 0                                                   17\n",
      "Chr                                                           2\n",
      "Start                                                 208248388\n",
      "End                                                   208248388\n",
      "Ref                                                           C\n",
      "                                    ...                        \n",
      "CLNDISDB      Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...\n",
      "CLNREVSTAT    criteria_provided\\x2c_multiple_submitters\\x2c_...\n",
      "CLNSIG                                               Pathogenic\n",
      "REVEL                                                     0.852\n",
      "rules                                                       OP4\n",
      "Name: 17, Length: 115, dtype: object\n",
      "Unnamed: 0                                                   18\n",
      "Chr                                                           2\n",
      "Start                                                 208248389\n",
      "End                                                   208248389\n",
      "Ref                                                           G\n",
      "                                    ...                        \n",
      "CLNDISDB      Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...\n",
      "CLNREVSTAT                       no_assertion_criteria_provided\n",
      "CLNSIG                             Pathogenic/Likely_pathogenic\n",
      "REVEL                                                     0.904\n",
      "rules                                                       OP4\n",
      "Name: 18, Length: 115, dtype: object\n",
      "Unnamed: 0                                                   19\n",
      "Chr                                                           2\n",
      "Start                                                 208248389\n",
      "End                                                   208248389\n",
      "Ref                                                           G\n",
      "                                    ...                        \n",
      "CLNDISDB      Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...\n",
      "CLNREVSTAT    criteria_provided\\x2c_multiple_submitters\\x2c_...\n",
      "CLNSIG                             Pathogenic/Likely_pathogenic\n",
      "REVEL                                                     0.814\n",
      "rules                                                       OP4\n",
      "Name: 19, Length: 115, dtype: object\n",
      "Unnamed: 0                                        20\n",
      "Chr                                                2\n",
      "Start                                      208248572\n",
      "End                                        208248572\n",
      "Ref                                                C\n",
      "                               ...                  \n",
      "CLNDISDB             MedGen:CN169374|MedGen:CN517202\n",
      "CLNREVSTAT    criteria_provided\\x2c_single_submitter\n",
      "CLNSIG                                        Benign\n",
      "REVEL                                          0.103\n",
      "rules                                           SBS1\n",
      "Name: 20, Length: 115, dtype: object\n",
      "Unnamed: 0                                        21\n",
      "Chr                                                2\n",
      "Start                                      208248664\n",
      "End                                        208248664\n",
      "Ref                                                G\n",
      "                               ...                  \n",
      "CLNDISDB                             MedGen:CN517202\n",
      "CLNREVSTAT    criteria_provided\\x2c_single_submitter\n",
      "CLNSIG                                        Benign\n",
      "REVEL                                              .\n",
      "rules                                           SBS1\n",
      "Name: 21, Length: 115, dtype: object\n",
      "Unnamed: 0                                        22\n",
      "Chr                                                2\n",
      "Start                                      208251458\n",
      "End                                        208251458\n",
      "Ref                                                A\n",
      "                               ...                  \n",
      "CLNDISDB             MedGen:CN169374|MedGen:CN517202\n",
      "CLNREVSTAT    criteria_provided\\x2c_single_submitter\n",
      "CLNSIG                                 Likely_benign\n",
      "REVEL                                          0.311\n",
      "rules                                            OP4\n",
      "Name: 22, Length: 115, dtype: object\n",
      "Unnamed: 0                                                   23\n",
      "Chr                                                           3\n",
      "Start                                                 179198826\n",
      "End                                                   179198826\n",
      "Ref                                                           A\n",
      "                                    ...                        \n",
      "CLNDISDB      Human_Phenotype_Ontology:HP:0010816\\x2cMONDO:M...\n",
      "CLNREVSTAT    criteria_provided\\x2c_multiple_submitters\\x2c_...\n",
      "CLNSIG                                   Uncertain_significance\n",
      "REVEL                                                     0.700\n",
      "rules                                                       OP4\n",
      "Name: 23, Length: 115, dtype: object\n",
      "Unnamed: 0                                        24\n",
      "Chr                                                3\n",
      "Start                                      179209622\n",
      "End                                        179209622\n",
      "Ref                                                A\n",
      "                               ...                  \n",
      "CLNDISDB                             MedGen:CN169374\n",
      "CLNREVSTAT    criteria_provided\\x2c_single_submitter\n",
      "CLNSIG                                        Benign\n",
      "REVEL                                          0.236\n",
      "rules                                          SBVS1\n",
      "Name: 24, Length: 115, dtype: object\n",
      "Unnamed: 0                                                   25\n",
      "Chr                                                           3\n",
      "Start                                                 179210186\n",
      "End                                                   179210186\n",
      "Ref                                                           G\n",
      "                                    ...                        \n",
      "CLNDISDB      MONDO:MONDO:0013038\\x2cMedGen:C2752042\\x2cOMIM...\n",
      "CLNREVSTAT               criteria_provided\\x2c_single_submitter\n",
      "CLNSIG                                   Uncertain_significance\n",
      "REVEL                                                     0.519\n",
      "rules                                                       OP4\n",
      "Name: 25, Length: 115, dtype: object\n",
      "Unnamed: 0                                                   26\n",
      "Chr                                                           3\n",
      "Start                                                 179210291\n",
      "End                                                   179210291\n",
      "Ref                                                           G\n",
      "                                    ...                        \n",
      "CLNDISDB      Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...\n",
      "CLNREVSTAT    criteria_provided\\x2c_multiple_submitters\\x2c_...\n",
      "CLNSIG                                               Pathogenic\n",
      "REVEL                                                     0.544\n",
      "rules                                                       OP4\n",
      "Name: 26, Length: 115, dtype: object\n",
      "Unnamed: 0           27\n",
      "Chr                   3\n",
      "Start         179218304\n",
      "End           179218304\n",
      "Ref                   A\n",
      "                ...    \n",
      "CLNDISDB              .\n",
      "CLNREVSTAT            .\n",
      "CLNSIG                .\n",
      "REVEL             0.715\n",
      "rules               OP4\n",
      "Name: 27, Length: 115, dtype: object\n",
      "Unnamed: 0                                                   28\n",
      "Chr                                                           3\n",
      "Start                                                 179224091\n",
      "End                                                   179224091\n",
      "Ref                                                           A\n",
      "                                    ...                        \n",
      "CLNDISDB      MONDO:MONDO:0016063\\x2cMedGen:C0018553\\x2cOMIM...\n",
      "CLNREVSTAT               criteria_provided\\x2c_single_submitter\n",
      "CLNSIG                                                   Benign\n",
      "REVEL                                                     0.269\n",
      "rules                                                       OP4\n",
      "Name: 28, Length: 115, dtype: object\n",
      "Unnamed: 0                                                   29\n",
      "Chr                                                           3\n",
      "Start                                                 179234297\n",
      "End                                                   179234297\n",
      "Ref                                                           A\n",
      "                                    ...                        \n",
      "CLNDISDB      .|Human_Phenotype_Ontology:HP:0001402\\x2cHuman...\n",
      "CLNREVSTAT    criteria_provided\\x2c_multiple_submitters\\x2c_...\n",
      "CLNSIG                                               Pathogenic\n",
      "REVEL                                                     0.455\n",
      "rules                                                       OP4\n",
      "Name: 29, Length: 115, dtype: object\n",
      "Unnamed: 0                                                   30\n",
      "Chr                                                           3\n",
      "Start                                                 179234298\n",
      "End                                                   179234298\n",
      "Ref                                                           T\n",
      "                                    ...                        \n",
      "CLNDISDB      Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...\n",
      "CLNREVSTAT                       no_assertion_criteria_provided\n",
      "CLNSIG                                        Likely_pathogenic\n",
      "REVEL                                                     0.428\n",
      "rules                                                       OP4\n",
      "Name: 30, Length: 115, dtype: object\n",
      "Unnamed: 0           31\n",
      "Chr                   3\n",
      "Start         179234350\n",
      "End           179234350\n",
      "Ref                   C\n",
      "                ...    \n",
      "CLNDISDB              .\n",
      "CLNREVSTAT            .\n",
      "CLNSIG                .\n",
      "REVEL             0.384\n",
      "rules               OP4\n",
      "Name: 31, Length: 115, dtype: object\n",
      "Unnamed: 0           32\n",
      "Chr                   3\n",
      "Start         179234360\n",
      "End           179234360\n",
      "Ref                   -\n",
      "                ...    \n",
      "CLNDISDB              .\n",
      "CLNREVSTAT            .\n",
      "CLNSIG                .\n",
      "REVEL                 .\n",
      "rules               OP4\n",
      "Name: 32, Length: 115, dtype: object\n",
      "Unnamed: 0                                                   33\n",
      "Chr                                                           5\n",
      "Start                                                   1282455\n",
      "End                                                     1282455\n",
      "Ref                                                           C\n",
      "                                    ...                        \n",
      "CLNDISDB      MONDO:MONDO:0008345\\x2cMedGen:C1800706\\x2cOMIM...\n",
      "CLNREVSTAT               criteria_provided\\x2c_single_submitter\n",
      "CLNSIG                                               Pathogenic\n",
      "REVEL                                                         .\n",
      "rules                                                       OP4\n",
      "Name: 33, Length: 115, dtype: object\n",
      "Unnamed: 0                                                   34\n",
      "Chr                                                           5\n",
      "Start                                                   1294198\n",
      "End                                                     1294198\n",
      "Ref                                                           G\n",
      "                                    ...                        \n",
      "CLNDISDB      MONDO:MONDO:0008345\\x2cMedGen:C1800706\\x2cOMIM...\n",
      "CLNREVSTAT               criteria_provided\\x2c_single_submitter\n",
      "CLNSIG                                               Pathogenic\n",
      "REVEL                                                         .\n",
      "rules                                                       OP4\n",
      "Name: 34, Length: 115, dtype: object\n",
      "Unnamed: 0                                                   35\n",
      "Chr                                                           7\n",
      "Start                                                 140753334\n",
      "End                                                   140753334\n",
      "Ref                                                           T\n",
      "                                    ...                        \n",
      "CLNDISDB      Human_Phenotype_Ontology:HP:0002664\\x2cHuman_P...\n",
      "CLNREVSTAT               criteria_provided\\x2c_single_submitter\n",
      "CLNSIG                                               Pathogenic\n",
      "REVEL                                                     0.916\n",
      "rules                                                       OP4\n",
      "Name: 35, Length: 115, dtype: object\n",
      "Unnamed: 0                                                   36\n",
      "Chr                                                           7\n",
      "Start                                                 140753336\n",
      "End                                                   140753337\n",
      "Ref                                                          AC\n",
      "                                    ...                        \n",
      "CLNDISDB      .|.|Human_Phenotype_Ontology:HP:0002861\\x2cHum...\n",
      "CLNREVSTAT               criteria_provided\\x2c_single_submitter\n",
      "CLNSIG                                            drug_response\n",
      "REVEL                                                         .\n",
      "rules                                                       OP4\n",
      "Name: 36, Length: 115, dtype: object\n",
      "Unnamed: 0                                                   37\n",
      "Chr                                                           7\n",
      "Start                                                 140753355\n",
      "End                                                   140753355\n",
      "Ref                                                           C\n",
      "                                    ...                        \n",
      "CLNDISDB      Human_Phenotype_Ontology:HP:0002861\\x2cHuman_P...\n",
      "CLNREVSTAT               criteria_provided\\x2c_single_submitter\n",
      "CLNSIG                                        Likely_pathogenic\n",
      "REVEL                                                     0.904\n",
      "rules                                                       OP4\n",
      "Name: 37, Length: 115, dtype: object\n",
      "Unnamed: 0                                38\n",
      "Chr                                        7\n",
      "Start                              140778037\n",
      "End                                140778051\n",
      "Ref                          TAGGTGCTGTCACAT\n",
      "                           ...              \n",
      "CLNDISDB                     MedGen:CN239586\n",
      "CLNREVSTAT    no_assertion_criteria_provided\n",
      "CLNSIG                         drug_response\n",
      "REVEL                                      .\n",
      "rules                                    OP4\n",
      "Name: 38, Length: 115, dtype: object\n",
      "Unnamed: 0                                                   39\n",
      "Chr                                                           7\n",
      "Start                                                 140781602\n",
      "End                                                   140781602\n",
      "Ref                                                           C\n",
      "                                    ...                        \n",
      "CLNDISDB      Human_Phenotype_Ontology:HP:0002861\\x2cHuman_P...\n",
      "CLNREVSTAT                             reviewed_by_expert_panel\n",
      "CLNSIG                                               Pathogenic\n",
      "REVEL                                                     0.921\n",
      "rules                                                       OP4\n",
      "Name: 39, Length: 115, dtype: object\n",
      "Unnamed: 0                                                   40\n",
      "Chr                                                           7\n",
      "Start                                                 140781602\n",
      "End                                                   140781602\n",
      "Ref                                                           C\n",
      "                                    ...                        \n",
      "CLNDISDB      Human_Phenotype_Ontology:HP:0002664\\x2cHuman_P...\n",
      "CLNREVSTAT               criteria_provided\\x2c_single_submitter\n",
      "CLNSIG                                               Pathogenic\n",
      "REVEL                                                     0.724\n",
      "rules                                                       OP4\n",
      "Name: 40, Length: 115, dtype: object\n",
      "Unnamed: 0           41\n",
      "Chr                   7\n",
      "Start         140781675\n",
      "End           140781675\n",
      "Ref                   C\n",
      "                ...    \n",
      "CLNDISDB              .\n",
      "CLNREVSTAT            .\n",
      "CLNSIG                .\n",
      "REVEL             0.627\n",
      "rules               OP4\n",
      "Name: 41, Length: 115, dtype: object\n",
      "Unnamed: 0           42\n",
      "Chr                   7\n",
      "Start         140783127\n",
      "End           140783127\n",
      "Ref                   G\n",
      "                ...    \n",
      "CLNDISDB              .\n",
      "CLNREVSTAT            .\n",
      "CLNSIG                .\n",
      "REVEL                 .\n",
      "rules               OP4\n",
      "Name: 42, Length: 115, dtype: object\n",
      "Unnamed: 0                                                   43\n",
      "Chr                                                           7\n",
      "Start                                                 140924612\n",
      "End                                                   140924612\n",
      "Ref                                                           G\n",
      "                                    ...                        \n",
      "CLNDISDB      MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...\n",
      "CLNREVSTAT                             reviewed_by_expert_panel\n",
      "CLNSIG                                            Likely_benign\n",
      "REVEL                                                     0.207\n",
      "rules                                                       OP4\n",
      "Name: 43, Length: 115, dtype: object\n",
      "Unnamed: 0                                                   44\n",
      "Chr                                                           7\n",
      "Start                                                 140924663\n",
      "End                                                   140924663\n",
      "Ref                                                           G\n",
      "                                    ...                        \n",
      "CLNDISDB      MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...\n",
      "CLNREVSTAT    criteria_provided\\x2c_multiple_submitters\\x2c_...\n",
      "CLNSIG                                   Uncertain_significance\n",
      "REVEL                                                     0.093\n",
      "rules                                                       OP4\n",
      "Name: 44, Length: 115, dtype: object\n"
     ]
    }
   ],
   "source": [
    "\n",
    "\n",
    "sample_df[\"rule\"] =  sample_df.apply(lambda x: in_silico_rules(x), axis=1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0</th>\n",
       "      <th>Chr</th>\n",
       "      <th>Start</th>\n",
       "      <th>End</th>\n",
       "      <th>Ref</th>\n",
       "      <th>Alt</th>\n",
       "      <th>Func.refGene</th>\n",
       "      <th>Gene.refGene</th>\n",
       "      <th>GeneDetail.refGene</th>\n",
       "      <th>ExonicFunc.refGene</th>\n",
       "      <th>...</th>\n",
       "      <th>dbscSNV_RF_SCORE</th>\n",
       "      <th>MCAP</th>\n",
       "      <th>CLNALLELEID</th>\n",
       "      <th>CLNDN</th>\n",
       "      <th>CLNDISDB</th>\n",
       "      <th>CLNREVSTAT</th>\n",
       "      <th>CLNSIG</th>\n",
       "      <th>REVEL</th>\n",
       "      <th>rules</th>\n",
       "      <th>rule</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>12</td>\n",
       "      <td>25209871</td>\n",
       "      <td>25209871</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.568996615134</td>\n",
       "      <td>54295</td>\n",
       "      <td>Rasopathy|not_specified</td>\n",
       "      <td>MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...</td>\n",
       "      <td>criteria_provided\\x2c_multiple_submitters\\x2c_...</td>\n",
       "      <td>Uncertain_significance</td>\n",
       "      <td>0.745</td>\n",
       "      <td>OP4</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>12</td>\n",
       "      <td>25227330</td>\n",
       "      <td>25227330</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.123105525756</td>\n",
       "      <td>432423</td>\n",
       "      <td>Noonan_syndrome_3|Rasopathy</td>\n",
       "      <td>MONDO:MONDO:0012371\\x2cMedGen:C1860991\\x2cOMIM...</td>\n",
       "      <td>reviewed_by_expert_panel</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>0.600</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>12</td>\n",
       "      <td>25227343</td>\n",
       "      <td>25227343</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0655137715661</td>\n",
       "      <td>175715</td>\n",
       "      <td>Non-small_cell_lung_cancer|Neoplasm_of_the_thy...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0030358\\x2cMONDO:M...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.630</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>12</td>\n",
       "      <td>25245350</td>\n",
       "      <td>25245350</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0910971912578</td>\n",
       "      <td>54289</td>\n",
       "      <td>Multiple_myeloma|Lung_adenocarcinoma|Non-small...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0006775\\x2cMONDO:M...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Conflicting_interpretations_of_pathogenicity</td>\n",
       "      <td>0.841</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>13</td>\n",
       "      <td>28018483</td>\n",
       "      <td>28018483</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.293007099215</td>\n",
       "      <td>362981</td>\n",
       "      <td>Acute_myeloid_leukemia</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>0.925</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows  116 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "   Unnamed: 0  Chr     Start       End Ref Alt Func.refGene Gene.refGene  \\\n",
       "0           0   12  25209871  25209871   C   T       exonic         KRAS   \n",
       "1           1   12  25227330  25227330   C   A       exonic         KRAS   \n",
       "2           2   12  25227343  25227343   G   T       exonic         KRAS   \n",
       "3           3   12  25245350  25245350   C   G       exonic         KRAS   \n",
       "4           4   13  28018483  28018483   T   C       exonic         FLT3   \n",
       "\n",
       "  GeneDetail.refGene ExonicFunc.refGene  ... dbscSNV_RF_SCORE  \\\n",
       "0                  .  nonsynonymous SNV  ...                .   \n",
       "1                  .  nonsynonymous SNV  ...                .   \n",
       "2                  .  nonsynonymous SNV  ...                .   \n",
       "3                  .  nonsynonymous SNV  ...                .   \n",
       "4                  .  nonsynonymous SNV  ...                .   \n",
       "\n",
       "              MCAP CLNALLELEID  \\\n",
       "0   0.568996615134       54295   \n",
       "1   0.123105525756      432423   \n",
       "2  0.0655137715661      175715   \n",
       "3  0.0910971912578       54289   \n",
       "4   0.293007099215      362981   \n",
       "\n",
       "                                               CLNDN  \\\n",
       "0                            Rasopathy|not_specified   \n",
       "1                        Noonan_syndrome_3|Rasopathy   \n",
       "2  Non-small_cell_lung_cancer|Neoplasm_of_the_thy...   \n",
       "3  Multiple_myeloma|Lung_adenocarcinoma|Non-small...   \n",
       "4                             Acute_myeloid_leukemia   \n",
       "\n",
       "                                            CLNDISDB  \\\n",
       "0  MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...   \n",
       "1  MONDO:MONDO:0012371\\x2cMedGen:C1860991\\x2cOMIM...   \n",
       "2  Human_Phenotype_Ontology:HP:0030358\\x2cMONDO:M...   \n",
       "3  Human_Phenotype_Ontology:HP:0006775\\x2cMONDO:M...   \n",
       "4  Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...   \n",
       "\n",
       "                                          CLNREVSTAT  \\\n",
       "0  criteria_provided\\x2c_multiple_submitters\\x2c_...   \n",
       "1                           reviewed_by_expert_panel   \n",
       "2             criteria_provided\\x2c_single_submitter   \n",
       "3                     no_assertion_criteria_provided   \n",
       "4                     no_assertion_criteria_provided   \n",
       "\n",
       "                                         CLNSIG  REVEL rules  rule  \n",
       "0                        Uncertain_significance  0.745   OP4  None  \n",
       "1                             Likely_pathogenic  0.600   OP4   OP1  \n",
       "2                                    Pathogenic  0.630   OP4   OP1  \n",
       "3  Conflicting_interpretations_of_pathogenicity  0.841   OP4   OP1  \n",
       "4                             Likely_pathogenic  0.925   OP4   OP1  \n",
       "\n",
       "[5 rows x 116 columns]"
      ]
     },
     "execution_count": 35,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sample_df.head(5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index(['Unnamed: 0', 'Chr', 'Start', 'End', 'Ref', 'Alt', 'Func.refGene',\n",
      "       'Gene.refGene', 'GeneDetail.refGene', 'ExonicFunc.refGene',\n",
      "       ...\n",
      "       'dbscSNV_RF_SCORE', 'MCAP', 'CLNALLELEID', 'CLNDN', 'CLNDISDB',\n",
      "       'CLNREVSTAT', 'CLNSIG', 'REVEL', 'rules', 'rule'],\n",
      "      dtype='object', length=116)\n"
     ]
    }
   ],
   "source": [
    "print(sample_df.columns)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0</th>\n",
       "      <th>Chr</th>\n",
       "      <th>Start</th>\n",
       "      <th>End</th>\n",
       "      <th>Ref</th>\n",
       "      <th>Alt</th>\n",
       "      <th>Func.refGene</th>\n",
       "      <th>Gene.refGene</th>\n",
       "      <th>GeneDetail.refGene</th>\n",
       "      <th>ExonicFunc.refGene</th>\n",
       "      <th>...</th>\n",
       "      <th>dbscSNV_RF_SCORE</th>\n",
       "      <th>MCAP</th>\n",
       "      <th>CLNALLELEID</th>\n",
       "      <th>CLNDN</th>\n",
       "      <th>CLNDISDB</th>\n",
       "      <th>CLNREVSTAT</th>\n",
       "      <th>CLNSIG</th>\n",
       "      <th>REVEL</th>\n",
       "      <th>rules</th>\n",
       "      <th>rule</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>12</td>\n",
       "      <td>25209871</td>\n",
       "      <td>25209871</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.568996615134</td>\n",
       "      <td>54295</td>\n",
       "      <td>Rasopathy|not_specified</td>\n",
       "      <td>MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...</td>\n",
       "      <td>criteria_provided\\x2c_multiple_submitters\\x2c_...</td>\n",
       "      <td>Uncertain_significance</td>\n",
       "      <td>0.745</td>\n",
       "      <td>OP4</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>12</td>\n",
       "      <td>25227330</td>\n",
       "      <td>25227330</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.123105525756</td>\n",
       "      <td>432423</td>\n",
       "      <td>Noonan_syndrome_3|Rasopathy</td>\n",
       "      <td>MONDO:MONDO:0012371\\x2cMedGen:C1860991\\x2cOMIM...</td>\n",
       "      <td>reviewed_by_expert_panel</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>0.600</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>12</td>\n",
       "      <td>25227343</td>\n",
       "      <td>25227343</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0655137715661</td>\n",
       "      <td>175715</td>\n",
       "      <td>Non-small_cell_lung_cancer|Neoplasm_of_the_thy...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0030358\\x2cMONDO:M...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.630</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>12</td>\n",
       "      <td>25245350</td>\n",
       "      <td>25245350</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0910971912578</td>\n",
       "      <td>54289</td>\n",
       "      <td>Multiple_myeloma|Lung_adenocarcinoma|Non-small...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0006775\\x2cMONDO:M...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Conflicting_interpretations_of_pathogenicity</td>\n",
       "      <td>0.841</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>13</td>\n",
       "      <td>28018483</td>\n",
       "      <td>28018483</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.293007099215</td>\n",
       "      <td>362981</td>\n",
       "      <td>Acute_myeloid_leukemia</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>0.925</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>5</td>\n",
       "      <td>13</td>\n",
       "      <td>28018503</td>\n",
       "      <td>28018503</td>\n",
       "      <td>A</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.225817333818</td>\n",
       "      <td>362851</td>\n",
       "      <td>Acute_myeloid_leukemia</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.629</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>6</td>\n",
       "      <td>13</td>\n",
       "      <td>28018504</td>\n",
       "      <td>28018505</td>\n",
       "      <td>TC</td>\n",
       "      <td>AT</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonframeshift substitution</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>7</td>\n",
       "      <td>13</td>\n",
       "      <td>28018504</td>\n",
       "      <td>28018504</td>\n",
       "      <td>T</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.264924784705</td>\n",
       "      <td>31311</td>\n",
       "      <td>Acute_myeloid_leukemia|Acute_lymphoid_leukemia</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.894</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>8</td>\n",
       "      <td>13</td>\n",
       "      <td>28018505</td>\n",
       "      <td>28018505</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0876328368379</td>\n",
       "      <td>31312</td>\n",
       "      <td>Acute_myeloid_leukemia</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.878</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>9</td>\n",
       "      <td>13</td>\n",
       "      <td>28018505</td>\n",
       "      <td>28018505</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0837176557721</td>\n",
       "      <td>31315</td>\n",
       "      <td>Acute_myeloid_leukemia|Acute_lymphoid_leukemia</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.875</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10</td>\n",
       "      <td>13</td>\n",
       "      <td>28027217</td>\n",
       "      <td>28027217</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.214409313169</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.883</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>11</td>\n",
       "      <td>13</td>\n",
       "      <td>28027218</td>\n",
       "      <td>28027218</td>\n",
       "      <td>A</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.297566568267</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.917</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>12</td>\n",
       "      <td>13</td>\n",
       "      <td>28027222</td>\n",
       "      <td>28027222</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0964035798672</td>\n",
       "      <td>362984</td>\n",
       "      <td>Acute_myeloid_leukemia</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>0.632</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>13</td>\n",
       "      <td>13</td>\n",
       "      <td>28028203</td>\n",
       "      <td>28028203</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.108688019891</td>\n",
       "      <td>362988</td>\n",
       "      <td>Acute_myeloid_leukemia</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>0.465</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>14</td>\n",
       "      <td>2</td>\n",
       "      <td>208242060</td>\n",
       "      <td>208242060</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0238719972834</td>\n",
       "      <td>626059</td>\n",
       "      <td>Lymphoma</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0002665\\x2cMONDO:M...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Uncertain_significance</td>\n",
       "      <td>0.264</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>15</td>\n",
       "      <td>2</td>\n",
       "      <td>208243466</td>\n",
       "      <td>208243466</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.417115451694</td>\n",
       "      <td>138258</td>\n",
       "      <td>not_specified</td>\n",
       "      <td>MedGen:CN169374</td>\n",
       "      <td>no_assertion_provided</td>\n",
       "      <td>not_provided</td>\n",
       "      <td>0.908</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>16</td>\n",
       "      <td>2</td>\n",
       "      <td>208243593</td>\n",
       "      <td>208243593</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>138259</td>\n",
       "      <td>not_specified</td>\n",
       "      <td>MedGen:CN169374</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Benign</td>\n",
       "      <td>0.421</td>\n",
       "      <td>SBVS1</td>\n",
       "      <td>SBP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>17</td>\n",
       "      <td>2</td>\n",
       "      <td>208248388</td>\n",
       "      <td>208248388</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.560930781166</td>\n",
       "      <td>166215</td>\n",
       "      <td>Hepatocellular_carcinoma|Acute_myeloid_leukemi...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...</td>\n",
       "      <td>criteria_provided\\x2c_multiple_submitters\\x2c_...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.852</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>18</td>\n",
       "      <td>2</td>\n",
       "      <td>208248389</td>\n",
       "      <td>208248389</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.612118206497</td>\n",
       "      <td>362772</td>\n",
       "      <td>Hepatocellular_carcinoma|Acute_myeloid_leukemi...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Pathogenic/Likely_pathogenic</td>\n",
       "      <td>0.904</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>19</td>\n",
       "      <td>2</td>\n",
       "      <td>208248389</td>\n",
       "      <td>208248389</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.530890426099</td>\n",
       "      <td>362770</td>\n",
       "      <td>Hepatocellular_carcinoma|Acute_myeloid_leukemi...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...</td>\n",
       "      <td>criteria_provided\\x2c_multiple_submitters\\x2c_...</td>\n",
       "      <td>Pathogenic/Likely_pathogenic</td>\n",
       "      <td>0.814</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>20</td>\n",
       "      <td>2</td>\n",
       "      <td>208248572</td>\n",
       "      <td>208248572</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>138255</td>\n",
       "      <td>not_specified|not_provided</td>\n",
       "      <td>MedGen:CN169374|MedGen:CN517202</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Benign</td>\n",
       "      <td>0.103</td>\n",
       "      <td>SBS1</td>\n",
       "      <td>SBP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>21</td>\n",
       "      <td>2</td>\n",
       "      <td>208248664</td>\n",
       "      <td>208248664</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>splicing</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>NM_001282387:exon4:c.123-4C&gt;T;NM_005896:exon4:...</td>\n",
       "      <td>.</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>.</td>\n",
       "      <td>778916</td>\n",
       "      <td>not_provided</td>\n",
       "      <td>MedGen:CN517202</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Benign</td>\n",
       "      <td>.</td>\n",
       "      <td>SBS1</td>\n",
       "      <td>SBP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>22</td>\n",
       "      <td>2</td>\n",
       "      <td>208251458</td>\n",
       "      <td>208251458</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0549463899991</td>\n",
       "      <td>138254</td>\n",
       "      <td>not_specified|not_provided</td>\n",
       "      <td>MedGen:CN169374|MedGen:CN517202</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Likely_benign</td>\n",
       "      <td>0.311</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>23</td>\n",
       "      <td>3</td>\n",
       "      <td>179198826</td>\n",
       "      <td>179198826</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>startloss</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.2576582387</td>\n",
       "      <td>827717</td>\n",
       "      <td>Epidermal_nevus|Megalencephaly-capillary_malfo...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0010816\\x2cMONDO:M...</td>\n",
       "      <td>criteria_provided\\x2c_multiple_submitters\\x2c_...</td>\n",
       "      <td>Uncertain_significance</td>\n",
       "      <td>0.700</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>24</td>\n",
       "      <td>3</td>\n",
       "      <td>179209622</td>\n",
       "      <td>179209622</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>138777</td>\n",
       "      <td>not_specified</td>\n",
       "      <td>MedGen:CN169374</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Benign</td>\n",
       "      <td>0.236</td>\n",
       "      <td>SBVS1</td>\n",
       "      <td>SBP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>25</td>\n",
       "      <td>3</td>\n",
       "      <td>179210186</td>\n",
       "      <td>179210186</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.914</td>\n",
       "      <td>0.134419997912</td>\n",
       "      <td>54631</td>\n",
       "      <td>Congenital_lipomatous_overgrowth\\x2c_vascular_...</td>\n",
       "      <td>MONDO:MONDO:0013038\\x2cMedGen:C2752042\\x2cOMIM...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Uncertain_significance</td>\n",
       "      <td>0.519</td>\n",
       "      <td>OP4</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>26</td>\n",
       "      <td>3</td>\n",
       "      <td>179210291</td>\n",
       "      <td>179210291</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.113788654374</td>\n",
       "      <td>363349</td>\n",
       "      <td>Hepatocellular_carcinoma|Abnormality_of_cardio...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...</td>\n",
       "      <td>criteria_provided\\x2c_multiple_submitters\\x2c_...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.544</td>\n",
       "      <td>OP4</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>27</td>\n",
       "      <td>3</td>\n",
       "      <td>179218304</td>\n",
       "      <td>179218304</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.088721466783</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.715</td>\n",
       "      <td>OP4</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>28</td>\n",
       "      <td>3</td>\n",
       "      <td>179224091</td>\n",
       "      <td>179224091</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0164815254628</td>\n",
       "      <td>452429</td>\n",
       "      <td>Cowden_syndrome</td>\n",
       "      <td>MONDO:MONDO:0016063\\x2cMedGen:C0018553\\x2cOMIM...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Benign</td>\n",
       "      <td>0.269</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>29</td>\n",
       "      <td>3</td>\n",
       "      <td>179234297</td>\n",
       "      <td>179234297</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0373781845074</td>\n",
       "      <td>28691</td>\n",
       "      <td>MACRODACTYLY\\x2c_SOMATIC|Hepatocellular_carcin...</td>\n",
       "      <td>.|Human_Phenotype_Ontology:HP:0001402\\x2cHuman...</td>\n",
       "      <td>criteria_provided\\x2c_multiple_submitters\\x2c_...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.455</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>30</td>\n",
       "      <td>3</td>\n",
       "      <td>179234298</td>\n",
       "      <td>179234298</td>\n",
       "      <td>T</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.026514517841</td>\n",
       "      <td>363360</td>\n",
       "      <td>Hepatocellular_carcinoma|Medulloblastoma|Adren...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>0.428</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>31</td>\n",
       "      <td>3</td>\n",
       "      <td>179234350</td>\n",
       "      <td>179234350</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0220153461798</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.384</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>32</td>\n",
       "      <td>3</td>\n",
       "      <td>179234360</td>\n",
       "      <td>179234360</td>\n",
       "      <td>-</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>frameshift insertion</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>33</td>\n",
       "      <td>5</td>\n",
       "      <td>1282455</td>\n",
       "      <td>1282455</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>TERT</td>\n",
       "      <td>.</td>\n",
       "      <td>stopgain</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>829942</td>\n",
       "      <td>Idiopathic_Pulmonary_Fibrosis|Dyskeratosis_con...</td>\n",
       "      <td>MONDO:MONDO:0008345\\x2cMedGen:C1800706\\x2cOMIM...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>.</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>34</td>\n",
       "      <td>5</td>\n",
       "      <td>1294198</td>\n",
       "      <td>1294198</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>TERT</td>\n",
       "      <td>.</td>\n",
       "      <td>stopgain</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>633029</td>\n",
       "      <td>Idiopathic_Pulmonary_Fibrosis|Dyskeratosis_con...</td>\n",
       "      <td>MONDO:MONDO:0008345\\x2cMedGen:C1800706\\x2cOMIM...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>.</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>35</td>\n",
       "      <td>7</td>\n",
       "      <td>140753334</td>\n",
       "      <td>140753334</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0820752694957</td>\n",
       "      <td>29005</td>\n",
       "      <td>Neoplasm|Melanoma|Chronic_lymphocytic_leukemia...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0002664\\x2cHuman_P...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.916</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>36</td>\n",
       "      <td>7</td>\n",
       "      <td>140753336</td>\n",
       "      <td>140753337</td>\n",
       "      <td>AC</td>\n",
       "      <td>TT</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonframeshift substitution</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>362820</td>\n",
       "      <td>Trametinib-Dabrafenib_Response|Vemurafenib-Cob...</td>\n",
       "      <td>.|.|Human_Phenotype_Ontology:HP:0002861\\x2cHum...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>drug_response</td>\n",
       "      <td>.</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>37</td>\n",
       "      <td>7</td>\n",
       "      <td>140753355</td>\n",
       "      <td>140753355</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.596855831475</td>\n",
       "      <td>53980</td>\n",
       "      <td>Melanoma|Adrenocortical_carcinoma|Chronic_lymp...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0002861\\x2cHuman_P...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>0.904</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>38</td>\n",
       "      <td>7</td>\n",
       "      <td>140778037</td>\n",
       "      <td>140778051</td>\n",
       "      <td>TAGGTGCTGTCACAT</td>\n",
       "      <td>-</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonframeshift deletion</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>653824</td>\n",
       "      <td>Dabrafenib_response</td>\n",
       "      <td>MedGen:CN239586</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>drug_response</td>\n",
       "      <td>.</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>39</td>\n",
       "      <td>7</td>\n",
       "      <td>140781602</td>\n",
       "      <td>140781602</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.612203233818</td>\n",
       "      <td>29013</td>\n",
       "      <td>Melanoma|Squamous_cell_carcinoma_of_the_skin|T...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0002861\\x2cHuman_P...</td>\n",
       "      <td>reviewed_by_expert_panel</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.921</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>40</td>\n",
       "      <td>7</td>\n",
       "      <td>140781602</td>\n",
       "      <td>140781602</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.219344139139</td>\n",
       "      <td>29010</td>\n",
       "      <td>Neoplasm|Squamous_cell_carcinoma_of_the_skin|T...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0002664\\x2cHuman_P...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.724</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>41</td>\n",
       "      <td>7</td>\n",
       "      <td>140781675</td>\n",
       "      <td>140781675</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.100260968362</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.627</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>42</td>\n",
       "      <td>7</td>\n",
       "      <td>140783127</td>\n",
       "      <td>140783127</td>\n",
       "      <td>G</td>\n",
       "      <td>-</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>frameshift deletion</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>43</td>\n",
       "      <td>7</td>\n",
       "      <td>140924612</td>\n",
       "      <td>140924612</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.261712773584</td>\n",
       "      <td>54000</td>\n",
       "      <td>Rasopathy|not_specified</td>\n",
       "      <td>MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...</td>\n",
       "      <td>reviewed_by_expert_panel</td>\n",
       "      <td>Likely_benign</td>\n",
       "      <td>0.207</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>44</td>\n",
       "      <td>7</td>\n",
       "      <td>140924663</td>\n",
       "      <td>140924663</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>0.0891133498287</td>\n",
       "      <td>48804</td>\n",
       "      <td>Rasopathy|not_provided</td>\n",
       "      <td>MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...</td>\n",
       "      <td>criteria_provided\\x2c_multiple_submitters\\x2c_...</td>\n",
       "      <td>Uncertain_significance</td>\n",
       "      <td>0.093</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>45 rows  116 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "    Unnamed: 0  Chr      Start        End              Ref Alt Func.refGene  \\\n",
       "0            0   12   25209871   25209871                C   T       exonic   \n",
       "1            1   12   25227330   25227330                C   A       exonic   \n",
       "2            2   12   25227343   25227343                G   T       exonic   \n",
       "3            3   12   25245350   25245350                C   G       exonic   \n",
       "4            4   13   28018483   28018483                T   C       exonic   \n",
       "5            5   13   28018503   28018503                A   T       exonic   \n",
       "6            6   13   28018504   28018505               TC  AT       exonic   \n",
       "7            7   13   28018504   28018504                T   A       exonic   \n",
       "8            8   13   28018505   28018505                C   G       exonic   \n",
       "9            9   13   28018505   28018505                C   A       exonic   \n",
       "10          10   13   28027217   28027217                T   C       exonic   \n",
       "11          11   13   28027218   28027218                A   T       exonic   \n",
       "12          12   13   28027222   28027222                A   C       exonic   \n",
       "13          13   13   28028203   28028203                G   T       exonic   \n",
       "14          14    2  208242060  208242060                C   T       exonic   \n",
       "15          15    2  208243466  208243466                T   C       exonic   \n",
       "16          16    2  208243593  208243593                C   T       exonic   \n",
       "17          17    2  208248388  208248388                C   T       exonic   \n",
       "18          18    2  208248389  208248389                G   T       exonic   \n",
       "19          19    2  208248389  208248389                G   A       exonic   \n",
       "20          20    2  208248572  208248572                C   T       exonic   \n",
       "21          21    2  208248664  208248664                G   A     splicing   \n",
       "22          22    2  208251458  208251458                A   C       exonic   \n",
       "23          23    3  179198826  179198826                A   G       exonic   \n",
       "24          24    3  179209622  179209622                A   G       exonic   \n",
       "25          25    3  179210186  179210186                G   A       exonic   \n",
       "26          26    3  179210291  179210291                G   A       exonic   \n",
       "27          27    3  179218304  179218304                A   C       exonic   \n",
       "28          28    3  179224091  179224091                A   G       exonic   \n",
       "29          29    3  179234297  179234297                A   G       exonic   \n",
       "30          30    3  179234298  179234298                T   G       exonic   \n",
       "31          31    3  179234350  179234350                C   T       exonic   \n",
       "32          32    3  179234360  179234360                -   A       exonic   \n",
       "33          33    5    1282455    1282455                C   T       exonic   \n",
       "34          34    5    1294198    1294198                G   A       exonic   \n",
       "35          35    7  140753334  140753334                T   C       exonic   \n",
       "36          36    7  140753336  140753337               AC  TT       exonic   \n",
       "37          37    7  140753355  140753355                C   T       exonic   \n",
       "38          38    7  140778037  140778051  TAGGTGCTGTCACAT   -       exonic   \n",
       "39          39    7  140781602  140781602                C   T       exonic   \n",
       "40          40    7  140781602  140781602                C   G       exonic   \n",
       "41          41    7  140781675  140781675                C   G       exonic   \n",
       "42          42    7  140783127  140783127                G   -       exonic   \n",
       "43          43    7  140924612  140924612                G   C       exonic   \n",
       "44          44    7  140924663  140924663                G   C       exonic   \n",
       "\n",
       "   Gene.refGene                                 GeneDetail.refGene  \\\n",
       "0          KRAS                                                  .   \n",
       "1          KRAS                                                  .   \n",
       "2          KRAS                                                  .   \n",
       "3          KRAS                                                  .   \n",
       "4          FLT3                                                  .   \n",
       "5          FLT3                                                  .   \n",
       "6          FLT3                                                  .   \n",
       "7          FLT3                                                  .   \n",
       "8          FLT3                                                  .   \n",
       "9          FLT3                                                  .   \n",
       "10         FLT3                                                  .   \n",
       "11         FLT3                                                  .   \n",
       "12         FLT3                                                  .   \n",
       "13         FLT3                                                  .   \n",
       "14         IDH1                                                  .   \n",
       "15         IDH1                                                  .   \n",
       "16         IDH1                                                  .   \n",
       "17         IDH1                                                  .   \n",
       "18         IDH1                                                  .   \n",
       "19         IDH1                                                  .   \n",
       "20         IDH1                                                  .   \n",
       "21         IDH1  NM_001282387:exon4:c.123-4C>T;NM_005896:exon4:...   \n",
       "22         IDH1                                                  .   \n",
       "23       PIK3CA                                                  .   \n",
       "24       PIK3CA                                                  .   \n",
       "25       PIK3CA                                                  .   \n",
       "26       PIK3CA                                                  .   \n",
       "27       PIK3CA                                                  .   \n",
       "28       PIK3CA                                                  .   \n",
       "29       PIK3CA                                                  .   \n",
       "30       PIK3CA                                                  .   \n",
       "31       PIK3CA                                                  .   \n",
       "32       PIK3CA                                                  .   \n",
       "33         TERT                                                  .   \n",
       "34         TERT                                                  .   \n",
       "35         BRAF                                                  .   \n",
       "36         BRAF                                                  .   \n",
       "37         BRAF                                                  .   \n",
       "38         BRAF                                                  .   \n",
       "39         BRAF                                                  .   \n",
       "40         BRAF                                                  .   \n",
       "41         BRAF                                                  .   \n",
       "42         BRAF                                                  .   \n",
       "43         BRAF                                                  .   \n",
       "44         BRAF                                                  .   \n",
       "\n",
       "            ExonicFunc.refGene  ... dbscSNV_RF_SCORE             MCAP  \\\n",
       "0            nonsynonymous SNV  ...                .   0.568996615134   \n",
       "1            nonsynonymous SNV  ...                .   0.123105525756   \n",
       "2            nonsynonymous SNV  ...                .  0.0655137715661   \n",
       "3            nonsynonymous SNV  ...                .  0.0910971912578   \n",
       "4            nonsynonymous SNV  ...                .   0.293007099215   \n",
       "5            nonsynonymous SNV  ...                .   0.225817333818   \n",
       "6   nonframeshift substitution  ...                .                .   \n",
       "7            nonsynonymous SNV  ...                .   0.264924784705   \n",
       "8            nonsynonymous SNV  ...                .  0.0876328368379   \n",
       "9            nonsynonymous SNV  ...                .  0.0837176557721   \n",
       "10           nonsynonymous SNV  ...                .   0.214409313169   \n",
       "11           nonsynonymous SNV  ...                .   0.297566568267   \n",
       "12           nonsynonymous SNV  ...                .  0.0964035798672   \n",
       "13           nonsynonymous SNV  ...                .   0.108688019891   \n",
       "14           nonsynonymous SNV  ...                .  0.0238719972834   \n",
       "15           nonsynonymous SNV  ...                .   0.417115451694   \n",
       "16           nonsynonymous SNV  ...                .                .   \n",
       "17           nonsynonymous SNV  ...                .   0.560930781166   \n",
       "18           nonsynonymous SNV  ...                .   0.612118206497   \n",
       "19           nonsynonymous SNV  ...                .   0.530890426099   \n",
       "20           nonsynonymous SNV  ...                .                .   \n",
       "21                           .  ...                0                .   \n",
       "22           nonsynonymous SNV  ...                .  0.0549463899991   \n",
       "23                   startloss  ...                .     0.2576582387   \n",
       "24           nonsynonymous SNV  ...                .                .   \n",
       "25           nonsynonymous SNV  ...            0.914   0.134419997912   \n",
       "26           nonsynonymous SNV  ...                .   0.113788654374   \n",
       "27           nonsynonymous SNV  ...                .   0.088721466783   \n",
       "28           nonsynonymous SNV  ...                .  0.0164815254628   \n",
       "29           nonsynonymous SNV  ...                .  0.0373781845074   \n",
       "30           nonsynonymous SNV  ...                .   0.026514517841   \n",
       "31           nonsynonymous SNV  ...                .  0.0220153461798   \n",
       "32        frameshift insertion  ...                .                .   \n",
       "33                    stopgain  ...                .                .   \n",
       "34                    stopgain  ...                .                .   \n",
       "35           nonsynonymous SNV  ...                .  0.0820752694957   \n",
       "36  nonframeshift substitution  ...                .                .   \n",
       "37           nonsynonymous SNV  ...                .   0.596855831475   \n",
       "38      nonframeshift deletion  ...                .                .   \n",
       "39           nonsynonymous SNV  ...                .   0.612203233818   \n",
       "40           nonsynonymous SNV  ...                .   0.219344139139   \n",
       "41           nonsynonymous SNV  ...                .   0.100260968362   \n",
       "42         frameshift deletion  ...                .                .   \n",
       "43           nonsynonymous SNV  ...                .   0.261712773584   \n",
       "44           nonsynonymous SNV  ...                .  0.0891133498287   \n",
       "\n",
       "   CLNALLELEID                                              CLNDN  \\\n",
       "0        54295                            Rasopathy|not_specified   \n",
       "1       432423                        Noonan_syndrome_3|Rasopathy   \n",
       "2       175715  Non-small_cell_lung_cancer|Neoplasm_of_the_thy...   \n",
       "3        54289  Multiple_myeloma|Lung_adenocarcinoma|Non-small...   \n",
       "4       362981                             Acute_myeloid_leukemia   \n",
       "5       362851                             Acute_myeloid_leukemia   \n",
       "6            .                                                  .   \n",
       "7        31311     Acute_myeloid_leukemia|Acute_lymphoid_leukemia   \n",
       "8        31312                             Acute_myeloid_leukemia   \n",
       "9        31315     Acute_myeloid_leukemia|Acute_lymphoid_leukemia   \n",
       "10           .                                                  .   \n",
       "11           .                                                  .   \n",
       "12      362984                             Acute_myeloid_leukemia   \n",
       "13      362988                             Acute_myeloid_leukemia   \n",
       "14      626059                                           Lymphoma   \n",
       "15      138258                                      not_specified   \n",
       "16      138259                                      not_specified   \n",
       "17      166215  Hepatocellular_carcinoma|Acute_myeloid_leukemi...   \n",
       "18      362772  Hepatocellular_carcinoma|Acute_myeloid_leukemi...   \n",
       "19      362770  Hepatocellular_carcinoma|Acute_myeloid_leukemi...   \n",
       "20      138255                         not_specified|not_provided   \n",
       "21      778916                                       not_provided   \n",
       "22      138254                         not_specified|not_provided   \n",
       "23      827717  Epidermal_nevus|Megalencephaly-capillary_malfo...   \n",
       "24      138777                                      not_specified   \n",
       "25       54631  Congenital_lipomatous_overgrowth\\x2c_vascular_...   \n",
       "26      363349  Hepatocellular_carcinoma|Abnormality_of_cardio...   \n",
       "27           .                                                  .   \n",
       "28      452429                                    Cowden_syndrome   \n",
       "29       28691  MACRODACTYLY\\x2c_SOMATIC|Hepatocellular_carcin...   \n",
       "30      363360  Hepatocellular_carcinoma|Medulloblastoma|Adren...   \n",
       "31           .                                                  .   \n",
       "32           .                                                  .   \n",
       "33      829942  Idiopathic_Pulmonary_Fibrosis|Dyskeratosis_con...   \n",
       "34      633029  Idiopathic_Pulmonary_Fibrosis|Dyskeratosis_con...   \n",
       "35       29005  Neoplasm|Melanoma|Chronic_lymphocytic_leukemia...   \n",
       "36      362820  Trametinib-Dabrafenib_Response|Vemurafenib-Cob...   \n",
       "37       53980  Melanoma|Adrenocortical_carcinoma|Chronic_lymp...   \n",
       "38      653824                                Dabrafenib_response   \n",
       "39       29013  Melanoma|Squamous_cell_carcinoma_of_the_skin|T...   \n",
       "40       29010  Neoplasm|Squamous_cell_carcinoma_of_the_skin|T...   \n",
       "41           .                                                  .   \n",
       "42           .                                                  .   \n",
       "43       54000                            Rasopathy|not_specified   \n",
       "44       48804                             Rasopathy|not_provided   \n",
       "\n",
       "                                             CLNDISDB  \\\n",
       "0   MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...   \n",
       "1   MONDO:MONDO:0012371\\x2cMedGen:C1860991\\x2cOMIM...   \n",
       "2   Human_Phenotype_Ontology:HP:0030358\\x2cMONDO:M...   \n",
       "3   Human_Phenotype_Ontology:HP:0006775\\x2cMONDO:M...   \n",
       "4   Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...   \n",
       "5   Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...   \n",
       "6                                                   .   \n",
       "7   Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...   \n",
       "8   Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...   \n",
       "9   Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...   \n",
       "10                                                  .   \n",
       "11                                                  .   \n",
       "12  Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...   \n",
       "13  Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...   \n",
       "14  Human_Phenotype_Ontology:HP:0002665\\x2cMONDO:M...   \n",
       "15                                    MedGen:CN169374   \n",
       "16                                    MedGen:CN169374   \n",
       "17  Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...   \n",
       "18  Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...   \n",
       "19  Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...   \n",
       "20                    MedGen:CN169374|MedGen:CN517202   \n",
       "21                                    MedGen:CN517202   \n",
       "22                    MedGen:CN169374|MedGen:CN517202   \n",
       "23  Human_Phenotype_Ontology:HP:0010816\\x2cMONDO:M...   \n",
       "24                                    MedGen:CN169374   \n",
       "25  MONDO:MONDO:0013038\\x2cMedGen:C2752042\\x2cOMIM...   \n",
       "26  Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...   \n",
       "27                                                  .   \n",
       "28  MONDO:MONDO:0016063\\x2cMedGen:C0018553\\x2cOMIM...   \n",
       "29  .|Human_Phenotype_Ontology:HP:0001402\\x2cHuman...   \n",
       "30  Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...   \n",
       "31                                                  .   \n",
       "32                                                  .   \n",
       "33  MONDO:MONDO:0008345\\x2cMedGen:C1800706\\x2cOMIM...   \n",
       "34  MONDO:MONDO:0008345\\x2cMedGen:C1800706\\x2cOMIM...   \n",
       "35  Human_Phenotype_Ontology:HP:0002664\\x2cHuman_P...   \n",
       "36  .|.|Human_Phenotype_Ontology:HP:0002861\\x2cHum...   \n",
       "37  Human_Phenotype_Ontology:HP:0002861\\x2cHuman_P...   \n",
       "38                                    MedGen:CN239586   \n",
       "39  Human_Phenotype_Ontology:HP:0002861\\x2cHuman_P...   \n",
       "40  Human_Phenotype_Ontology:HP:0002664\\x2cHuman_P...   \n",
       "41                                                  .   \n",
       "42                                                  .   \n",
       "43  MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...   \n",
       "44  MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...   \n",
       "\n",
       "                                           CLNREVSTAT  \\\n",
       "0   criteria_provided\\x2c_multiple_submitters\\x2c_...   \n",
       "1                            reviewed_by_expert_panel   \n",
       "2              criteria_provided\\x2c_single_submitter   \n",
       "3                      no_assertion_criteria_provided   \n",
       "4                      no_assertion_criteria_provided   \n",
       "5                      no_assertion_criteria_provided   \n",
       "6                                                   .   \n",
       "7                      no_assertion_criteria_provided   \n",
       "8                      no_assertion_criteria_provided   \n",
       "9                      no_assertion_criteria_provided   \n",
       "10                                                  .   \n",
       "11                                                  .   \n",
       "12                     no_assertion_criteria_provided   \n",
       "13                     no_assertion_criteria_provided   \n",
       "14                     no_assertion_criteria_provided   \n",
       "15                              no_assertion_provided   \n",
       "16             criteria_provided\\x2c_single_submitter   \n",
       "17  criteria_provided\\x2c_multiple_submitters\\x2c_...   \n",
       "18                     no_assertion_criteria_provided   \n",
       "19  criteria_provided\\x2c_multiple_submitters\\x2c_...   \n",
       "20             criteria_provided\\x2c_single_submitter   \n",
       "21             criteria_provided\\x2c_single_submitter   \n",
       "22             criteria_provided\\x2c_single_submitter   \n",
       "23  criteria_provided\\x2c_multiple_submitters\\x2c_...   \n",
       "24             criteria_provided\\x2c_single_submitter   \n",
       "25             criteria_provided\\x2c_single_submitter   \n",
       "26  criteria_provided\\x2c_multiple_submitters\\x2c_...   \n",
       "27                                                  .   \n",
       "28             criteria_provided\\x2c_single_submitter   \n",
       "29  criteria_provided\\x2c_multiple_submitters\\x2c_...   \n",
       "30                     no_assertion_criteria_provided   \n",
       "31                                                  .   \n",
       "32                                                  .   \n",
       "33             criteria_provided\\x2c_single_submitter   \n",
       "34             criteria_provided\\x2c_single_submitter   \n",
       "35             criteria_provided\\x2c_single_submitter   \n",
       "36             criteria_provided\\x2c_single_submitter   \n",
       "37             criteria_provided\\x2c_single_submitter   \n",
       "38                     no_assertion_criteria_provided   \n",
       "39                           reviewed_by_expert_panel   \n",
       "40             criteria_provided\\x2c_single_submitter   \n",
       "41                                                  .   \n",
       "42                                                  .   \n",
       "43                           reviewed_by_expert_panel   \n",
       "44  criteria_provided\\x2c_multiple_submitters\\x2c_...   \n",
       "\n",
       "                                          CLNSIG  REVEL  rules  rule  \n",
       "0                         Uncertain_significance  0.745    OP4  None  \n",
       "1                              Likely_pathogenic  0.600    OP4   OP1  \n",
       "2                                     Pathogenic  0.630    OP4   OP1  \n",
       "3   Conflicting_interpretations_of_pathogenicity  0.841    OP4   OP1  \n",
       "4                              Likely_pathogenic  0.925    OP4   OP1  \n",
       "5                                     Pathogenic  0.629    OP4   OP1  \n",
       "6                                              .      .    OP4  SBP1  \n",
       "7                                     Pathogenic  0.894    OP4   OP1  \n",
       "8                                     Pathogenic  0.878    OP4   OP1  \n",
       "9                                     Pathogenic  0.875    OP4   OP1  \n",
       "10                                             .  0.883    OP4   OP1  \n",
       "11                                             .  0.917    OP4   OP1  \n",
       "12                             Likely_pathogenic  0.632    OP4   OP1  \n",
       "13                             Likely_pathogenic  0.465    OP4   OP1  \n",
       "14                        Uncertain_significance  0.264    OP4  SBP1  \n",
       "15                                  not_provided  0.908    OP4   OP1  \n",
       "16                                        Benign  0.421  SBVS1  SBP1  \n",
       "17                                    Pathogenic  0.852    OP4   OP1  \n",
       "18                  Pathogenic/Likely_pathogenic  0.904    OP4   OP1  \n",
       "19                  Pathogenic/Likely_pathogenic  0.814    OP4   OP1  \n",
       "20                                        Benign  0.103   SBS1  SBP1  \n",
       "21                                        Benign      .   SBS1  SBP1  \n",
       "22                                 Likely_benign  0.311    OP4  SBP1  \n",
       "23                        Uncertain_significance  0.700    OP4  SBP1  \n",
       "24                                        Benign  0.236  SBVS1  SBP1  \n",
       "25                        Uncertain_significance  0.519    OP4  None  \n",
       "26                                    Pathogenic  0.544    OP4  None  \n",
       "27                                             .  0.715    OP4  None  \n",
       "28                                        Benign  0.269    OP4  SBP1  \n",
       "29                                    Pathogenic  0.455    OP4  SBP1  \n",
       "30                             Likely_pathogenic  0.428    OP4  SBP1  \n",
       "31                                             .  0.384    OP4  SBP1  \n",
       "32                                             .      .    OP4  SBP1  \n",
       "33                                    Pathogenic      .    OP4   OP1  \n",
       "34                                    Pathogenic      .    OP4  SBP1  \n",
       "35                                    Pathogenic  0.916    OP4   OP1  \n",
       "36                                 drug_response      .    OP4  SBP1  \n",
       "37                             Likely_pathogenic  0.904    OP4   OP1  \n",
       "38                                 drug_response      .    OP4  SBP1  \n",
       "39                                    Pathogenic  0.921    OP4   OP1  \n",
       "40                                    Pathogenic  0.724    OP4   OP1  \n",
       "41                                             .  0.627    OP4   OP1  \n",
       "42                                             .      .    OP4  SBP1  \n",
       "43                                 Likely_benign  0.207    OP4  SBP1  \n",
       "44                        Uncertain_significance  0.093    OP4  SBP1  \n",
       "\n",
       "[45 rows x 116 columns]"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sample_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [],
   "source": [
    "import re\n",
    "sample_df['id'] = sample_df.apply(lambda x: x['AAChange.refGene'].split(':')[0]+'_'+x['AAChange.refGene'].split(':p.')[-1], axis=1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0</th>\n",
       "      <th>Chr</th>\n",
       "      <th>Start</th>\n",
       "      <th>End</th>\n",
       "      <th>Ref</th>\n",
       "      <th>Alt</th>\n",
       "      <th>Func.refGene</th>\n",
       "      <th>Gene.refGene</th>\n",
       "      <th>GeneDetail.refGene</th>\n",
       "      <th>ExonicFunc.refGene</th>\n",
       "      <th>...</th>\n",
       "      <th>MCAP</th>\n",
       "      <th>CLNALLELEID</th>\n",
       "      <th>CLNDN</th>\n",
       "      <th>CLNDISDB</th>\n",
       "      <th>CLNREVSTAT</th>\n",
       "      <th>CLNSIG</th>\n",
       "      <th>REVEL</th>\n",
       "      <th>rules</th>\n",
       "      <th>rule</th>\n",
       "      <th>id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>12</td>\n",
       "      <td>25209871</td>\n",
       "      <td>25209871</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.568996615134</td>\n",
       "      <td>54295</td>\n",
       "      <td>Rasopathy|not_specified</td>\n",
       "      <td>MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...</td>\n",
       "      <td>criteria_provided\\x2c_multiple_submitters\\x2c_...</td>\n",
       "      <td>Uncertain_significance</td>\n",
       "      <td>0.745</td>\n",
       "      <td>OP4</td>\n",
       "      <td>None</td>\n",
       "      <td>KRAS_R164Q</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>12</td>\n",
       "      <td>25227330</td>\n",
       "      <td>25227330</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.123105525756</td>\n",
       "      <td>432423</td>\n",
       "      <td>Noonan_syndrome_3|Rasopathy</td>\n",
       "      <td>MONDO:MONDO:0012371\\x2cMedGen:C1860991\\x2cOMIM...</td>\n",
       "      <td>reviewed_by_expert_panel</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>0.600</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "      <td>KRAS_S65I</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>12</td>\n",
       "      <td>25227343</td>\n",
       "      <td>25227343</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0655137715661</td>\n",
       "      <td>175715</td>\n",
       "      <td>Non-small_cell_lung_cancer|Neoplasm_of_the_thy...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0030358\\x2cMONDO:M...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.630</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "      <td>KRAS_Q61K</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>12</td>\n",
       "      <td>25245350</td>\n",
       "      <td>25245350</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0910971912578</td>\n",
       "      <td>54289</td>\n",
       "      <td>Multiple_myeloma|Lung_adenocarcinoma|Non-small...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0006775\\x2cMONDO:M...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Conflicting_interpretations_of_pathogenicity</td>\n",
       "      <td>0.841</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "      <td>KRAS_G12A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>13</td>\n",
       "      <td>28018483</td>\n",
       "      <td>28018483</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.293007099215</td>\n",
       "      <td>362981</td>\n",
       "      <td>Acute_myeloid_leukemia</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>0.925</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "      <td>FLT3_Y842C</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>5</td>\n",
       "      <td>13</td>\n",
       "      <td>28018503</td>\n",
       "      <td>28018503</td>\n",
       "      <td>A</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.225817333818</td>\n",
       "      <td>362851</td>\n",
       "      <td>Acute_myeloid_leukemia</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.629</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "      <td>FLT3_D835E</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>6</td>\n",
       "      <td>13</td>\n",
       "      <td>28018504</td>\n",
       "      <td>28018505</td>\n",
       "      <td>TC</td>\n",
       "      <td>AT</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonframeshift substitution</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "      <td>FLT3_D835I</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>7</td>\n",
       "      <td>13</td>\n",
       "      <td>28018504</td>\n",
       "      <td>28018504</td>\n",
       "      <td>T</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.264924784705</td>\n",
       "      <td>31311</td>\n",
       "      <td>Acute_myeloid_leukemia|Acute_lymphoid_leukemia</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.894</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "      <td>FLT3_D835V</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>8</td>\n",
       "      <td>13</td>\n",
       "      <td>28018505</td>\n",
       "      <td>28018505</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0876328368379</td>\n",
       "      <td>31312</td>\n",
       "      <td>Acute_myeloid_leukemia</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.878</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "      <td>FLT3_D835H</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>9</td>\n",
       "      <td>13</td>\n",
       "      <td>28018505</td>\n",
       "      <td>28018505</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0837176557721</td>\n",
       "      <td>31315</td>\n",
       "      <td>Acute_myeloid_leukemia|Acute_lymphoid_leukemia</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.875</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "      <td>FLT3_D835Y</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10</td>\n",
       "      <td>13</td>\n",
       "      <td>28027217</td>\n",
       "      <td>28027217</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.214409313169</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.883</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "      <td>FLT3_Y693C</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>11</td>\n",
       "      <td>13</td>\n",
       "      <td>28027218</td>\n",
       "      <td>28027218</td>\n",
       "      <td>A</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.297566568267</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.917</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "      <td>FLT3_Y693N</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>12</td>\n",
       "      <td>13</td>\n",
       "      <td>28027222</td>\n",
       "      <td>28027222</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0964035798672</td>\n",
       "      <td>362984</td>\n",
       "      <td>Acute_myeloid_leukemia</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>0.632</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "      <td>FLT3_F691L</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>13</td>\n",
       "      <td>13</td>\n",
       "      <td>28028203</td>\n",
       "      <td>28028203</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.108688019891</td>\n",
       "      <td>362988</td>\n",
       "      <td>Acute_myeloid_leukemia</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>0.465</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "      <td>FLT3_N676K</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>14</td>\n",
       "      <td>2</td>\n",
       "      <td>208242060</td>\n",
       "      <td>208242060</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0238719972834</td>\n",
       "      <td>626059</td>\n",
       "      <td>Lymphoma</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0002665\\x2cMONDO:M...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Uncertain_significance</td>\n",
       "      <td>0.264</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "      <td>IDH1_E262K</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>15</td>\n",
       "      <td>2</td>\n",
       "      <td>208243466</td>\n",
       "      <td>208243466</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.417115451694</td>\n",
       "      <td>138258</td>\n",
       "      <td>not_specified</td>\n",
       "      <td>MedGen:CN169374</td>\n",
       "      <td>no_assertion_provided</td>\n",
       "      <td>not_provided</td>\n",
       "      <td>0.908</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "      <td>IDH1_D220G</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>16</td>\n",
       "      <td>2</td>\n",
       "      <td>208243593</td>\n",
       "      <td>208243593</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>138259</td>\n",
       "      <td>not_specified</td>\n",
       "      <td>MedGen:CN169374</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Benign</td>\n",
       "      <td>0.421</td>\n",
       "      <td>SBVS1</td>\n",
       "      <td>SBP1</td>\n",
       "      <td>IDH1_V178I</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>17</td>\n",
       "      <td>2</td>\n",
       "      <td>208248388</td>\n",
       "      <td>208248388</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.560930781166</td>\n",
       "      <td>166215</td>\n",
       "      <td>Hepatocellular_carcinoma|Acute_myeloid_leukemi...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...</td>\n",
       "      <td>criteria_provided\\x2c_multiple_submitters\\x2c_...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.852</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "      <td>IDH1_R132H</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>18</td>\n",
       "      <td>2</td>\n",
       "      <td>208248389</td>\n",
       "      <td>208248389</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.612118206497</td>\n",
       "      <td>362772</td>\n",
       "      <td>Hepatocellular_carcinoma|Acute_myeloid_leukemi...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Pathogenic/Likely_pathogenic</td>\n",
       "      <td>0.904</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "      <td>IDH1_R132S</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>19</td>\n",
       "      <td>2</td>\n",
       "      <td>208248389</td>\n",
       "      <td>208248389</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.530890426099</td>\n",
       "      <td>362770</td>\n",
       "      <td>Hepatocellular_carcinoma|Acute_myeloid_leukemi...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...</td>\n",
       "      <td>criteria_provided\\x2c_multiple_submitters\\x2c_...</td>\n",
       "      <td>Pathogenic/Likely_pathogenic</td>\n",
       "      <td>0.814</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "      <td>IDH1_R132C</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>20</td>\n",
       "      <td>2</td>\n",
       "      <td>208248572</td>\n",
       "      <td>208248572</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>138255</td>\n",
       "      <td>not_specified|not_provided</td>\n",
       "      <td>MedGen:CN169374|MedGen:CN517202</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Benign</td>\n",
       "      <td>0.103</td>\n",
       "      <td>SBS1</td>\n",
       "      <td>SBP1</td>\n",
       "      <td>IDH1_V71I</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>21</td>\n",
       "      <td>2</td>\n",
       "      <td>208248664</td>\n",
       "      <td>208248664</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>splicing</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>NM_001282387:exon4:c.123-4C&gt;T;NM_005896:exon4:...</td>\n",
       "      <td>.</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>778916</td>\n",
       "      <td>not_provided</td>\n",
       "      <td>MedGen:CN517202</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Benign</td>\n",
       "      <td>.</td>\n",
       "      <td>SBS1</td>\n",
       "      <td>SBP1</td>\n",
       "      <td>._.</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>22</td>\n",
       "      <td>2</td>\n",
       "      <td>208251458</td>\n",
       "      <td>208251458</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0549463899991</td>\n",
       "      <td>138254</td>\n",
       "      <td>not_specified|not_provided</td>\n",
       "      <td>MedGen:CN169374|MedGen:CN517202</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Likely_benign</td>\n",
       "      <td>0.311</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "      <td>IDH1_F32V</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>23</td>\n",
       "      <td>3</td>\n",
       "      <td>179198826</td>\n",
       "      <td>179198826</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>startloss</td>\n",
       "      <td>...</td>\n",
       "      <td>0.2576582387</td>\n",
       "      <td>827717</td>\n",
       "      <td>Epidermal_nevus|Megalencephaly-capillary_malfo...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0010816\\x2cMONDO:M...</td>\n",
       "      <td>criteria_provided\\x2c_multiple_submitters\\x2c_...</td>\n",
       "      <td>Uncertain_significance</td>\n",
       "      <td>0.700</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "      <td>PIK3CA_M1?</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>24</td>\n",
       "      <td>3</td>\n",
       "      <td>179209622</td>\n",
       "      <td>179209622</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>138777</td>\n",
       "      <td>not_specified</td>\n",
       "      <td>MedGen:CN169374</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Benign</td>\n",
       "      <td>0.236</td>\n",
       "      <td>SBVS1</td>\n",
       "      <td>SBP1</td>\n",
       "      <td>PIK3CA_I391M</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>25</td>\n",
       "      <td>3</td>\n",
       "      <td>179210186</td>\n",
       "      <td>179210186</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.134419997912</td>\n",
       "      <td>54631</td>\n",
       "      <td>Congenital_lipomatous_overgrowth\\x2c_vascular_...</td>\n",
       "      <td>MONDO:MONDO:0013038\\x2cMedGen:C2752042\\x2cOMIM...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Uncertain_significance</td>\n",
       "      <td>0.519</td>\n",
       "      <td>OP4</td>\n",
       "      <td>None</td>\n",
       "      <td>PIK3CA_E418K</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>26</td>\n",
       "      <td>3</td>\n",
       "      <td>179210291</td>\n",
       "      <td>179210291</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.113788654374</td>\n",
       "      <td>363349</td>\n",
       "      <td>Hepatocellular_carcinoma|Abnormality_of_cardio...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...</td>\n",
       "      <td>criteria_provided\\x2c_multiple_submitters\\x2c_...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.544</td>\n",
       "      <td>OP4</td>\n",
       "      <td>None</td>\n",
       "      <td>PIK3CA_E453K</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>27</td>\n",
       "      <td>3</td>\n",
       "      <td>179218304</td>\n",
       "      <td>179218304</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.088721466783</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.715</td>\n",
       "      <td>OP4</td>\n",
       "      <td>None</td>\n",
       "      <td>PIK3CA_E545A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>28</td>\n",
       "      <td>3</td>\n",
       "      <td>179224091</td>\n",
       "      <td>179224091</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0164815254628</td>\n",
       "      <td>452429</td>\n",
       "      <td>Cowden_syndrome</td>\n",
       "      <td>MONDO:MONDO:0016063\\x2cMedGen:C0018553\\x2cOMIM...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Benign</td>\n",
       "      <td>0.269</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "      <td>PIK3CA_K733R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>29</td>\n",
       "      <td>3</td>\n",
       "      <td>179234297</td>\n",
       "      <td>179234297</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0373781845074</td>\n",
       "      <td>28691</td>\n",
       "      <td>MACRODACTYLY\\x2c_SOMATIC|Hepatocellular_carcin...</td>\n",
       "      <td>.|Human_Phenotype_Ontology:HP:0001402\\x2cHuman...</td>\n",
       "      <td>criteria_provided\\x2c_multiple_submitters\\x2c_...</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.455</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "      <td>PIK3CA_H1047R</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>30</td>\n",
       "      <td>3</td>\n",
       "      <td>179234298</td>\n",
       "      <td>179234298</td>\n",
       "      <td>T</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.026514517841</td>\n",
       "      <td>363360</td>\n",
       "      <td>Hepatocellular_carcinoma|Medulloblastoma|Adren...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>0.428</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "      <td>PIK3CA_H1047Q</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>31</td>\n",
       "      <td>3</td>\n",
       "      <td>179234350</td>\n",
       "      <td>179234350</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0220153461798</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.384</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "      <td>PIK3CA_H1065Y</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>32</td>\n",
       "      <td>3</td>\n",
       "      <td>179234360</td>\n",
       "      <td>179234360</td>\n",
       "      <td>-</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>frameshift insertion</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "      <td>PIK3CA_N1068Kfs*5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>33</td>\n",
       "      <td>5</td>\n",
       "      <td>1282455</td>\n",
       "      <td>1282455</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>TERT</td>\n",
       "      <td>.</td>\n",
       "      <td>stopgain</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>829942</td>\n",
       "      <td>Idiopathic_Pulmonary_Fibrosis|Dyskeratosis_con...</td>\n",
       "      <td>MONDO:MONDO:0008345\\x2cMedGen:C1800706\\x2cOMIM...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>.</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "      <td>TERT_W581X</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>34</td>\n",
       "      <td>5</td>\n",
       "      <td>1294198</td>\n",
       "      <td>1294198</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>TERT</td>\n",
       "      <td>.</td>\n",
       "      <td>stopgain</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>633029</td>\n",
       "      <td>Idiopathic_Pulmonary_Fibrosis|Dyskeratosis_con...</td>\n",
       "      <td>MONDO:MONDO:0008345\\x2cMedGen:C1800706\\x2cOMIM...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>.</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "      <td>TERT_R230X</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>35</td>\n",
       "      <td>7</td>\n",
       "      <td>140753334</td>\n",
       "      <td>140753334</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0820752694957</td>\n",
       "      <td>29005</td>\n",
       "      <td>Neoplasm|Melanoma|Chronic_lymphocytic_leukemia...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0002664\\x2cHuman_P...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.916</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "      <td>BRAF_K601E</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>36</td>\n",
       "      <td>7</td>\n",
       "      <td>140753336</td>\n",
       "      <td>140753337</td>\n",
       "      <td>AC</td>\n",
       "      <td>TT</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonframeshift substitution</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>362820</td>\n",
       "      <td>Trametinib-Dabrafenib_Response|Vemurafenib-Cob...</td>\n",
       "      <td>.|.|Human_Phenotype_Ontology:HP:0002861\\x2cHum...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>drug_response</td>\n",
       "      <td>.</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "      <td>BRAF_V600K</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>37</td>\n",
       "      <td>7</td>\n",
       "      <td>140753355</td>\n",
       "      <td>140753355</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.596855831475</td>\n",
       "      <td>53980</td>\n",
       "      <td>Melanoma|Adrenocortical_carcinoma|Chronic_lymp...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0002861\\x2cHuman_P...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Likely_pathogenic</td>\n",
       "      <td>0.904</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "      <td>BRAF_D594N</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>38</td>\n",
       "      <td>7</td>\n",
       "      <td>140778037</td>\n",
       "      <td>140778051</td>\n",
       "      <td>TAGGTGCTGTCACAT</td>\n",
       "      <td>-</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonframeshift deletion</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>653824</td>\n",
       "      <td>Dabrafenib_response</td>\n",
       "      <td>MedGen:CN239586</td>\n",
       "      <td>no_assertion_criteria_provided</td>\n",
       "      <td>drug_response</td>\n",
       "      <td>.</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "      <td>BRAF_N486_P490del</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>39</td>\n",
       "      <td>7</td>\n",
       "      <td>140781602</td>\n",
       "      <td>140781602</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.612203233818</td>\n",
       "      <td>29013</td>\n",
       "      <td>Melanoma|Squamous_cell_carcinoma_of_the_skin|T...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0002861\\x2cHuman_P...</td>\n",
       "      <td>reviewed_by_expert_panel</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.921</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "      <td>BRAF_G469E</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>40</td>\n",
       "      <td>7</td>\n",
       "      <td>140781602</td>\n",
       "      <td>140781602</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.219344139139</td>\n",
       "      <td>29010</td>\n",
       "      <td>Neoplasm|Squamous_cell_carcinoma_of_the_skin|T...</td>\n",
       "      <td>Human_Phenotype_Ontology:HP:0002664\\x2cHuman_P...</td>\n",
       "      <td>criteria_provided\\x2c_single_submitter</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>0.724</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "      <td>BRAF_G469A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>41</td>\n",
       "      <td>7</td>\n",
       "      <td>140781675</td>\n",
       "      <td>140781675</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.100260968362</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>0.627</td>\n",
       "      <td>OP4</td>\n",
       "      <td>OP1</td>\n",
       "      <td>BRAF_D445H</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>42</td>\n",
       "      <td>7</td>\n",
       "      <td>140783127</td>\n",
       "      <td>140783127</td>\n",
       "      <td>G</td>\n",
       "      <td>-</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>frameshift deletion</td>\n",
       "      <td>...</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>.</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "      <td>BRAF_P403Lfs*8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>43</td>\n",
       "      <td>7</td>\n",
       "      <td>140924612</td>\n",
       "      <td>140924612</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.261712773584</td>\n",
       "      <td>54000</td>\n",
       "      <td>Rasopathy|not_specified</td>\n",
       "      <td>MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...</td>\n",
       "      <td>reviewed_by_expert_panel</td>\n",
       "      <td>Likely_benign</td>\n",
       "      <td>0.207</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "      <td>BRAF_A31G</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>44</td>\n",
       "      <td>7</td>\n",
       "      <td>140924663</td>\n",
       "      <td>140924663</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0891133498287</td>\n",
       "      <td>48804</td>\n",
       "      <td>Rasopathy|not_provided</td>\n",
       "      <td>MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...</td>\n",
       "      <td>criteria_provided\\x2c_multiple_submitters\\x2c_...</td>\n",
       "      <td>Uncertain_significance</td>\n",
       "      <td>0.093</td>\n",
       "      <td>OP4</td>\n",
       "      <td>SBP1</td>\n",
       "      <td>BRAF_P14R</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>45 rows  117 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "    Unnamed: 0  Chr      Start        End              Ref Alt Func.refGene  \\\n",
       "0            0   12   25209871   25209871                C   T       exonic   \n",
       "1            1   12   25227330   25227330                C   A       exonic   \n",
       "2            2   12   25227343   25227343                G   T       exonic   \n",
       "3            3   12   25245350   25245350                C   G       exonic   \n",
       "4            4   13   28018483   28018483                T   C       exonic   \n",
       "5            5   13   28018503   28018503                A   T       exonic   \n",
       "6            6   13   28018504   28018505               TC  AT       exonic   \n",
       "7            7   13   28018504   28018504                T   A       exonic   \n",
       "8            8   13   28018505   28018505                C   G       exonic   \n",
       "9            9   13   28018505   28018505                C   A       exonic   \n",
       "10          10   13   28027217   28027217                T   C       exonic   \n",
       "11          11   13   28027218   28027218                A   T       exonic   \n",
       "12          12   13   28027222   28027222                A   C       exonic   \n",
       "13          13   13   28028203   28028203                G   T       exonic   \n",
       "14          14    2  208242060  208242060                C   T       exonic   \n",
       "15          15    2  208243466  208243466                T   C       exonic   \n",
       "16          16    2  208243593  208243593                C   T       exonic   \n",
       "17          17    2  208248388  208248388                C   T       exonic   \n",
       "18          18    2  208248389  208248389                G   T       exonic   \n",
       "19          19    2  208248389  208248389                G   A       exonic   \n",
       "20          20    2  208248572  208248572                C   T       exonic   \n",
       "21          21    2  208248664  208248664                G   A     splicing   \n",
       "22          22    2  208251458  208251458                A   C       exonic   \n",
       "23          23    3  179198826  179198826                A   G       exonic   \n",
       "24          24    3  179209622  179209622                A   G       exonic   \n",
       "25          25    3  179210186  179210186                G   A       exonic   \n",
       "26          26    3  179210291  179210291                G   A       exonic   \n",
       "27          27    3  179218304  179218304                A   C       exonic   \n",
       "28          28    3  179224091  179224091                A   G       exonic   \n",
       "29          29    3  179234297  179234297                A   G       exonic   \n",
       "30          30    3  179234298  179234298                T   G       exonic   \n",
       "31          31    3  179234350  179234350                C   T       exonic   \n",
       "32          32    3  179234360  179234360                -   A       exonic   \n",
       "33          33    5    1282455    1282455                C   T       exonic   \n",
       "34          34    5    1294198    1294198                G   A       exonic   \n",
       "35          35    7  140753334  140753334                T   C       exonic   \n",
       "36          36    7  140753336  140753337               AC  TT       exonic   \n",
       "37          37    7  140753355  140753355                C   T       exonic   \n",
       "38          38    7  140778037  140778051  TAGGTGCTGTCACAT   -       exonic   \n",
       "39          39    7  140781602  140781602                C   T       exonic   \n",
       "40          40    7  140781602  140781602                C   G       exonic   \n",
       "41          41    7  140781675  140781675                C   G       exonic   \n",
       "42          42    7  140783127  140783127                G   -       exonic   \n",
       "43          43    7  140924612  140924612                G   C       exonic   \n",
       "44          44    7  140924663  140924663                G   C       exonic   \n",
       "\n",
       "   Gene.refGene                                 GeneDetail.refGene  \\\n",
       "0          KRAS                                                  .   \n",
       "1          KRAS                                                  .   \n",
       "2          KRAS                                                  .   \n",
       "3          KRAS                                                  .   \n",
       "4          FLT3                                                  .   \n",
       "5          FLT3                                                  .   \n",
       "6          FLT3                                                  .   \n",
       "7          FLT3                                                  .   \n",
       "8          FLT3                                                  .   \n",
       "9          FLT3                                                  .   \n",
       "10         FLT3                                                  .   \n",
       "11         FLT3                                                  .   \n",
       "12         FLT3                                                  .   \n",
       "13         FLT3                                                  .   \n",
       "14         IDH1                                                  .   \n",
       "15         IDH1                                                  .   \n",
       "16         IDH1                                                  .   \n",
       "17         IDH1                                                  .   \n",
       "18         IDH1                                                  .   \n",
       "19         IDH1                                                  .   \n",
       "20         IDH1                                                  .   \n",
       "21         IDH1  NM_001282387:exon4:c.123-4C>T;NM_005896:exon4:...   \n",
       "22         IDH1                                                  .   \n",
       "23       PIK3CA                                                  .   \n",
       "24       PIK3CA                                                  .   \n",
       "25       PIK3CA                                                  .   \n",
       "26       PIK3CA                                                  .   \n",
       "27       PIK3CA                                                  .   \n",
       "28       PIK3CA                                                  .   \n",
       "29       PIK3CA                                                  .   \n",
       "30       PIK3CA                                                  .   \n",
       "31       PIK3CA                                                  .   \n",
       "32       PIK3CA                                                  .   \n",
       "33         TERT                                                  .   \n",
       "34         TERT                                                  .   \n",
       "35         BRAF                                                  .   \n",
       "36         BRAF                                                  .   \n",
       "37         BRAF                                                  .   \n",
       "38         BRAF                                                  .   \n",
       "39         BRAF                                                  .   \n",
       "40         BRAF                                                  .   \n",
       "41         BRAF                                                  .   \n",
       "42         BRAF                                                  .   \n",
       "43         BRAF                                                  .   \n",
       "44         BRAF                                                  .   \n",
       "\n",
       "            ExonicFunc.refGene  ...             MCAP CLNALLELEID  \\\n",
       "0            nonsynonymous SNV  ...   0.568996615134       54295   \n",
       "1            nonsynonymous SNV  ...   0.123105525756      432423   \n",
       "2            nonsynonymous SNV  ...  0.0655137715661      175715   \n",
       "3            nonsynonymous SNV  ...  0.0910971912578       54289   \n",
       "4            nonsynonymous SNV  ...   0.293007099215      362981   \n",
       "5            nonsynonymous SNV  ...   0.225817333818      362851   \n",
       "6   nonframeshift substitution  ...                .           .   \n",
       "7            nonsynonymous SNV  ...   0.264924784705       31311   \n",
       "8            nonsynonymous SNV  ...  0.0876328368379       31312   \n",
       "9            nonsynonymous SNV  ...  0.0837176557721       31315   \n",
       "10           nonsynonymous SNV  ...   0.214409313169           .   \n",
       "11           nonsynonymous SNV  ...   0.297566568267           .   \n",
       "12           nonsynonymous SNV  ...  0.0964035798672      362984   \n",
       "13           nonsynonymous SNV  ...   0.108688019891      362988   \n",
       "14           nonsynonymous SNV  ...  0.0238719972834      626059   \n",
       "15           nonsynonymous SNV  ...   0.417115451694      138258   \n",
       "16           nonsynonymous SNV  ...                .      138259   \n",
       "17           nonsynonymous SNV  ...   0.560930781166      166215   \n",
       "18           nonsynonymous SNV  ...   0.612118206497      362772   \n",
       "19           nonsynonymous SNV  ...   0.530890426099      362770   \n",
       "20           nonsynonymous SNV  ...                .      138255   \n",
       "21                           .  ...                .      778916   \n",
       "22           nonsynonymous SNV  ...  0.0549463899991      138254   \n",
       "23                   startloss  ...     0.2576582387      827717   \n",
       "24           nonsynonymous SNV  ...                .      138777   \n",
       "25           nonsynonymous SNV  ...   0.134419997912       54631   \n",
       "26           nonsynonymous SNV  ...   0.113788654374      363349   \n",
       "27           nonsynonymous SNV  ...   0.088721466783           .   \n",
       "28           nonsynonymous SNV  ...  0.0164815254628      452429   \n",
       "29           nonsynonymous SNV  ...  0.0373781845074       28691   \n",
       "30           nonsynonymous SNV  ...   0.026514517841      363360   \n",
       "31           nonsynonymous SNV  ...  0.0220153461798           .   \n",
       "32        frameshift insertion  ...                .           .   \n",
       "33                    stopgain  ...                .      829942   \n",
       "34                    stopgain  ...                .      633029   \n",
       "35           nonsynonymous SNV  ...  0.0820752694957       29005   \n",
       "36  nonframeshift substitution  ...                .      362820   \n",
       "37           nonsynonymous SNV  ...   0.596855831475       53980   \n",
       "38      nonframeshift deletion  ...                .      653824   \n",
       "39           nonsynonymous SNV  ...   0.612203233818       29013   \n",
       "40           nonsynonymous SNV  ...   0.219344139139       29010   \n",
       "41           nonsynonymous SNV  ...   0.100260968362           .   \n",
       "42         frameshift deletion  ...                .           .   \n",
       "43           nonsynonymous SNV  ...   0.261712773584       54000   \n",
       "44           nonsynonymous SNV  ...  0.0891133498287       48804   \n",
       "\n",
       "                                                CLNDN  \\\n",
       "0                             Rasopathy|not_specified   \n",
       "1                         Noonan_syndrome_3|Rasopathy   \n",
       "2   Non-small_cell_lung_cancer|Neoplasm_of_the_thy...   \n",
       "3   Multiple_myeloma|Lung_adenocarcinoma|Non-small...   \n",
       "4                              Acute_myeloid_leukemia   \n",
       "5                              Acute_myeloid_leukemia   \n",
       "6                                                   .   \n",
       "7      Acute_myeloid_leukemia|Acute_lymphoid_leukemia   \n",
       "8                              Acute_myeloid_leukemia   \n",
       "9      Acute_myeloid_leukemia|Acute_lymphoid_leukemia   \n",
       "10                                                  .   \n",
       "11                                                  .   \n",
       "12                             Acute_myeloid_leukemia   \n",
       "13                             Acute_myeloid_leukemia   \n",
       "14                                           Lymphoma   \n",
       "15                                      not_specified   \n",
       "16                                      not_specified   \n",
       "17  Hepatocellular_carcinoma|Acute_myeloid_leukemi...   \n",
       "18  Hepatocellular_carcinoma|Acute_myeloid_leukemi...   \n",
       "19  Hepatocellular_carcinoma|Acute_myeloid_leukemi...   \n",
       "20                         not_specified|not_provided   \n",
       "21                                       not_provided   \n",
       "22                         not_specified|not_provided   \n",
       "23  Epidermal_nevus|Megalencephaly-capillary_malfo...   \n",
       "24                                      not_specified   \n",
       "25  Congenital_lipomatous_overgrowth\\x2c_vascular_...   \n",
       "26  Hepatocellular_carcinoma|Abnormality_of_cardio...   \n",
       "27                                                  .   \n",
       "28                                    Cowden_syndrome   \n",
       "29  MACRODACTYLY\\x2c_SOMATIC|Hepatocellular_carcin...   \n",
       "30  Hepatocellular_carcinoma|Medulloblastoma|Adren...   \n",
       "31                                                  .   \n",
       "32                                                  .   \n",
       "33  Idiopathic_Pulmonary_Fibrosis|Dyskeratosis_con...   \n",
       "34  Idiopathic_Pulmonary_Fibrosis|Dyskeratosis_con...   \n",
       "35  Neoplasm|Melanoma|Chronic_lymphocytic_leukemia...   \n",
       "36  Trametinib-Dabrafenib_Response|Vemurafenib-Cob...   \n",
       "37  Melanoma|Adrenocortical_carcinoma|Chronic_lymp...   \n",
       "38                                Dabrafenib_response   \n",
       "39  Melanoma|Squamous_cell_carcinoma_of_the_skin|T...   \n",
       "40  Neoplasm|Squamous_cell_carcinoma_of_the_skin|T...   \n",
       "41                                                  .   \n",
       "42                                                  .   \n",
       "43                            Rasopathy|not_specified   \n",
       "44                             Rasopathy|not_provided   \n",
       "\n",
       "                                             CLNDISDB  \\\n",
       "0   MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...   \n",
       "1   MONDO:MONDO:0012371\\x2cMedGen:C1860991\\x2cOMIM...   \n",
       "2   Human_Phenotype_Ontology:HP:0030358\\x2cMONDO:M...   \n",
       "3   Human_Phenotype_Ontology:HP:0006775\\x2cMONDO:M...   \n",
       "4   Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...   \n",
       "5   Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...   \n",
       "6                                                   .   \n",
       "7   Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...   \n",
       "8   Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...   \n",
       "9   Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...   \n",
       "10                                                  .   \n",
       "11                                                  .   \n",
       "12  Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...   \n",
       "13  Human_Phenotype_Ontology:HP:0001914\\x2cHuman_P...   \n",
       "14  Human_Phenotype_Ontology:HP:0002665\\x2cMONDO:M...   \n",
       "15                                    MedGen:CN169374   \n",
       "16                                    MedGen:CN169374   \n",
       "17  Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...   \n",
       "18  Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...   \n",
       "19  Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...   \n",
       "20                    MedGen:CN169374|MedGen:CN517202   \n",
       "21                                    MedGen:CN517202   \n",
       "22                    MedGen:CN169374|MedGen:CN517202   \n",
       "23  Human_Phenotype_Ontology:HP:0010816\\x2cMONDO:M...   \n",
       "24                                    MedGen:CN169374   \n",
       "25  MONDO:MONDO:0013038\\x2cMedGen:C2752042\\x2cOMIM...   \n",
       "26  Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...   \n",
       "27                                                  .   \n",
       "28  MONDO:MONDO:0016063\\x2cMedGen:C0018553\\x2cOMIM...   \n",
       "29  .|Human_Phenotype_Ontology:HP:0001402\\x2cHuman...   \n",
       "30  Human_Phenotype_Ontology:HP:0001402\\x2cHuman_P...   \n",
       "31                                                  .   \n",
       "32                                                  .   \n",
       "33  MONDO:MONDO:0008345\\x2cMedGen:C1800706\\x2cOMIM...   \n",
       "34  MONDO:MONDO:0008345\\x2cMedGen:C1800706\\x2cOMIM...   \n",
       "35  Human_Phenotype_Ontology:HP:0002664\\x2cHuman_P...   \n",
       "36  .|.|Human_Phenotype_Ontology:HP:0002861\\x2cHum...   \n",
       "37  Human_Phenotype_Ontology:HP:0002861\\x2cHuman_P...   \n",
       "38                                    MedGen:CN239586   \n",
       "39  Human_Phenotype_Ontology:HP:0002861\\x2cHuman_P...   \n",
       "40  Human_Phenotype_Ontology:HP:0002664\\x2cHuman_P...   \n",
       "41                                                  .   \n",
       "42                                                  .   \n",
       "43  MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...   \n",
       "44  MONDO:MONDO:0021060\\x2cMedGen:CN166718\\x2cOrph...   \n",
       "\n",
       "                                           CLNREVSTAT  \\\n",
       "0   criteria_provided\\x2c_multiple_submitters\\x2c_...   \n",
       "1                            reviewed_by_expert_panel   \n",
       "2              criteria_provided\\x2c_single_submitter   \n",
       "3                      no_assertion_criteria_provided   \n",
       "4                      no_assertion_criteria_provided   \n",
       "5                      no_assertion_criteria_provided   \n",
       "6                                                   .   \n",
       "7                      no_assertion_criteria_provided   \n",
       "8                      no_assertion_criteria_provided   \n",
       "9                      no_assertion_criteria_provided   \n",
       "10                                                  .   \n",
       "11                                                  .   \n",
       "12                     no_assertion_criteria_provided   \n",
       "13                     no_assertion_criteria_provided   \n",
       "14                     no_assertion_criteria_provided   \n",
       "15                              no_assertion_provided   \n",
       "16             criteria_provided\\x2c_single_submitter   \n",
       "17  criteria_provided\\x2c_multiple_submitters\\x2c_...   \n",
       "18                     no_assertion_criteria_provided   \n",
       "19  criteria_provided\\x2c_multiple_submitters\\x2c_...   \n",
       "20             criteria_provided\\x2c_single_submitter   \n",
       "21             criteria_provided\\x2c_single_submitter   \n",
       "22             criteria_provided\\x2c_single_submitter   \n",
       "23  criteria_provided\\x2c_multiple_submitters\\x2c_...   \n",
       "24             criteria_provided\\x2c_single_submitter   \n",
       "25             criteria_provided\\x2c_single_submitter   \n",
       "26  criteria_provided\\x2c_multiple_submitters\\x2c_...   \n",
       "27                                                  .   \n",
       "28             criteria_provided\\x2c_single_submitter   \n",
       "29  criteria_provided\\x2c_multiple_submitters\\x2c_...   \n",
       "30                     no_assertion_criteria_provided   \n",
       "31                                                  .   \n",
       "32                                                  .   \n",
       "33             criteria_provided\\x2c_single_submitter   \n",
       "34             criteria_provided\\x2c_single_submitter   \n",
       "35             criteria_provided\\x2c_single_submitter   \n",
       "36             criteria_provided\\x2c_single_submitter   \n",
       "37             criteria_provided\\x2c_single_submitter   \n",
       "38                     no_assertion_criteria_provided   \n",
       "39                           reviewed_by_expert_panel   \n",
       "40             criteria_provided\\x2c_single_submitter   \n",
       "41                                                  .   \n",
       "42                                                  .   \n",
       "43                           reviewed_by_expert_panel   \n",
       "44  criteria_provided\\x2c_multiple_submitters\\x2c_...   \n",
       "\n",
       "                                          CLNSIG  REVEL  rules  rule  \\\n",
       "0                         Uncertain_significance  0.745    OP4  None   \n",
       "1                              Likely_pathogenic  0.600    OP4   OP1   \n",
       "2                                     Pathogenic  0.630    OP4   OP1   \n",
       "3   Conflicting_interpretations_of_pathogenicity  0.841    OP4   OP1   \n",
       "4                              Likely_pathogenic  0.925    OP4   OP1   \n",
       "5                                     Pathogenic  0.629    OP4   OP1   \n",
       "6                                              .      .    OP4  SBP1   \n",
       "7                                     Pathogenic  0.894    OP4   OP1   \n",
       "8                                     Pathogenic  0.878    OP4   OP1   \n",
       "9                                     Pathogenic  0.875    OP4   OP1   \n",
       "10                                             .  0.883    OP4   OP1   \n",
       "11                                             .  0.917    OP4   OP1   \n",
       "12                             Likely_pathogenic  0.632    OP4   OP1   \n",
       "13                             Likely_pathogenic  0.465    OP4   OP1   \n",
       "14                        Uncertain_significance  0.264    OP4  SBP1   \n",
       "15                                  not_provided  0.908    OP4   OP1   \n",
       "16                                        Benign  0.421  SBVS1  SBP1   \n",
       "17                                    Pathogenic  0.852    OP4   OP1   \n",
       "18                  Pathogenic/Likely_pathogenic  0.904    OP4   OP1   \n",
       "19                  Pathogenic/Likely_pathogenic  0.814    OP4   OP1   \n",
       "20                                        Benign  0.103   SBS1  SBP1   \n",
       "21                                        Benign      .   SBS1  SBP1   \n",
       "22                                 Likely_benign  0.311    OP4  SBP1   \n",
       "23                        Uncertain_significance  0.700    OP4  SBP1   \n",
       "24                                        Benign  0.236  SBVS1  SBP1   \n",
       "25                        Uncertain_significance  0.519    OP4  None   \n",
       "26                                    Pathogenic  0.544    OP4  None   \n",
       "27                                             .  0.715    OP4  None   \n",
       "28                                        Benign  0.269    OP4  SBP1   \n",
       "29                                    Pathogenic  0.455    OP4  SBP1   \n",
       "30                             Likely_pathogenic  0.428    OP4  SBP1   \n",
       "31                                             .  0.384    OP4  SBP1   \n",
       "32                                             .      .    OP4  SBP1   \n",
       "33                                    Pathogenic      .    OP4   OP1   \n",
       "34                                    Pathogenic      .    OP4  SBP1   \n",
       "35                                    Pathogenic  0.916    OP4   OP1   \n",
       "36                                 drug_response      .    OP4  SBP1   \n",
       "37                             Likely_pathogenic  0.904    OP4   OP1   \n",
       "38                                 drug_response      .    OP4  SBP1   \n",
       "39                                    Pathogenic  0.921    OP4   OP1   \n",
       "40                                    Pathogenic  0.724    OP4   OP1   \n",
       "41                                             .  0.627    OP4   OP1   \n",
       "42                                             .      .    OP4  SBP1   \n",
       "43                                 Likely_benign  0.207    OP4  SBP1   \n",
       "44                        Uncertain_significance  0.093    OP4  SBP1   \n",
       "\n",
       "                   id  \n",
       "0          KRAS_R164Q  \n",
       "1           KRAS_S65I  \n",
       "2           KRAS_Q61K  \n",
       "3           KRAS_G12A  \n",
       "4          FLT3_Y842C  \n",
       "5          FLT3_D835E  \n",
       "6          FLT3_D835I  \n",
       "7          FLT3_D835V  \n",
       "8          FLT3_D835H  \n",
       "9          FLT3_D835Y  \n",
       "10         FLT3_Y693C  \n",
       "11         FLT3_Y693N  \n",
       "12         FLT3_F691L  \n",
       "13         FLT3_N676K  \n",
       "14         IDH1_E262K  \n",
       "15         IDH1_D220G  \n",
       "16         IDH1_V178I  \n",
       "17         IDH1_R132H  \n",
       "18         IDH1_R132S  \n",
       "19         IDH1_R132C  \n",
       "20          IDH1_V71I  \n",
       "21                ._.  \n",
       "22          IDH1_F32V  \n",
       "23         PIK3CA_M1?  \n",
       "24       PIK3CA_I391M  \n",
       "25       PIK3CA_E418K  \n",
       "26       PIK3CA_E453K  \n",
       "27       PIK3CA_E545A  \n",
       "28       PIK3CA_K733R  \n",
       "29      PIK3CA_H1047R  \n",
       "30      PIK3CA_H1047Q  \n",
       "31      PIK3CA_H1065Y  \n",
       "32  PIK3CA_N1068Kfs*5  \n",
       "33         TERT_W581X  \n",
       "34         TERT_R230X  \n",
       "35         BRAF_K601E  \n",
       "36         BRAF_V600K  \n",
       "37         BRAF_D594N  \n",
       "38  BRAF_N486_P490del  \n",
       "39         BRAF_G469E  \n",
       "40         BRAF_G469A  \n",
       "41         BRAF_D445H  \n",
       "42     BRAF_P403Lfs*8  \n",
       "43          BRAF_A31G  \n",
       "44          BRAF_P14R  \n",
       "\n",
       "[45 rows x 117 columns]"
      ]
     },
     "execution_count": 46,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sample_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [],
   "source": [
    "cancer_hotspots_df = pd.read_csv(\"/Users/hatice.irmak/Desktop/somatic_rules/cancer_hotspots_revised.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [],
   "source": [
    "cancer_hotspots_df = cancer_hotspots_df.drop(columns=[\"Unnamed: 0\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Hugo_Symbol</th>\n",
       "      <th>Amino_Acid_Position</th>\n",
       "      <th>Mutation_Count</th>\n",
       "      <th>Reference_Amino_Acid</th>\n",
       "      <th>Total_Mutations_in_Gene</th>\n",
       "      <th>Variant_Amino_Acid</th>\n",
       "      <th>Codon_Change</th>\n",
       "      <th>Genomic_Position</th>\n",
       "      <th>Detailed_Cancer_Types</th>\n",
       "      <th>Organ_Types</th>\n",
       "      <th>ref</th>\n",
       "      <th>ct</th>\n",
       "      <th>Samples</th>\n",
       "      <th>id</th>\n",
       "      <th>total_variant_count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NRAS</td>\n",
       "      <td>61</td>\n",
       "      <td>422</td>\n",
       "      <td>Q:422</td>\n",
       "      <td>620</td>\n",
       "      <td>R:204</td>\n",
       "      <td>cAa/cGa:203|Caa/Aaa:140|cAa/cTa:46|caA/caT:14|...</td>\n",
       "      <td>1:115256529_252|1:115256530_143|1:115256528_27</td>\n",
       "      <td>skcm:787:186|thpa:486:43|mm:275:27|thpd:58:18|...</td>\n",
       "      <td>skin:974:187|thyroid:618:71|blood:890:37|bowel...</td>\n",
       "      <td>Q</td>\n",
       "      <td>skin</td>\n",
       "      <td>skin:88|thyroid:54|blood:15|bowel:8|testis:5|b...</td>\n",
       "      <td>NRAS_Q61R</td>\n",
       "      <td>204</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>NRAS</td>\n",
       "      <td>61</td>\n",
       "      <td>422</td>\n",
       "      <td>Q:422</td>\n",
       "      <td>620</td>\n",
       "      <td>K:142</td>\n",
       "      <td>cAa/cGa:203|Caa/Aaa:140|cAa/cTa:46|caA/caT:14|...</td>\n",
       "      <td>1:115256529_252|1:115256530_143|1:115256528_27</td>\n",
       "      <td>skcm:787:186|thpa:486:43|mm:275:27|thpd:58:18|...</td>\n",
       "      <td>skin:974:187|thyroid:618:71|blood:890:37|bowel...</td>\n",
       "      <td>Q</td>\n",
       "      <td>skin</td>\n",
       "      <td>skin:62|bowel:18|thyroid:17|blood:12|softtissu...</td>\n",
       "      <td>NRAS_Q61K</td>\n",
       "      <td>142</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>NRAS</td>\n",
       "      <td>61</td>\n",
       "      <td>422</td>\n",
       "      <td>Q:422</td>\n",
       "      <td>620</td>\n",
       "      <td>L:46</td>\n",
       "      <td>cAa/cGa:203|Caa/Aaa:140|cAa/cTa:46|caA/caT:14|...</td>\n",
       "      <td>1:115256529_252|1:115256530_143|1:115256528_27</td>\n",
       "      <td>skcm:787:186|thpa:486:43|mm:275:27|thpd:58:18|...</td>\n",
       "      <td>skin:974:187|thyroid:618:71|blood:890:37|bowel...</td>\n",
       "      <td>Q</td>\n",
       "      <td>skin</td>\n",
       "      <td>skin:24|bowel:7|lung:6|blood:2|cnsbrain:2|unk:...</td>\n",
       "      <td>NRAS_Q61L</td>\n",
       "      <td>46</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>NRAS</td>\n",
       "      <td>61</td>\n",
       "      <td>422</td>\n",
       "      <td>Q:422</td>\n",
       "      <td>620</td>\n",
       "      <td>H:27</td>\n",
       "      <td>cAa/cGa:203|Caa/Aaa:140|cAa/cTa:46|caA/caT:14|...</td>\n",
       "      <td>1:115256529_252|1:115256530_143|1:115256528_27</td>\n",
       "      <td>skcm:787:186|thpa:486:43|mm:275:27|thpd:58:18|...</td>\n",
       "      <td>skin:974:187|thyroid:618:71|blood:890:37|bowel...</td>\n",
       "      <td>Q</td>\n",
       "      <td>skin</td>\n",
       "      <td>skin:12|blood:7|bowel:2|lung:2|testis:2|softti...</td>\n",
       "      <td>NRAS_Q61H</td>\n",
       "      <td>27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>NRAS</td>\n",
       "      <td>61</td>\n",
       "      <td>422</td>\n",
       "      <td>Q:422</td>\n",
       "      <td>620</td>\n",
       "      <td>P:2</td>\n",
       "      <td>cAa/cGa:203|Caa/Aaa:140|cAa/cTa:46|caA/caT:14|...</td>\n",
       "      <td>1:115256529_252|1:115256530_143|1:115256528_27</td>\n",
       "      <td>skcm:787:186|thpa:486:43|mm:275:27|thpd:58:18|...</td>\n",
       "      <td>skin:974:187|thyroid:618:71|blood:890:37|bowel...</td>\n",
       "      <td>Q</td>\n",
       "      <td>skin</td>\n",
       "      <td>blood:1|testis:1</td>\n",
       "      <td>NRAS_Q61P</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2999</th>\n",
       "      <td>FBXW7</td>\n",
       "      <td>385</td>\n",
       "      <td>4</td>\n",
       "      <td>T:4</td>\n",
       "      <td>795</td>\n",
       "      <td>K:2</td>\n",
       "      <td>aCa/aAa:2|Aca/Cca:1|aCa/aTa:1</td>\n",
       "      <td>4:153250906_3|4:153250907_1</td>\n",
       "      <td>esca:556:1|all:92:1|blca:852:1|ucs:117:1</td>\n",
       "      <td>esophagusstomach:1407:1|blood:890:1|bladder:95...</td>\n",
       "      <td>T</td>\n",
       "      <td>esophagusstomach</td>\n",
       "      <td>bladder:1|blood:1</td>\n",
       "      <td>FBXW7_T385K</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3000</th>\n",
       "      <td>FBXW7</td>\n",
       "      <td>385</td>\n",
       "      <td>4</td>\n",
       "      <td>T:4</td>\n",
       "      <td>795</td>\n",
       "      <td>I:1</td>\n",
       "      <td>aCa/aAa:2|Aca/Cca:1|aCa/aTa:1</td>\n",
       "      <td>4:153250906_3|4:153250907_1</td>\n",
       "      <td>esca:556:1|all:92:1|blca:852:1|ucs:117:1</td>\n",
       "      <td>esophagusstomach:1407:1|blood:890:1|bladder:95...</td>\n",
       "      <td>T</td>\n",
       "      <td>esophagusstomach</td>\n",
       "      <td>uterus:1</td>\n",
       "      <td>FBXW7_T385I</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3001</th>\n",
       "      <td>FBXW7</td>\n",
       "      <td>385</td>\n",
       "      <td>4</td>\n",
       "      <td>T:4</td>\n",
       "      <td>795</td>\n",
       "      <td>P:1</td>\n",
       "      <td>aCa/aAa:2|Aca/Cca:1|aCa/aTa:1</td>\n",
       "      <td>4:153250906_3|4:153250907_1</td>\n",
       "      <td>esca:556:1|all:92:1|blca:852:1|ucs:117:1</td>\n",
       "      <td>esophagusstomach:1407:1|blood:890:1|bladder:95...</td>\n",
       "      <td>T</td>\n",
       "      <td>esophagusstomach</td>\n",
       "      <td>esophagusstomach:1</td>\n",
       "      <td>FBXW7_T385P</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3002</th>\n",
       "      <td>FBXW7</td>\n",
       "      <td>393</td>\n",
       "      <td>7</td>\n",
       "      <td>R:7</td>\n",
       "      <td>795</td>\n",
       "      <td>*:4</td>\n",
       "      <td>Cga/Tga:4|cGa/cAa:3</td>\n",
       "      <td>4:153250883_4|4:153250882_3</td>\n",
       "      <td>paad:932:1|coad:712:1|coadread:683:1|hnsc:643:...</td>\n",
       "      <td>bowel:1782:2|pancreas:1059:1|headandneck:988:1...</td>\n",
       "      <td>R</td>\n",
       "      <td>esophagusstomach</td>\n",
       "      <td>bowel:1|esophagusstomach:1|headandneck:1|uterus:1</td>\n",
       "      <td>FBXW7_R393*</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3003</th>\n",
       "      <td>FBXW7</td>\n",
       "      <td>393</td>\n",
       "      <td>7</td>\n",
       "      <td>R:7</td>\n",
       "      <td>795</td>\n",
       "      <td>Q:3</td>\n",
       "      <td>Cga/Tga:4|cGa/cAa:3</td>\n",
       "      <td>4:153250883_4|4:153250882_3</td>\n",
       "      <td>paad:932:1|coad:712:1|coadread:683:1|hnsc:643:...</td>\n",
       "      <td>bowel:1782:2|pancreas:1059:1|headandneck:988:1...</td>\n",
       "      <td>R</td>\n",
       "      <td>esophagusstomach</td>\n",
       "      <td>biliarytract:1|bowel:1|pancreas:1</td>\n",
       "      <td>FBXW7_R393Q</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3004 rows  15 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     Hugo_Symbol Amino_Acid_Position  Mutation_Count Reference_Amino_Acid  \\\n",
       "0           NRAS                  61             422                Q:422   \n",
       "1           NRAS                  61             422                Q:422   \n",
       "2           NRAS                  61             422                Q:422   \n",
       "3           NRAS                  61             422                Q:422   \n",
       "4           NRAS                  61             422                Q:422   \n",
       "...          ...                 ...             ...                  ...   \n",
       "2999       FBXW7                 385               4                  T:4   \n",
       "3000       FBXW7                 385               4                  T:4   \n",
       "3001       FBXW7                 385               4                  T:4   \n",
       "3002       FBXW7                 393               7                  R:7   \n",
       "3003       FBXW7                 393               7                  R:7   \n",
       "\n",
       "      Total_Mutations_in_Gene Variant_Amino_Acid  \\\n",
       "0                         620              R:204   \n",
       "1                         620              K:142   \n",
       "2                         620               L:46   \n",
       "3                         620               H:27   \n",
       "4                         620                P:2   \n",
       "...                       ...                ...   \n",
       "2999                      795                K:2   \n",
       "3000                      795                I:1   \n",
       "3001                      795                P:1   \n",
       "3002                      795                *:4   \n",
       "3003                      795                Q:3   \n",
       "\n",
       "                                           Codon_Change  \\\n",
       "0     cAa/cGa:203|Caa/Aaa:140|cAa/cTa:46|caA/caT:14|...   \n",
       "1     cAa/cGa:203|Caa/Aaa:140|cAa/cTa:46|caA/caT:14|...   \n",
       "2     cAa/cGa:203|Caa/Aaa:140|cAa/cTa:46|caA/caT:14|...   \n",
       "3     cAa/cGa:203|Caa/Aaa:140|cAa/cTa:46|caA/caT:14|...   \n",
       "4     cAa/cGa:203|Caa/Aaa:140|cAa/cTa:46|caA/caT:14|...   \n",
       "...                                                 ...   \n",
       "2999                      aCa/aAa:2|Aca/Cca:1|aCa/aTa:1   \n",
       "3000                      aCa/aAa:2|Aca/Cca:1|aCa/aTa:1   \n",
       "3001                      aCa/aAa:2|Aca/Cca:1|aCa/aTa:1   \n",
       "3002                                Cga/Tga:4|cGa/cAa:3   \n",
       "3003                                Cga/Tga:4|cGa/cAa:3   \n",
       "\n",
       "                                    Genomic_Position  \\\n",
       "0     1:115256529_252|1:115256530_143|1:115256528_27   \n",
       "1     1:115256529_252|1:115256530_143|1:115256528_27   \n",
       "2     1:115256529_252|1:115256530_143|1:115256528_27   \n",
       "3     1:115256529_252|1:115256530_143|1:115256528_27   \n",
       "4     1:115256529_252|1:115256530_143|1:115256528_27   \n",
       "...                                              ...   \n",
       "2999                     4:153250906_3|4:153250907_1   \n",
       "3000                     4:153250906_3|4:153250907_1   \n",
       "3001                     4:153250906_3|4:153250907_1   \n",
       "3002                     4:153250883_4|4:153250882_3   \n",
       "3003                     4:153250883_4|4:153250882_3   \n",
       "\n",
       "                                  Detailed_Cancer_Types  \\\n",
       "0     skcm:787:186|thpa:486:43|mm:275:27|thpd:58:18|...   \n",
       "1     skcm:787:186|thpa:486:43|mm:275:27|thpd:58:18|...   \n",
       "2     skcm:787:186|thpa:486:43|mm:275:27|thpd:58:18|...   \n",
       "3     skcm:787:186|thpa:486:43|mm:275:27|thpd:58:18|...   \n",
       "4     skcm:787:186|thpa:486:43|mm:275:27|thpd:58:18|...   \n",
       "...                                                 ...   \n",
       "2999           esca:556:1|all:92:1|blca:852:1|ucs:117:1   \n",
       "3000           esca:556:1|all:92:1|blca:852:1|ucs:117:1   \n",
       "3001           esca:556:1|all:92:1|blca:852:1|ucs:117:1   \n",
       "3002  paad:932:1|coad:712:1|coadread:683:1|hnsc:643:...   \n",
       "3003  paad:932:1|coad:712:1|coadread:683:1|hnsc:643:...   \n",
       "\n",
       "                                            Organ_Types ref                ct  \\\n",
       "0     skin:974:187|thyroid:618:71|blood:890:37|bowel...   Q              skin   \n",
       "1     skin:974:187|thyroid:618:71|blood:890:37|bowel...   Q              skin   \n",
       "2     skin:974:187|thyroid:618:71|blood:890:37|bowel...   Q              skin   \n",
       "3     skin:974:187|thyroid:618:71|blood:890:37|bowel...   Q              skin   \n",
       "4     skin:974:187|thyroid:618:71|blood:890:37|bowel...   Q              skin   \n",
       "...                                                 ...  ..               ...   \n",
       "2999  esophagusstomach:1407:1|blood:890:1|bladder:95...   T  esophagusstomach   \n",
       "3000  esophagusstomach:1407:1|blood:890:1|bladder:95...   T  esophagusstomach   \n",
       "3001  esophagusstomach:1407:1|blood:890:1|bladder:95...   T  esophagusstomach   \n",
       "3002  bowel:1782:2|pancreas:1059:1|headandneck:988:1...   R  esophagusstomach   \n",
       "3003  bowel:1782:2|pancreas:1059:1|headandneck:988:1...   R  esophagusstomach   \n",
       "\n",
       "                                                Samples           id  \\\n",
       "0     skin:88|thyroid:54|blood:15|bowel:8|testis:5|b...    NRAS_Q61R   \n",
       "1     skin:62|bowel:18|thyroid:17|blood:12|softtissu...    NRAS_Q61K   \n",
       "2     skin:24|bowel:7|lung:6|blood:2|cnsbrain:2|unk:...    NRAS_Q61L   \n",
       "3     skin:12|blood:7|bowel:2|lung:2|testis:2|softti...    NRAS_Q61H   \n",
       "4                                      blood:1|testis:1    NRAS_Q61P   \n",
       "...                                                 ...          ...   \n",
       "2999                                  bladder:1|blood:1  FBXW7_T385K   \n",
       "3000                                           uterus:1  FBXW7_T385I   \n",
       "3001                                 esophagusstomach:1  FBXW7_T385P   \n",
       "3002  bowel:1|esophagusstomach:1|headandneck:1|uterus:1  FBXW7_R393*   \n",
       "3003                  biliarytract:1|bowel:1|pancreas:1  FBXW7_R393Q   \n",
       "\n",
       "      total_variant_count  \n",
       "0                     204  \n",
       "1                     142  \n",
       "2                      46  \n",
       "3                      27  \n",
       "4                       2  \n",
       "...                   ...  \n",
       "2999                    2  \n",
       "3000                    1  \n",
       "3001                    1  \n",
       "3002                    4  \n",
       "3003                    3  \n",
       "\n",
       "[3004 rows x 15 columns]"
      ]
     },
     "execution_count": 52,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cancer_hotspots_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [],
   "source": [
    "merged_df = sample_df.merge(cancer_hotspots_df, left_on=[\"id\"], right_on=[\"id\"], how=\"left\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {},
   "outputs": [],
   "source": [
    "merged_df = merged_df.fillna(0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "metadata": {},
   "outputs": [],
   "source": [
    "def cancer_hotspot_rules(row):\n",
    "    if int(row[\"total_variant_count\"]) >= 10:\n",
    "        if int(row[\"Mutation_Count\"]) >= 50:\n",
    "            return \"OS3\"\n",
    "        else:\n",
    "            return \"OM3\"\n",
    "    elif int(row[\"total_variant_count\"]) > 0:\n",
    "        return \"OP3\"\n",
    "    else:\n",
    "        return False\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0</th>\n",
       "      <th>Chr</th>\n",
       "      <th>Start</th>\n",
       "      <th>End</th>\n",
       "      <th>Ref</th>\n",
       "      <th>Alt</th>\n",
       "      <th>Func.refGene</th>\n",
       "      <th>Gene.refGene</th>\n",
       "      <th>GeneDetail.refGene</th>\n",
       "      <th>ExonicFunc.refGene</th>\n",
       "      <th>...</th>\n",
       "      <th>Total_Mutations_in_Gene</th>\n",
       "      <th>Variant_Amino_Acid</th>\n",
       "      <th>Codon_Change</th>\n",
       "      <th>Genomic_Position</th>\n",
       "      <th>Detailed_Cancer_Types</th>\n",
       "      <th>Organ_Types</th>\n",
       "      <th>ref</th>\n",
       "      <th>ct</th>\n",
       "      <th>Samples</th>\n",
       "      <th>total_variant_count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>12</td>\n",
       "      <td>25209871</td>\n",
       "      <td>25209871</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>12</td>\n",
       "      <td>25227330</td>\n",
       "      <td>25227330</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>12</td>\n",
       "      <td>25227343</td>\n",
       "      <td>25227343</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>2885.0</td>\n",
       "      <td>K:20</td>\n",
       "      <td>caA/caC:83|cAa/cGa:36|caA/caT:25|cAa/cTa:16|Ca...</td>\n",
       "      <td>12:25380275_115|12:25380276_54|12:25380277_21</td>\n",
       "      <td>paad:932:61|luad:2057:24|mm:275:15|coad:712:14...</td>\n",
       "      <td>pancreas:1059:64|bowel:1782:31|lung:2761:28|bl...</td>\n",
       "      <td>Q</td>\n",
       "      <td>pancreas</td>\n",
       "      <td>bowel:9|pancreas:3|thyroid:2|biliarytract:1|bl...</td>\n",
       "      <td>20.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>12</td>\n",
       "      <td>25245350</td>\n",
       "      <td>25245350</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>2885.0</td>\n",
       "      <td>A:134</td>\n",
       "      <td>gGt/gAt:757|gGt/gTt:657|Ggt/Tgt:380|Ggt/Cgt:16...</td>\n",
       "      <td>12:25398284_1557|12:25398285_617|12:25398283_1</td>\n",
       "      <td>paad:932:728|luad:2057:535|coad:712:212|coadre...</td>\n",
       "      <td>pancreas:1059:745|lung:2761:570|bowel:1782:500...</td>\n",
       "      <td>G</td>\n",
       "      <td>bowel</td>\n",
       "      <td>lung:58|bowel:36|uterus:11|unk:5|breast:4|blad...</td>\n",
       "      <td>134.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>13</td>\n",
       "      <td>28018483</td>\n",
       "      <td>28018483</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>5</td>\n",
       "      <td>13</td>\n",
       "      <td>28018503</td>\n",
       "      <td>28018503</td>\n",
       "      <td>A</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>529.0</td>\n",
       "      <td>E:3</td>\n",
       "      <td>Gat/Tat:11|gaT/gaA:2|Gat/Cat:2|gAt/gTt:1|gaT/g...</td>\n",
       "      <td>13:28592642_13|13:28592640_3|13:28592641_1</td>\n",
       "      <td>aml:198:16|all:92:1</td>\n",
       "      <td>blood:890:17</td>\n",
       "      <td>D</td>\n",
       "      <td>blood</td>\n",
       "      <td>blood:3</td>\n",
       "      <td>3.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>6</td>\n",
       "      <td>13</td>\n",
       "      <td>28018504</td>\n",
       "      <td>28018505</td>\n",
       "      <td>TC</td>\n",
       "      <td>AT</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonframeshift substitution</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>7</td>\n",
       "      <td>13</td>\n",
       "      <td>28018504</td>\n",
       "      <td>28018504</td>\n",
       "      <td>T</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>529.0</td>\n",
       "      <td>V:1</td>\n",
       "      <td>Gat/Tat:11|gaT/gaA:2|Gat/Cat:2|gAt/gTt:1|gaT/g...</td>\n",
       "      <td>13:28592642_13|13:28592640_3|13:28592641_1</td>\n",
       "      <td>aml:198:16|all:92:1</td>\n",
       "      <td>blood:890:17</td>\n",
       "      <td>D</td>\n",
       "      <td>blood</td>\n",
       "      <td>blood:1</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>8</td>\n",
       "      <td>13</td>\n",
       "      <td>28018505</td>\n",
       "      <td>28018505</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>529.0</td>\n",
       "      <td>H:2</td>\n",
       "      <td>Gat/Tat:11|gaT/gaA:2|Gat/Cat:2|gAt/gTt:1|gaT/g...</td>\n",
       "      <td>13:28592642_13|13:28592640_3|13:28592641_1</td>\n",
       "      <td>aml:198:16|all:92:1</td>\n",
       "      <td>blood:890:17</td>\n",
       "      <td>D</td>\n",
       "      <td>blood</td>\n",
       "      <td>blood:2</td>\n",
       "      <td>2.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>9</td>\n",
       "      <td>13</td>\n",
       "      <td>28018505</td>\n",
       "      <td>28018505</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>529.0</td>\n",
       "      <td>Y:11</td>\n",
       "      <td>Gat/Tat:11|gaT/gaA:2|Gat/Cat:2|gAt/gTt:1|gaT/g...</td>\n",
       "      <td>13:28592642_13|13:28592640_3|13:28592641_1</td>\n",
       "      <td>aml:198:16|all:92:1</td>\n",
       "      <td>blood:890:17</td>\n",
       "      <td>D</td>\n",
       "      <td>blood</td>\n",
       "      <td>blood:11</td>\n",
       "      <td>11.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10</td>\n",
       "      <td>13</td>\n",
       "      <td>28027217</td>\n",
       "      <td>28027217</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>11</td>\n",
       "      <td>13</td>\n",
       "      <td>28027218</td>\n",
       "      <td>28027218</td>\n",
       "      <td>A</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>12</td>\n",
       "      <td>13</td>\n",
       "      <td>28027222</td>\n",
       "      <td>28027222</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>13</td>\n",
       "      <td>13</td>\n",
       "      <td>28028203</td>\n",
       "      <td>28028203</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>FLT3</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>14</td>\n",
       "      <td>2</td>\n",
       "      <td>208242060</td>\n",
       "      <td>208242060</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>15</td>\n",
       "      <td>2</td>\n",
       "      <td>208243466</td>\n",
       "      <td>208243466</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>16</td>\n",
       "      <td>2</td>\n",
       "      <td>208243593</td>\n",
       "      <td>208243593</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>17</td>\n",
       "      <td>2</td>\n",
       "      <td>208248388</td>\n",
       "      <td>208248388</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>916.0</td>\n",
       "      <td>H:570</td>\n",
       "      <td>cGt/cAt:570|Cgt/Tgt:139|Cgt/Ggt:28|Cgt/Agt:16|...</td>\n",
       "      <td>2:209113112_583|2:209113113_183</td>\n",
       "      <td>lggnos:544:424|aastr:82:39|gbm:688:38|aodg:43:...</td>\n",
       "      <td>cnsbrain:2270:612|biliarytract:358:57|skin:974...</td>\n",
       "      <td>R</td>\n",
       "      <td>cnsbrain</td>\n",
       "      <td>cnsbrain:553|prostate:6|blood:5|headandneck:2|...</td>\n",
       "      <td>570.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>18</td>\n",
       "      <td>2</td>\n",
       "      <td>208248389</td>\n",
       "      <td>208248389</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>916.0</td>\n",
       "      <td>S:16</td>\n",
       "      <td>cGt/cAt:570|Cgt/Tgt:139|Cgt/Ggt:28|Cgt/Agt:16|...</td>\n",
       "      <td>2:209113112_583|2:209113113_183</td>\n",
       "      <td>lggnos:544:424|aastr:82:39|gbm:688:38|aodg:43:...</td>\n",
       "      <td>cnsbrain:2270:612|biliarytract:358:57|skin:974...</td>\n",
       "      <td>R</td>\n",
       "      <td>cnsbrain</td>\n",
       "      <td>cnsbrain:13|blood:2|biliarytract:1</td>\n",
       "      <td>16.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>19</td>\n",
       "      <td>2</td>\n",
       "      <td>208248389</td>\n",
       "      <td>208248389</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>916.0</td>\n",
       "      <td>C:139</td>\n",
       "      <td>cGt/cAt:570|Cgt/Tgt:139|Cgt/Ggt:28|Cgt/Agt:16|...</td>\n",
       "      <td>2:209113112_583|2:209113113_183</td>\n",
       "      <td>lggnos:544:424|aastr:82:39|gbm:688:38|aodg:43:...</td>\n",
       "      <td>cnsbrain:2270:612|biliarytract:358:57|skin:974...</td>\n",
       "      <td>R</td>\n",
       "      <td>cnsbrain</td>\n",
       "      <td>biliarytract:48|cnsbrain:26|skin:23|blood:15|l...</td>\n",
       "      <td>139.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>20</td>\n",
       "      <td>2</td>\n",
       "      <td>208248572</td>\n",
       "      <td>208248572</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>21</td>\n",
       "      <td>2</td>\n",
       "      <td>208248664</td>\n",
       "      <td>208248664</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>splicing</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>NM_001282387:exon4:c.123-4C&gt;T;NM_005896:exon4:...</td>\n",
       "      <td>.</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>22</td>\n",
       "      <td>2</td>\n",
       "      <td>208251458</td>\n",
       "      <td>208251458</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>IDH1</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>23</td>\n",
       "      <td>3</td>\n",
       "      <td>179198826</td>\n",
       "      <td>179198826</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>startloss</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>24</td>\n",
       "      <td>3</td>\n",
       "      <td>179209622</td>\n",
       "      <td>179209622</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>25</td>\n",
       "      <td>3</td>\n",
       "      <td>179210186</td>\n",
       "      <td>179210186</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>26</td>\n",
       "      <td>3</td>\n",
       "      <td>179210291</td>\n",
       "      <td>179210291</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>2933.0</td>\n",
       "      <td>K:31</td>\n",
       "      <td>Gaa/Aaa:31|Gaa/Caa:10|gaA/gaC:1</td>\n",
       "      <td>3:178928079_41|3:178928081_1</td>\n",
       "      <td>brca:1345:8|blca:852:7|idc:870:5|lggnos:544:3|...</td>\n",
       "      <td>breast:2561:16|bladder:958:8|cnsbrain:2270:6|l...</td>\n",
       "      <td>E</td>\n",
       "      <td>breast</td>\n",
       "      <td>breast:13|cnsbrain:5|bladder:4|esophagusstomac...</td>\n",
       "      <td>31.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>27</td>\n",
       "      <td>3</td>\n",
       "      <td>179218304</td>\n",
       "      <td>179218304</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>2933.0</td>\n",
       "      <td>A:12</td>\n",
       "      <td>Gag/Aag:575|gAg/gGg:21|Gag/Cag:20|gAg/gCg:12|g...</td>\n",
       "      <td>3:178936091_595|3:178936092_33|3:178936093_5</td>\n",
       "      <td>brca:1345:82|blca:852:79|idc:870:59|coadread:6...</td>\n",
       "      <td>breast:2561:174|bowel:1782:97|bladder:958:90|l...</td>\n",
       "      <td>E</td>\n",
       "      <td>breast</td>\n",
       "      <td>breast:4|bowel:2|cnsbrain:2|uterus:2|bladder:1...</td>\n",
       "      <td>12.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>28</td>\n",
       "      <td>3</td>\n",
       "      <td>179224091</td>\n",
       "      <td>179224091</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>29</td>\n",
       "      <td>3</td>\n",
       "      <td>179234297</td>\n",
       "      <td>179234297</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>2933.0</td>\n",
       "      <td>R:537</td>\n",
       "      <td>cAt/cGt:536|cAt/cTt:85|Cat/Tat:18|caT/caG:4|ca...</td>\n",
       "      <td>3:178952085_622|3:178952084_18|3:178952086_7</td>\n",
       "      <td>brca:1345:174|idc:870:119|ilc:184:40|uec:339:3...</td>\n",
       "      <td>breast:2561:361|bowel:1782:54|uterus:618:51|es...</td>\n",
       "      <td>H</td>\n",
       "      <td>breast</td>\n",
       "      <td>breast:320|bowel:45|uterus:33|cnsbrain:22|esop...</td>\n",
       "      <td>537.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>30</td>\n",
       "      <td>3</td>\n",
       "      <td>179234298</td>\n",
       "      <td>179234298</td>\n",
       "      <td>T</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>2933.0</td>\n",
       "      <td>Q:7</td>\n",
       "      <td>cAt/cGt:536|cAt/cTt:85|Cat/Tat:18|caT/caG:4|ca...</td>\n",
       "      <td>3:178952085_622|3:178952084_18|3:178952086_7</td>\n",
       "      <td>brca:1345:174|idc:870:119|ilc:184:40|uec:339:3...</td>\n",
       "      <td>breast:2561:361|bowel:1782:54|uterus:618:51|es...</td>\n",
       "      <td>H</td>\n",
       "      <td>breast</td>\n",
       "      <td>bowel:2|breast:1|cervix:1|headandneck:1|kidney...</td>\n",
       "      <td>7.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>31</td>\n",
       "      <td>3</td>\n",
       "      <td>179234350</td>\n",
       "      <td>179234350</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>2933.0</td>\n",
       "      <td>Y:4</td>\n",
       "      <td>Cat/Tat:4|cAt/cTt:1</td>\n",
       "      <td>3:178952138_4|3:178952139_1</td>\n",
       "      <td>luad:2057:1|coad:712:1|ccrcc:771:1|blca:852:1|...</td>\n",
       "      <td>lung:2761:1|bowel:1782:1|kidney:1304:1|bladder...</td>\n",
       "      <td>H</td>\n",
       "      <td>lung</td>\n",
       "      <td>bladder:1|bowel:1|kidney:1|lung:1</td>\n",
       "      <td>4.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>32</td>\n",
       "      <td>3</td>\n",
       "      <td>179234360</td>\n",
       "      <td>179234360</td>\n",
       "      <td>-</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>.</td>\n",
       "      <td>frameshift insertion</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>33</td>\n",
       "      <td>5</td>\n",
       "      <td>1282455</td>\n",
       "      <td>1282455</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>TERT</td>\n",
       "      <td>.</td>\n",
       "      <td>stopgain</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>34</td>\n",
       "      <td>5</td>\n",
       "      <td>1294198</td>\n",
       "      <td>1294198</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>exonic</td>\n",
       "      <td>TERT</td>\n",
       "      <td>.</td>\n",
       "      <td>stopgain</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>35</td>\n",
       "      <td>7</td>\n",
       "      <td>140753334</td>\n",
       "      <td>140753334</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>1450.0</td>\n",
       "      <td>E:32</td>\n",
       "      <td>Aaa/Gaa:32|aaA/aaT:4|aaATct/aaTAct:1|aAa/aCa:1</td>\n",
       "      <td>7:140453134_32|7:140453132_4|7:140453133_1|7:1...</td>\n",
       "      <td>skcm:787:8|luad:2057:6|prad:1366:6|dlbcl:246:3...</td>\n",
       "      <td>skin:974:8|lung:2761:6|prostate:1379:6|lymph:3...</td>\n",
       "      <td>K</td>\n",
       "      <td>skin</td>\n",
       "      <td>skin:7|prostate:6|lung:5|lymph:3|bowel:2|thyro...</td>\n",
       "      <td>32.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>36</td>\n",
       "      <td>7</td>\n",
       "      <td>140753336</td>\n",
       "      <td>140753337</td>\n",
       "      <td>AC</td>\n",
       "      <td>TT</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonframeshift substitution</td>\n",
       "      <td>...</td>\n",
       "      <td>1450.0</td>\n",
       "      <td>K:24</td>\n",
       "      <td>gTg/gAg:831|Gtg/Atg:29|GTg/AAg:24|gtG/gtA:4|GT...</td>\n",
       "      <td>7:140453136_862|7:140453137_29|7:140453135_6</td>\n",
       "      <td>skcm:787:355|thpa:486:293|coad:712:54|coadread...</td>\n",
       "      <td>skin:974:357|thyroid:618:316|bowel:1782:113|lu...</td>\n",
       "      <td>V</td>\n",
       "      <td>skin</td>\n",
       "      <td>skin:20|unk:2|esophagusstomach:1|headandneck:1</td>\n",
       "      <td>24.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>37</td>\n",
       "      <td>7</td>\n",
       "      <td>140753355</td>\n",
       "      <td>140753355</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>1450.0</td>\n",
       "      <td>N:24</td>\n",
       "      <td>Gat/Aat:24|gAt/gGt:21|Gat/Tat:2|gAt/gTt:2|gaT/...</td>\n",
       "      <td>7:140453155_27|7:140453154_23|7:140453153_1</td>\n",
       "      <td>luad:2057:10|coad:712:8|blca:852:5|skcm:787:5|...</td>\n",
       "      <td>bowel:1782:12|lung:2761:10|bladder:958:6|skin:...</td>\n",
       "      <td>D</td>\n",
       "      <td>bowel</td>\n",
       "      <td>bowel:5|skin:5|lung:4|bladder:3|unk:3|biliaryt...</td>\n",
       "      <td>24.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>38</td>\n",
       "      <td>7</td>\n",
       "      <td>140778037</td>\n",
       "      <td>140778051</td>\n",
       "      <td>TAGGTGCTGTCACAT</td>\n",
       "      <td>-</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonframeshift deletion</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>39</td>\n",
       "      <td>7</td>\n",
       "      <td>140781602</td>\n",
       "      <td>140781602</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>1450.0</td>\n",
       "      <td>E:4</td>\n",
       "      <td>gGa/gCa:34|gGa/gTa:11|Gga/Aga:8|gGa/gAa:4|GGa/...</td>\n",
       "      <td>7:140481402_51|7:140481403_9|7:140481401_1</td>\n",
       "      <td>luad:2057:18|skcm:787:12|coad:712:4|mm:275:3|l...</td>\n",
       "      <td>lung:2761:23|skin:974:13|bowel:1782:6|blood:89...</td>\n",
       "      <td>G</td>\n",
       "      <td>lung</td>\n",
       "      <td>skin:3|bowel:1</td>\n",
       "      <td>4.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>40</td>\n",
       "      <td>7</td>\n",
       "      <td>140781602</td>\n",
       "      <td>140781602</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>1450.0</td>\n",
       "      <td>A:34</td>\n",
       "      <td>gGa/gCa:34|gGa/gTa:11|Gga/Aga:8|gGa/gAa:4|GGa/...</td>\n",
       "      <td>7:140481402_51|7:140481403_9|7:140481401_1</td>\n",
       "      <td>luad:2057:18|skcm:787:12|coad:712:4|mm:275:3|l...</td>\n",
       "      <td>lung:2761:23|skin:974:13|bowel:1782:6|blood:89...</td>\n",
       "      <td>G</td>\n",
       "      <td>lung</td>\n",
       "      <td>lung:15|bowel:5|blood:2|prostate:2|skin:2|adre...</td>\n",
       "      <td>34.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>41</td>\n",
       "      <td>7</td>\n",
       "      <td>140781675</td>\n",
       "      <td>140781675</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>42</td>\n",
       "      <td>7</td>\n",
       "      <td>140783127</td>\n",
       "      <td>140783127</td>\n",
       "      <td>G</td>\n",
       "      <td>-</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>frameshift deletion</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>43</td>\n",
       "      <td>7</td>\n",
       "      <td>140924612</td>\n",
       "      <td>140924612</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>44</td>\n",
       "      <td>7</td>\n",
       "      <td>140924663</td>\n",
       "      <td>140924663</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>exonic</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>.</td>\n",
       "      <td>nonsynonymous SNV</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>45 rows  131 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "    Unnamed: 0  Chr      Start        End              Ref Alt Func.refGene  \\\n",
       "0            0   12   25209871   25209871                C   T       exonic   \n",
       "1            1   12   25227330   25227330                C   A       exonic   \n",
       "2            2   12   25227343   25227343                G   T       exonic   \n",
       "3            3   12   25245350   25245350                C   G       exonic   \n",
       "4            4   13   28018483   28018483                T   C       exonic   \n",
       "5            5   13   28018503   28018503                A   T       exonic   \n",
       "6            6   13   28018504   28018505               TC  AT       exonic   \n",
       "7            7   13   28018504   28018504                T   A       exonic   \n",
       "8            8   13   28018505   28018505                C   G       exonic   \n",
       "9            9   13   28018505   28018505                C   A       exonic   \n",
       "10          10   13   28027217   28027217                T   C       exonic   \n",
       "11          11   13   28027218   28027218                A   T       exonic   \n",
       "12          12   13   28027222   28027222                A   C       exonic   \n",
       "13          13   13   28028203   28028203                G   T       exonic   \n",
       "14          14    2  208242060  208242060                C   T       exonic   \n",
       "15          15    2  208243466  208243466                T   C       exonic   \n",
       "16          16    2  208243593  208243593                C   T       exonic   \n",
       "17          17    2  208248388  208248388                C   T       exonic   \n",
       "18          18    2  208248389  208248389                G   T       exonic   \n",
       "19          19    2  208248389  208248389                G   A       exonic   \n",
       "20          20    2  208248572  208248572                C   T       exonic   \n",
       "21          21    2  208248664  208248664                G   A     splicing   \n",
       "22          22    2  208251458  208251458                A   C       exonic   \n",
       "23          23    3  179198826  179198826                A   G       exonic   \n",
       "24          24    3  179209622  179209622                A   G       exonic   \n",
       "25          25    3  179210186  179210186                G   A       exonic   \n",
       "26          26    3  179210291  179210291                G   A       exonic   \n",
       "27          27    3  179218304  179218304                A   C       exonic   \n",
       "28          28    3  179224091  179224091                A   G       exonic   \n",
       "29          29    3  179234297  179234297                A   G       exonic   \n",
       "30          30    3  179234298  179234298                T   G       exonic   \n",
       "31          31    3  179234350  179234350                C   T       exonic   \n",
       "32          32    3  179234360  179234360                -   A       exonic   \n",
       "33          33    5    1282455    1282455                C   T       exonic   \n",
       "34          34    5    1294198    1294198                G   A       exonic   \n",
       "35          35    7  140753334  140753334                T   C       exonic   \n",
       "36          36    7  140753336  140753337               AC  TT       exonic   \n",
       "37          37    7  140753355  140753355                C   T       exonic   \n",
       "38          38    7  140778037  140778051  TAGGTGCTGTCACAT   -       exonic   \n",
       "39          39    7  140781602  140781602                C   T       exonic   \n",
       "40          40    7  140781602  140781602                C   G       exonic   \n",
       "41          41    7  140781675  140781675                C   G       exonic   \n",
       "42          42    7  140783127  140783127                G   -       exonic   \n",
       "43          43    7  140924612  140924612                G   C       exonic   \n",
       "44          44    7  140924663  140924663                G   C       exonic   \n",
       "\n",
       "   Gene.refGene                                 GeneDetail.refGene  \\\n",
       "0          KRAS                                                  .   \n",
       "1          KRAS                                                  .   \n",
       "2          KRAS                                                  .   \n",
       "3          KRAS                                                  .   \n",
       "4          FLT3                                                  .   \n",
       "5          FLT3                                                  .   \n",
       "6          FLT3                                                  .   \n",
       "7          FLT3                                                  .   \n",
       "8          FLT3                                                  .   \n",
       "9          FLT3                                                  .   \n",
       "10         FLT3                                                  .   \n",
       "11         FLT3                                                  .   \n",
       "12         FLT3                                                  .   \n",
       "13         FLT3                                                  .   \n",
       "14         IDH1                                                  .   \n",
       "15         IDH1                                                  .   \n",
       "16         IDH1                                                  .   \n",
       "17         IDH1                                                  .   \n",
       "18         IDH1                                                  .   \n",
       "19         IDH1                                                  .   \n",
       "20         IDH1                                                  .   \n",
       "21         IDH1  NM_001282387:exon4:c.123-4C>T;NM_005896:exon4:...   \n",
       "22         IDH1                                                  .   \n",
       "23       PIK3CA                                                  .   \n",
       "24       PIK3CA                                                  .   \n",
       "25       PIK3CA                                                  .   \n",
       "26       PIK3CA                                                  .   \n",
       "27       PIK3CA                                                  .   \n",
       "28       PIK3CA                                                  .   \n",
       "29       PIK3CA                                                  .   \n",
       "30       PIK3CA                                                  .   \n",
       "31       PIK3CA                                                  .   \n",
       "32       PIK3CA                                                  .   \n",
       "33         TERT                                                  .   \n",
       "34         TERT                                                  .   \n",
       "35         BRAF                                                  .   \n",
       "36         BRAF                                                  .   \n",
       "37         BRAF                                                  .   \n",
       "38         BRAF                                                  .   \n",
       "39         BRAF                                                  .   \n",
       "40         BRAF                                                  .   \n",
       "41         BRAF                                                  .   \n",
       "42         BRAF                                                  .   \n",
       "43         BRAF                                                  .   \n",
       "44         BRAF                                                  .   \n",
       "\n",
       "            ExonicFunc.refGene  ... Total_Mutations_in_Gene  \\\n",
       "0            nonsynonymous SNV  ...                     0.0   \n",
       "1            nonsynonymous SNV  ...                     0.0   \n",
       "2            nonsynonymous SNV  ...                  2885.0   \n",
       "3            nonsynonymous SNV  ...                  2885.0   \n",
       "4            nonsynonymous SNV  ...                     0.0   \n",
       "5            nonsynonymous SNV  ...                   529.0   \n",
       "6   nonframeshift substitution  ...                     0.0   \n",
       "7            nonsynonymous SNV  ...                   529.0   \n",
       "8            nonsynonymous SNV  ...                   529.0   \n",
       "9            nonsynonymous SNV  ...                   529.0   \n",
       "10           nonsynonymous SNV  ...                     0.0   \n",
       "11           nonsynonymous SNV  ...                     0.0   \n",
       "12           nonsynonymous SNV  ...                     0.0   \n",
       "13           nonsynonymous SNV  ...                     0.0   \n",
       "14           nonsynonymous SNV  ...                     0.0   \n",
       "15           nonsynonymous SNV  ...                     0.0   \n",
       "16           nonsynonymous SNV  ...                     0.0   \n",
       "17           nonsynonymous SNV  ...                   916.0   \n",
       "18           nonsynonymous SNV  ...                   916.0   \n",
       "19           nonsynonymous SNV  ...                   916.0   \n",
       "20           nonsynonymous SNV  ...                     0.0   \n",
       "21                           .  ...                     0.0   \n",
       "22           nonsynonymous SNV  ...                     0.0   \n",
       "23                   startloss  ...                     0.0   \n",
       "24           nonsynonymous SNV  ...                     0.0   \n",
       "25           nonsynonymous SNV  ...                     0.0   \n",
       "26           nonsynonymous SNV  ...                  2933.0   \n",
       "27           nonsynonymous SNV  ...                  2933.0   \n",
       "28           nonsynonymous SNV  ...                     0.0   \n",
       "29           nonsynonymous SNV  ...                  2933.0   \n",
       "30           nonsynonymous SNV  ...                  2933.0   \n",
       "31           nonsynonymous SNV  ...                  2933.0   \n",
       "32        frameshift insertion  ...                     0.0   \n",
       "33                    stopgain  ...                     0.0   \n",
       "34                    stopgain  ...                     0.0   \n",
       "35           nonsynonymous SNV  ...                  1450.0   \n",
       "36  nonframeshift substitution  ...                  1450.0   \n",
       "37           nonsynonymous SNV  ...                  1450.0   \n",
       "38      nonframeshift deletion  ...                     0.0   \n",
       "39           nonsynonymous SNV  ...                  1450.0   \n",
       "40           nonsynonymous SNV  ...                  1450.0   \n",
       "41           nonsynonymous SNV  ...                     0.0   \n",
       "42         frameshift deletion  ...                     0.0   \n",
       "43           nonsynonymous SNV  ...                     0.0   \n",
       "44           nonsynonymous SNV  ...                     0.0   \n",
       "\n",
       "   Variant_Amino_Acid                                       Codon_Change  \\\n",
       "0                   0                                                  0   \n",
       "1                   0                                                  0   \n",
       "2                K:20  caA/caC:83|cAa/cGa:36|caA/caT:25|cAa/cTa:16|Ca...   \n",
       "3               A:134  gGt/gAt:757|gGt/gTt:657|Ggt/Tgt:380|Ggt/Cgt:16...   \n",
       "4                   0                                                  0   \n",
       "5                 E:3  Gat/Tat:11|gaT/gaA:2|Gat/Cat:2|gAt/gTt:1|gaT/g...   \n",
       "6                   0                                                  0   \n",
       "7                 V:1  Gat/Tat:11|gaT/gaA:2|Gat/Cat:2|gAt/gTt:1|gaT/g...   \n",
       "8                 H:2  Gat/Tat:11|gaT/gaA:2|Gat/Cat:2|gAt/gTt:1|gaT/g...   \n",
       "9                Y:11  Gat/Tat:11|gaT/gaA:2|Gat/Cat:2|gAt/gTt:1|gaT/g...   \n",
       "10                  0                                                  0   \n",
       "11                  0                                                  0   \n",
       "12                  0                                                  0   \n",
       "13                  0                                                  0   \n",
       "14                  0                                                  0   \n",
       "15                  0                                                  0   \n",
       "16                  0                                                  0   \n",
       "17              H:570  cGt/cAt:570|Cgt/Tgt:139|Cgt/Ggt:28|Cgt/Agt:16|...   \n",
       "18               S:16  cGt/cAt:570|Cgt/Tgt:139|Cgt/Ggt:28|Cgt/Agt:16|...   \n",
       "19              C:139  cGt/cAt:570|Cgt/Tgt:139|Cgt/Ggt:28|Cgt/Agt:16|...   \n",
       "20                  0                                                  0   \n",
       "21                  0                                                  0   \n",
       "22                  0                                                  0   \n",
       "23                  0                                                  0   \n",
       "24                  0                                                  0   \n",
       "25                  0                                                  0   \n",
       "26               K:31                    Gaa/Aaa:31|Gaa/Caa:10|gaA/gaC:1   \n",
       "27               A:12  Gag/Aag:575|gAg/gGg:21|Gag/Cag:20|gAg/gCg:12|g...   \n",
       "28                  0                                                  0   \n",
       "29              R:537  cAt/cGt:536|cAt/cTt:85|Cat/Tat:18|caT/caG:4|ca...   \n",
       "30                Q:7  cAt/cGt:536|cAt/cTt:85|Cat/Tat:18|caT/caG:4|ca...   \n",
       "31                Y:4                                Cat/Tat:4|cAt/cTt:1   \n",
       "32                  0                                                  0   \n",
       "33                  0                                                  0   \n",
       "34                  0                                                  0   \n",
       "35               E:32     Aaa/Gaa:32|aaA/aaT:4|aaATct/aaTAct:1|aAa/aCa:1   \n",
       "36               K:24  gTg/gAg:831|Gtg/Atg:29|GTg/AAg:24|gtG/gtA:4|GT...   \n",
       "37               N:24  Gat/Aat:24|gAt/gGt:21|Gat/Tat:2|gAt/gTt:2|gaT/...   \n",
       "38                  0                                                  0   \n",
       "39                E:4  gGa/gCa:34|gGa/gTa:11|Gga/Aga:8|gGa/gAa:4|GGa/...   \n",
       "40               A:34  gGa/gCa:34|gGa/gTa:11|Gga/Aga:8|gGa/gAa:4|GGa/...   \n",
       "41                  0                                                  0   \n",
       "42                  0                                                  0   \n",
       "43                  0                                                  0   \n",
       "44                  0                                                  0   \n",
       "\n",
       "                                     Genomic_Position  \\\n",
       "0                                                   0   \n",
       "1                                                   0   \n",
       "2       12:25380275_115|12:25380276_54|12:25380277_21   \n",
       "3      12:25398284_1557|12:25398285_617|12:25398283_1   \n",
       "4                                                   0   \n",
       "5          13:28592642_13|13:28592640_3|13:28592641_1   \n",
       "6                                                   0   \n",
       "7          13:28592642_13|13:28592640_3|13:28592641_1   \n",
       "8          13:28592642_13|13:28592640_3|13:28592641_1   \n",
       "9          13:28592642_13|13:28592640_3|13:28592641_1   \n",
       "10                                                  0   \n",
       "11                                                  0   \n",
       "12                                                  0   \n",
       "13                                                  0   \n",
       "14                                                  0   \n",
       "15                                                  0   \n",
       "16                                                  0   \n",
       "17                    2:209113112_583|2:209113113_183   \n",
       "18                    2:209113112_583|2:209113113_183   \n",
       "19                    2:209113112_583|2:209113113_183   \n",
       "20                                                  0   \n",
       "21                                                  0   \n",
       "22                                                  0   \n",
       "23                                                  0   \n",
       "24                                                  0   \n",
       "25                                                  0   \n",
       "26                       3:178928079_41|3:178928081_1   \n",
       "27       3:178936091_595|3:178936092_33|3:178936093_5   \n",
       "28                                                  0   \n",
       "29       3:178952085_622|3:178952084_18|3:178952086_7   \n",
       "30       3:178952085_622|3:178952084_18|3:178952086_7   \n",
       "31                        3:178952138_4|3:178952139_1   \n",
       "32                                                  0   \n",
       "33                                                  0   \n",
       "34                                                  0   \n",
       "35  7:140453134_32|7:140453132_4|7:140453133_1|7:1...   \n",
       "36       7:140453136_862|7:140453137_29|7:140453135_6   \n",
       "37        7:140453155_27|7:140453154_23|7:140453153_1   \n",
       "38                                                  0   \n",
       "39         7:140481402_51|7:140481403_9|7:140481401_1   \n",
       "40         7:140481402_51|7:140481403_9|7:140481401_1   \n",
       "41                                                  0   \n",
       "42                                                  0   \n",
       "43                                                  0   \n",
       "44                                                  0   \n",
       "\n",
       "                                Detailed_Cancer_Types  \\\n",
       "0                                                   0   \n",
       "1                                                   0   \n",
       "2   paad:932:61|luad:2057:24|mm:275:15|coad:712:14...   \n",
       "3   paad:932:728|luad:2057:535|coad:712:212|coadre...   \n",
       "4                                                   0   \n",
       "5                                 aml:198:16|all:92:1   \n",
       "6                                                   0   \n",
       "7                                 aml:198:16|all:92:1   \n",
       "8                                 aml:198:16|all:92:1   \n",
       "9                                 aml:198:16|all:92:1   \n",
       "10                                                  0   \n",
       "11                                                  0   \n",
       "12                                                  0   \n",
       "13                                                  0   \n",
       "14                                                  0   \n",
       "15                                                  0   \n",
       "16                                                  0   \n",
       "17  lggnos:544:424|aastr:82:39|gbm:688:38|aodg:43:...   \n",
       "18  lggnos:544:424|aastr:82:39|gbm:688:38|aodg:43:...   \n",
       "19  lggnos:544:424|aastr:82:39|gbm:688:38|aodg:43:...   \n",
       "20                                                  0   \n",
       "21                                                  0   \n",
       "22                                                  0   \n",
       "23                                                  0   \n",
       "24                                                  0   \n",
       "25                                                  0   \n",
       "26  brca:1345:8|blca:852:7|idc:870:5|lggnos:544:3|...   \n",
       "27  brca:1345:82|blca:852:79|idc:870:59|coadread:6...   \n",
       "28                                                  0   \n",
       "29  brca:1345:174|idc:870:119|ilc:184:40|uec:339:3...   \n",
       "30  brca:1345:174|idc:870:119|ilc:184:40|uec:339:3...   \n",
       "31  luad:2057:1|coad:712:1|ccrcc:771:1|blca:852:1|...   \n",
       "32                                                  0   \n",
       "33                                                  0   \n",
       "34                                                  0   \n",
       "35  skcm:787:8|luad:2057:6|prad:1366:6|dlbcl:246:3...   \n",
       "36  skcm:787:355|thpa:486:293|coad:712:54|coadread...   \n",
       "37  luad:2057:10|coad:712:8|blca:852:5|skcm:787:5|...   \n",
       "38                                                  0   \n",
       "39  luad:2057:18|skcm:787:12|coad:712:4|mm:275:3|l...   \n",
       "40  luad:2057:18|skcm:787:12|coad:712:4|mm:275:3|l...   \n",
       "41                                                  0   \n",
       "42                                                  0   \n",
       "43                                                  0   \n",
       "44                                                  0   \n",
       "\n",
       "                                          Organ_Types ref        ct  \\\n",
       "0                                                   0   0         0   \n",
       "1                                                   0   0         0   \n",
       "2   pancreas:1059:64|bowel:1782:31|lung:2761:28|bl...   Q  pancreas   \n",
       "3   pancreas:1059:745|lung:2761:570|bowel:1782:500...   G     bowel   \n",
       "4                                                   0   0         0   \n",
       "5                                        blood:890:17   D     blood   \n",
       "6                                                   0   0         0   \n",
       "7                                        blood:890:17   D     blood   \n",
       "8                                        blood:890:17   D     blood   \n",
       "9                                        blood:890:17   D     blood   \n",
       "10                                                  0   0         0   \n",
       "11                                                  0   0         0   \n",
       "12                                                  0   0         0   \n",
       "13                                                  0   0         0   \n",
       "14                                                  0   0         0   \n",
       "15                                                  0   0         0   \n",
       "16                                                  0   0         0   \n",
       "17  cnsbrain:2270:612|biliarytract:358:57|skin:974...   R  cnsbrain   \n",
       "18  cnsbrain:2270:612|biliarytract:358:57|skin:974...   R  cnsbrain   \n",
       "19  cnsbrain:2270:612|biliarytract:358:57|skin:974...   R  cnsbrain   \n",
       "20                                                  0   0         0   \n",
       "21                                                  0   0         0   \n",
       "22                                                  0   0         0   \n",
       "23                                                  0   0         0   \n",
       "24                                                  0   0         0   \n",
       "25                                                  0   0         0   \n",
       "26  breast:2561:16|bladder:958:8|cnsbrain:2270:6|l...   E    breast   \n",
       "27  breast:2561:174|bowel:1782:97|bladder:958:90|l...   E    breast   \n",
       "28                                                  0   0         0   \n",
       "29  breast:2561:361|bowel:1782:54|uterus:618:51|es...   H    breast   \n",
       "30  breast:2561:361|bowel:1782:54|uterus:618:51|es...   H    breast   \n",
       "31  lung:2761:1|bowel:1782:1|kidney:1304:1|bladder...   H      lung   \n",
       "32                                                  0   0         0   \n",
       "33                                                  0   0         0   \n",
       "34                                                  0   0         0   \n",
       "35  skin:974:8|lung:2761:6|prostate:1379:6|lymph:3...   K      skin   \n",
       "36  skin:974:357|thyroid:618:316|bowel:1782:113|lu...   V      skin   \n",
       "37  bowel:1782:12|lung:2761:10|bladder:958:6|skin:...   D     bowel   \n",
       "38                                                  0   0         0   \n",
       "39  lung:2761:23|skin:974:13|bowel:1782:6|blood:89...   G      lung   \n",
       "40  lung:2761:23|skin:974:13|bowel:1782:6|blood:89...   G      lung   \n",
       "41                                                  0   0         0   \n",
       "42                                                  0   0         0   \n",
       "43                                                  0   0         0   \n",
       "44                                                  0   0         0   \n",
       "\n",
       "                                              Samples total_variant_count  \n",
       "0                                                   0                 0.0  \n",
       "1                                                   0                 0.0  \n",
       "2   bowel:9|pancreas:3|thyroid:2|biliarytract:1|bl...                20.0  \n",
       "3   lung:58|bowel:36|uterus:11|unk:5|breast:4|blad...               134.0  \n",
       "4                                                   0                 0.0  \n",
       "5                                             blood:3                 3.0  \n",
       "6                                                   0                 0.0  \n",
       "7                                             blood:1                 1.0  \n",
       "8                                             blood:2                 2.0  \n",
       "9                                            blood:11                11.0  \n",
       "10                                                  0                 0.0  \n",
       "11                                                  0                 0.0  \n",
       "12                                                  0                 0.0  \n",
       "13                                                  0                 0.0  \n",
       "14                                                  0                 0.0  \n",
       "15                                                  0                 0.0  \n",
       "16                                                  0                 0.0  \n",
       "17  cnsbrain:553|prostate:6|blood:5|headandneck:2|...               570.0  \n",
       "18                 cnsbrain:13|blood:2|biliarytract:1                16.0  \n",
       "19  biliarytract:48|cnsbrain:26|skin:23|blood:15|l...               139.0  \n",
       "20                                                  0                 0.0  \n",
       "21                                                  0                 0.0  \n",
       "22                                                  0                 0.0  \n",
       "23                                                  0                 0.0  \n",
       "24                                                  0                 0.0  \n",
       "25                                                  0                 0.0  \n",
       "26  breast:13|cnsbrain:5|bladder:4|esophagusstomac...                31.0  \n",
       "27  breast:4|bowel:2|cnsbrain:2|uterus:2|bladder:1...                12.0  \n",
       "28                                                  0                 0.0  \n",
       "29  breast:320|bowel:45|uterus:33|cnsbrain:22|esop...               537.0  \n",
       "30  bowel:2|breast:1|cervix:1|headandneck:1|kidney...                 7.0  \n",
       "31                  bladder:1|bowel:1|kidney:1|lung:1                 4.0  \n",
       "32                                                  0                 0.0  \n",
       "33                                                  0                 0.0  \n",
       "34                                                  0                 0.0  \n",
       "35  skin:7|prostate:6|lung:5|lymph:3|bowel:2|thyro...                32.0  \n",
       "36     skin:20|unk:2|esophagusstomach:1|headandneck:1                24.0  \n",
       "37  bowel:5|skin:5|lung:4|bladder:3|unk:3|biliaryt...                24.0  \n",
       "38                                                  0                 0.0  \n",
       "39                                     skin:3|bowel:1                 4.0  \n",
       "40  lung:15|bowel:5|blood:2|prostate:2|skin:2|adre...                34.0  \n",
       "41                                                  0                 0.0  \n",
       "42                                                  0                 0.0  \n",
       "43                                                  0                 0.0  \n",
       "44                                                  0                 0.0  \n",
       "\n",
       "[45 rows x 131 columns]"
      ]
     },
     "execution_count": 59,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "merged_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "metadata": {},
   "outputs": [],
   "source": [
    "merged_df[\"hotspot_rules\"] = merged_df.apply(lambda x: cancer_hotspot_rules(x), axis=1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "metadata": {},
   "outputs": [],
   "source": [
    "merged_df.to_csv(\"anno_with_hotspot.csv\", index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "base",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
